---
title: '![PhylogatR](Segoe1.png)' 
subtitle: 'Genetic Diversity'
author: "by Tara Pelletier and Danielle Parsons"
date: "Updated 14 Oct 2020"
output:
  html_document:
    toc: yes
    toc_depth: 1
    toc_float: yes
---
# 1. Purpose
***
#### **Genetic diversity** refers to variation in the base pairs of DNA. We can observe the patterns in genetic diversity to make inferences about evolutionary processes such as migration, selection, and drift. Genetic diversity among individuals within a species might increase with geographic distance; therefore, you might expect the larger a species range, the higher the genetic diversity.

#### The code presented here loops through all the sequence alignments from a [phylogatR](https://phylogatr.org/) download and calculates genetic diversity & the size of the geographic range of sampling. Next, it runs a regression analysis to ask the question: does the size of the geographic range of sampling predict levels of genetic diversity?

The data you will download from PhylogatR will consist of a series of folders for each species. In each folder there will be unaligned and aligned fasta files for each locus[^1] that is present for that species in the database. These files contain the genetic data for a species. Unaligned fasta files will have the .fa extension and aligned fasta files will have the .afa extension. In addition to these files, each species folder will contain an occurrence file, named "occurrences.txt". This file contains metadata associated with individual species occurrences, such as GenBank accession number, GBIF ID, and geographic coordinates. In the root folder of your download, you will find a "genes.txt" file that includes metadata from your download, such as taxonomy for each species, number of individuals per alignment, and how many sequences were removed after data cleaning steps.

[^1]: A locus is a specific position in the genome where a particular gene or genetic marker is located. As such, not all loci necessarily correspond to a gene. Because PhylogatR alignments reflect genetic data uploaded to GenBank, you may also encounter alignments representing anonymous loci.

First, load all the necessary packages. If you have not already installed these packages, you will need to do that first. See [here](https://www.neonscience.org/packages-in-r){target="_blank" rel="noopener"} for more information on package installation in R.

```{r message=FALSE, warning=FALSE}
#load necessary packages
library(tools)
library(seqinr)
library(pegas)
library(raster)
library(rgeos)
library(ggplot2)
library(dplyr)
```

# 2. Import sequence data
***
In this case, there is a folder on my Desktop called “phylogatr-results”. When you import your own data from PhylogatR, you will need to change the path to the folder accordingly.

This example uses data from the genus *Plethodon*, a group of terrestrial salamanders from North America.

First, we are going to create a list of all the sequence alignments and put them in an object called "myfiles". 
```{r}
myfiles <- list.files(path = "/Users/tpelletier/Desktop/phylogatr-results", pattern = ".afa", full.names=TRUE, recursive = TRUE)
```

# 3. Calculate genetic diversity and sampling range
***
Here we loop through the list stored in "myfiles" to calculate genetic diversity for each sequence alignment and the area that the geographic coordinates from each species occupies. All data will be written to a csv file called "phylogatr-data.csv" and stored at the path specified at the end of the loop. Here, we are storing the file on my desktop, but you will need to change the path to where you want the file stored accordingly.
```{r message=F, warning=F}
for (f in myfiles) {

  #get species and gene name
  sp <- basename(f)
  sp <- file_path_sans_ext(sp)
  
  #read in fasta file
  seq <- fasta2DNAbin(f, quiet=TRUE)
  
  #get number of sequences
  n <- nrow(seq)

  #calculate nucleotide diversity
  pi <- nuc.div(seq)
  
  #navigate to species folder to get GPS coordinates
  s <- sub("/[^/]+$", "", f)
  c <- paste(s, "/occurrences.txt", sep="")
  occurrences <- read.delim(c)
  xy <- occurrences[,c(3:4)]
    
  #get area based on species GPS coordinates
  ch <- chull(xy)
  coords <- xy[c(ch, ch[1]), ]
  sp_poly <- SpatialPolygons(list(Polygons(list(Polygon(coords)), ID=1)))
  area<-gArea(sp_poly)
  format(area, scientific = FALSE)
  
  #write data to file
  write.table(data.frame(sp, pi, area, n), file="/Users/tpelletier/Desktop/phylogatr-data.csv", quote=FALSE, row.names=FALSE, col.names=!file.exists("phylogatr-data.csv"), append=TRUE, sep=",")
}
```

# 3. Regression analysis
***
First, import the output from the previous step. If necessary, change the path to the file accordingly.
```{r}
data <- read.csv("/Users/tpelletier/Desktop/phylogatr-data.csv")
```

Run the regression analysis.
```{r}
r <- lm(pi~area, data=data)
summary(r)
```
In this case, the relationship between area and pi is almost zero (R-squared = 0.001) and not significant (p = 0.86). Let's plot it so we can see what our data look like.
```{r}
ggplot(data, aes(x=area, y=pi)) + geom_point(color="red")
```

There are some clear outliers in these data, so I am going to explore these further.
```{r}
outlier.pi <- filter(data, pi > 0.15)
outlier.pi
```
```{r}
outlier.area <- filter(data, area > 80)
outlier.area
```

We chose to explore the high level of genetic diversity of *P. serratus* by looking at the sequence alignment for this species and gene (see [here](https://phylogatr.org/assets/modules/Checking_Your_Data.html) for guidance. After visually inspecting the alignment, we suspect that one of the individual sequences in this alignment might have been mis-idantified and so we will remove this species from the analysis for now. However, in some circumstances you could remove an odd looking sequence, and keep that species in the analysis.

```{r, echo=FALSE, out.width="50%"}
knitr::include_graphics("0414.jpeg")
```
<font size="1"> Photo credit: apmbibiaweb.org: Tod Pierson </font>

Given what we know about the distribution of *P. cinereus* this species might be an outlier, but is likley not a mistake in the data. This species has a VERY large distribution compared to other *Plethodon* species, but it is interesting that correspondingly its genetic diveristy isn't higher, as expected.

```{r, echo=FALSE, out.width="50%"}
knitr::include_graphics("0005.jpeg")
```
<font size="1"> Photo credit: apmbibiaweb.org: John White </font>
```{r, echo=FALSE, out.width="50%"}
knitr::include_graphics("map.png")
```
<font size="1"> Map credit: iucnredlist.org </font>

In most cases where you have an obvious geographic outlier, we recommend plotting the GPS coordinates for any outlier species and making a decision about the GPS coordinates for that species. Once the GPS coordinates for any outlier species are cleaned up, those species may be able to remain in the analysis. See [here](https://phylogatr.org/assets/modules/Checking_Your_Data.html) for guidance. For now, we will simply remove it and redo our analysis. 

```{r}
data_pruned <- filter(data, area < 80, pi < 0.15)
```

```{r}
r_pruned <- lm(pi~area, data=data_pruned)
summary(r_pruned)
ggplot(data_pruned, aes(x=area, y=pi)) + geom_point(color="red")
```

This changes things, but not in any meaningful way (R-squared = 0.0.009, p = 0.6373). We are not advocating removing data until you obtain significant results, but I am curious what will happen after removing the other points that stand out. 

```{r}
data_pruned2 <- filter(data_pruned, area < 0.2)
```

```{r}
r_pruned2 <- lm(pi~area, data=data_pruned2)
summary(r_pruned2)
ggplot(data_pruned2, aes(x=area, y=pi)) + geom_point(color="red")
```

Our hypothesis was not supported. The size of the geographic range, as measured here, does not explain variation in intra-specific genetic diveristy and there are likley other factors that influence genetic diversity within the species of this group. However, the removal of outliers did influnce the results showing more of a trend (R-squared = 0.1797, p = 0.0796) and those outliers might be worth explaring futher.

