<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>6 Conservation genetics | PhylogatR</title>
  <meta name="description" content="" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="6 Conservation genetics | PhylogatR" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="6 Conservation genetics | PhylogatR" />
  
  
  

<meta name="author" content="list of people" />


<meta name="date" content="2022-03-12" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="population-genetics.html"/>

<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>




</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Background</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#how-to-use-this-book"><i class="fa fa-check"></i><b>1.1</b> How to use this book</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#prerequisites-if-using-r"><i class="fa fa-check"></i><b>1.2</b> Prerequisites if using R</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="index.html"><a href="index.html#r-and-rstudio"><i class="fa fa-check"></i><b>1.2.1</b> R and RStudio</a></li>
<li class="chapter" data-level="1.2.2" data-path="index.html"><a href="index.html#r-packages"><i class="fa fa-check"></i><b>1.2.2</b> R packages</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>2</b> Introduction</a>
<ul>
<li class="chapter" data-level="2.1" data-path="introduction.html"><a href="introduction.html#data-in-the-biological-sciences"><i class="fa fa-check"></i><b>2.1</b> Data in the biological sciences</a></li>
<li class="chapter" data-level="2.2" data-path="introduction.html"><a href="introduction.html#why-species"><i class="fa fa-check"></i><b>2.2</b> Why species?</a></li>
<li class="chapter" data-level="2.3" data-path="introduction.html"><a href="introduction.html#the-scientific-method"><i class="fa fa-check"></i><b>2.3</b> The Scientific Method</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="dna.html"><a href="dna.html"><i class="fa fa-check"></i><b>3</b> DNA</a>
<ul>
<li class="chapter" data-level="3.1" data-path="dna.html"><a href="dna.html#practice"><i class="fa fa-check"></i><b>3.1</b> Practice</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html"><i class="fa fa-check"></i><b>4</b> Biodiversity Databases</a>
<ul>
<li class="chapter" data-level="4.1" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#global-biodiversity-information-facility-gbif"><i class="fa fa-check"></i><b>4.1</b> Global Biodiversity Information Facility (GBIF)</a></li>
<li class="chapter" data-level="4.2" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#genbank"><i class="fa fa-check"></i><b>4.2</b> GenBank</a></li>
<li class="chapter" data-level="4.3" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#bold"><i class="fa fa-check"></i><b>4.3</b> BOLD</a></li>
<li class="chapter" data-level="4.4" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#phylogatr"><i class="fa fa-check"></i><b>4.4</b> <em>PhylogatR</em></a></li>
<li class="chapter" data-level="4.5" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#using-pylogatr"><i class="fa fa-check"></i><b>4.5</b> Using <em>PylogatR</em></a></li>
<li class="chapter" data-level="4.6" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#practice-1"><i class="fa fa-check"></i><b>4.6</b> Practice</a>
<ul>
<li class="chapter" data-level="4.6.1" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#check-for-outliers"><i class="fa fa-check"></i><b>4.6.1</b> Check for outliers</a></li>
<li class="chapter" data-level="4.6.2" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#other-checking-your-data"><i class="fa fa-check"></i><b>4.6.2</b> Other checking your data</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="population-genetics.html"><a href="population-genetics.html"><i class="fa fa-check"></i><b>5</b> Population genetics</a></li>
<li class="chapter" data-level="6" data-path="conservation-genetics.html"><a href="conservation-genetics.html"><i class="fa fa-check"></i><b>6</b> Conservation genetics</a>
<ul>
<li class="chapter" data-level="6.1" data-path="conservation-genetics.html"><a href="conservation-genetics.html#loss-of-genetic-diversity"><i class="fa fa-check"></i><b>6.1</b> Loss of genetic diversity</a>
<ul>
<li class="chapter" data-level="6.1.1" data-path="conservation-genetics.html"><a href="conservation-genetics.html#small-population-size"><i class="fa fa-check"></i><b>6.1.1</b> Small population size</a></li>
<li class="chapter" data-level="6.1.2" data-path="conservation-genetics.html"><a href="conservation-genetics.html#genetic-drift"><i class="fa fa-check"></i><b>6.1.2</b> Genetic drift</a></li>
<li class="chapter" data-level="6.1.3" data-path="conservation-genetics.html"><a href="conservation-genetics.html#gene-flow"><i class="fa fa-check"></i><b>6.1.3</b> Gene flow</a></li>
<li class="chapter" data-level="6.1.4" data-path="conservation-genetics.html"><a href="conservation-genetics.html#inbreeding"><i class="fa fa-check"></i><b>6.1.4</b> Inbreeding</a></li>
</ul></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"><img src="Segoe1.png" title="fig:" style="width:75.0%" alt="PhylogatR" /></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="conservation-genetics" class="section level1" number="6">
<h1><span class="header-section-number">6</span> Conservation genetics</h1>
<p>Conservation genetics is the use of this genetic information to inform management decisions to preserve biodiversity.</p>
<div id="loss-of-genetic-diversity" class="section level2" number="6.1">
<h2><span class="header-section-number">6.1</span> Loss of genetic diversity</h2>
<p>Loss of genetic diversity is a common concern for conservation biologists. Many populations of threatened and endangered species suffer from low diversity of genes at the individual and population levels. But how is genetic diversity lost? There are several factors that can contribute, singly or in concert together, to the loss of genetic variation both in individuals and populations.</p>
<div id="small-population-size" class="section level3" number="6.1.1">
<h3><span class="header-section-number">6.1.1</span> Small population size</h3>
<p>Endangered species typically have declining numbers of individuals. As individuals are lost, so is the variation of the DNA that they carry. Factors that increase mortality or reduce birth rates, such as disease, pollution, overhunting, or low resource availability due to habitat destruction all cause populations to shrink. Additionally, destruction of a species habitat in particular places can create smaller populations by breaking up the remaining habitat areas that are suitable for the species. This <strong>habitat fragmentation</strong> also isolates populations from each other; populations are trapped in pockets of remaining habitat but are unable to interact with other populations due to inhospitable habitat or barriers like roads or fences blocking their movement.</p>
<p>The total number of individuals within a population is not the only factor that needs to be considered when thinking about how population size affects genetic diversity. Within any population, only a subset of the individuals will be capable of reproducing at a given time; others may be too young, too old, or too unhealthy to reproduce. The number of breeding individuals in a population is called the <strong>effective population size</strong>, and is necessarily smaller than the total population size. Because it is only these breeding individuals who will pass their DNA on to the next generation, the effective population size directly impacts genetic diversity. Certain factors, such as selective hunting or poaching, can disproportionately reduce effective population size. For example, older adult African elephants have been the primary target of poachers in search of ivory because elephant tusk size increases with age. This targeting of adults has affected elephant population demographics, skewing populations younger overall and inhibiting reproduction in heavily poached populations.</p>
<p>Because each individual in a population can only carry a certain number of alleles for each gene, small populations tend to have fewer alleles overall and the alleles that are present may only be carried by a handful of individuals, making them rare. These <strong>rare alleles</strong> are vulnerable to loss; if anything prevents those individuals from breeding, they will not be present in the next generation of that population. However, this is not the only reason that small population size is concerning. For <strong>sexually-reproducing</strong> species, smaller populations also means fewer potential mates; lack of options for mates may result in fewer individuals breeding (further reducing effective population size) or reduced offspring quality due to poor compatibility of parents who might not have paired if they had had more choices of mates.</p>
</div>
<div id="genetic-drift" class="section level3" number="6.1.2">
<h3><span class="header-section-number">6.1.2</span> Genetic drift</h3>
<p><strong>Genetic drift</strong> is a process in which populations diverge genetically from other populations of the same species. This occurs partly through the loss of rare alleles due to random occurrences that have nothing to do with the function or adaptive value of the allele in question. When populations are small, the loss of genetic diversity due to random processes is bigger. <strong><em>Stuff here about random sampling of alleles with image.</em></strong></p>
<p>Conservation biologists must consider the negative impacts of genetic drift not only in existing populations, but also in captive breeding programs and in newly-established populations created during reintroduction efforts. <strong>Founder effects</strong> occur whenever a subset of an existing population is used to create a new one. These founding individuals may not carry a representative proportion of the genetic diversity in the existing population, especially if only a small number of founders are used. This in turn leads to the new population having lower genetic diversity than the previous population it was derived from. Conservationists selecting individuals for use in captive breeding and reintroduction efforts must pay careful attention to the genotypes of these individuals to ensure that as many alleles as possible from the parent population are carried forward into the new populations, minimizing loss of genetic diversity in this process.</p>
<p>A slightly different example of a random loss of alleles is a <strong>population bottleneck</strong>. For example, if a hiker accidentally stepping on a group of brown and green beetles; if the hiker’s foot happens to fall on all of the green beetles in the group, the allele that causes this green coloration may be lost in this beetle population, even though an individual beetle’s color did not make it more or less likely to be stepped on. This example also highlights a connection between small population size and genetic drift; if the beetle population were larger, it is unlikely that the hiker’s foot would randomly land on all of the green beetles. Although genetic drift occurs in all populations, larger populations are less likely to suffer extreme effects such as total loss of an allele than small populations, due to increased redundancy in the larger populations; larger population size simply means there are probably more copies of any given allele. Small populations are more likely to have alleles that are carried by only a few individuals, and these rare alleles are vulnerable to loss by random chance.</p>
</div>
<div id="gene-flow" class="section level3" number="6.1.3">
<h3><span class="header-section-number">6.1.3</span> Gene flow</h3>
<p>Isolation of populations also makes them more vulnerable to the effects of genetic drift. Isolation prevents <strong>gene flow</strong>, the sharing of alleles between populations through the dispersal of individuals into and out of these different breeding groups. A population that frequently trades individuals with other nearby populations of the species may lose a particular allele through genetic drift, but then regain it through gene flow when a new member carrying that allele joins the population. Isolated populations do not have this outside source of alleles; once an allele is lost through drift, it remains gone. Conservation biologists are particularly concerned about small, isolated populations because these populations drift faster than larger, better-connected populations. Habitat fragmentation contributes to population isolation as well as reducing population size, and so is a major factor contributing to loss of genetic diversity.</p>
</div>
<div id="inbreeding" class="section level3" number="6.1.4">
<h3><span class="header-section-number">6.1.4</span> Inbreeding</h3>
<p>When two individuals with many of the same alleles breed, their offspring are likely to have lower genetic diversity than the offspring of individuals with very different alleles. They are also more likely to be <strong>homozygous</strong> at many loci; in other words, they are likely to carry two copies of the same allele for a given gene, because that was the allele carried by both of their parents. In contrast, an individual with two different alleles for a gene is considered <strong>heterozygous</strong>. Breeding of genetically similar individuals who produce highly homozygous offspring is referred to as <strong>inbreeding</strong>, and is often a result of matings between close relatives such as siblings.</p>
<p>Inbreeding occurs when mate choice is limited, such as when overall or effective population size is very small. In these situations, relatives may be the only available mate options. This is especially common in isolated populations, where dispersal to a new population in search of mates is not possible and unrelated individuals are not immigrating in.</p>
<p>In some cases, inbreeding can be exacerbated by population-specific changes in behaviors that affect mate choice. In these situations, individuals may prefer to breed with members of their own population, who are more genetically similar to them, even when presented with potential mates from outside populations. For example, a study of rufous-collared sparrows found that female sparrows preferred the songs of males from their own population to those from a neighboring population that sang slightly differently, a preference that likely limits gene flow between these two populations (Danner et al. 2011)</p>
<p>Inbreeding is of concern in conservation biology because of its potential to reduce genetic diversity at both the individual and population levels. One reason for concern is inbreeding depression, a situation in which homozygosity at one or more loci produces individuals who are lower quality and less fit. This happens when certain alleles are maladaptive. For example, a gene may code for a particular protein, but a mutation in one allele for that gene causes that allele to produce a malformed protein that cannot be used. Such ‘broken’ alleles can be compensated for by the presence of a different allele for the same gene; in this example, a heterozygous individual could carry one copy of the broken allele and one of a different allele that produces a working version of the protein. However, homozygous individuals are at risk of carrying two broken versions, and thus being unable to produce the necessary protein. While natural selection often removes deleterious mutations from populations over time, alleles with negative mutations can remain in populations indefinitely if they are recessive and so their maladaptive effects only appear in individuals that are homozygous for them.
Heterozygosity is also beneficial at the population level. Different combinations of alleles can produce unique phenotypes not created by any allele paired with itself. This increased diversity of phenotypes in a population produces greater flexibility when faced with environmental challenges. A diverse population is more likely to have individuals within it whose combination of traits is adaptive in the current environmental conditions and so are able to survive the challenge and pass on those alleles. Therefore, populations that are more heterozygous overall have greater potential to adapt to environmental changes, rather than be driven extinct by them.</p>
<p>Measuring heterozygosity within and between populations
Heterozygosity is a measure of genetic variation that can be estimated within an individual, within a population of individuals of the same species, and between populations of the same species.</p>
<p>Heterozygosity in a population is the proportion of individuals that are heterozygous at that locus, and can be averaged across many loci (H). This estimate is used as a measure of genetic variation within a population to assess population stability, in other words, is there enough genetic variation among individuals within a population to avoid a risk of extinction? H can decrease to small population size and inbreeding as discussed above, and can be the result of a population bottleneck. A bottleneck is an event that suffers a fast (evolutionary speaking) reduction in population size where may alleles are lost. In addition to a decrease in genetic diversity, events such as that described above often result in population fragmentation among populations within a species. This fragmentation maintains a small effective population size within each fragment and reduces gene flow within the species. These characteristics put these populations at a higher risk of extinction. A reduction in gene flow leads to population differentiation and can be measured using genetic data. EXPLAIN FST HERE. These types of data can inform conservation management decisions by guiding the development of dispersal corridors and/or translocation events to restore levels of genetic variation within species.</p>
<p>Image that represents heterozygosity at each level and includes fragmentation</p>
<p>Heterozygosity example
In the 1800s and most of the 1900s black bears in FL were hunted without regulation and much of their habitat was destroyed due to human development. By the 1970s, only 300-400 bears remained. This type of reduction in population can severely reduce genetic diversity and black bears were subsequently listed as threatened in FL (cite). Cryptorchidism (undescended tesicles, a common phenotype when mammals suffer from inbreeding depression) was observed in black bears of FL in the mid-nineties (Dunbar et al. 1996). Not only did the FL black bears suffer a decrease in population size and a loss of genetic diversity, their habitat was severely fragmented. Researchers decided to use genetic data to assess the levels of genetic variation within and between populations (Dixon et al. 2007). The used 12 genetic loci to estimate heterozygosity within and between populations. They found that genetic diversity in the FL balck bears was lower in populations that are smaller and less connected to other populations, and that populations with at least 200 individuals is needed to maintain levels of genetic variation to match that of unaffected balck bears in other parts of the country. Finally, even though many of these populations are close enough for bears to disperse, the ability to cross highways is limited (find recent numbers), so the researchers recommend creating dispersal corridors, either naturally, or with the use of crossing structures. In this case, considering populations as units of conservation is important, rather than considering the whole species. For example, if one small population is particularly vulnerable, it is much more cost-effective to focus management decisions on this population, rather than the entire geographic region.</p>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="population-genetics.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
