<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>13 Conservation genetics | PhylogatR</title>
  <meta name="description" content="" />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="13 Conservation genetics | PhylogatR" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="13 Conservation genetics | PhylogatR" />
  
  
  

<meta name="author" content="list of people" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="predictive-phylogeography.html"/>

<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />











<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Background</a><ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#prerequisites"><i class="fa fa-check"></i><b>1.1</b> Prerequisites</a><ul>
<li class="chapter" data-level="1.1.1" data-path="index.html"><a href="index.html#r-and-rstudio"><i class="fa fa-check"></i><b>1.1.1</b> R and RStudio</a></li>
<li class="chapter" data-level="1.1.2" data-path="index.html"><a href="index.html#r-packages"><i class="fa fa-check"></i><b>1.1.2</b> R packages</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>2</b> Introduction</a><ul>
<li class="chapter" data-level="2.1" data-path="introduction.html"><a href="introduction.html#data-in-the-biological-sciences"><i class="fa fa-check"></i><b>2.1</b> Data in the biological sciences</a></li>
<li class="chapter" data-level="2.2" data-path="introduction.html"><a href="introduction.html#why-species"><i class="fa fa-check"></i><b>2.2</b> Why species?</a></li>
<li class="chapter" data-level="2.3" data-path="introduction.html"><a href="introduction.html#the-scientific-method"><i class="fa fa-check"></i><b>2.3</b> The Scientific Method</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="the-form-and-structure-of-sequence-data.html"><a href="the-form-and-structure-of-sequence-data.html"><i class="fa fa-check"></i><b>3</b> The form and structure of sequence data</a><ul>
<li class="chapter" data-level="3.1" data-path="the-form-and-structure-of-sequence-data.html"><a href="the-form-and-structure-of-sequence-data.html#dna-amino-acids-make-proteins"><i class="fa fa-check"></i><b>3.1</b> DNA &amp; amino acids make proteins</a></li>
<li class="chapter" data-level="3.2" data-path="the-form-and-structure-of-sequence-data.html"><a href="the-form-and-structure-of-sequence-data.html#snps"><i class="fa fa-check"></i><b>3.2</b> SNPs</a></li>
<li class="chapter" data-level="3.3" data-path="the-form-and-structure-of-sequence-data.html"><a href="the-form-and-structure-of-sequence-data.html#synonymous-vs.-nonsynonymous-variation"><i class="fa fa-check"></i><b>3.3</b> synonymous vs. nonsynonymous variation</a></li>
<li class="chapter" data-level="3.4" data-path="the-form-and-structure-of-sequence-data.html"><a href="the-form-and-structure-of-sequence-data.html#organellar-genomes-mitochondria-chloroplast"><i class="fa fa-check"></i><b>3.4</b> organellar genomes (mitochondria / chloroplast)</a></li>
<li class="chapter" data-level="3.5" data-path="the-form-and-structure-of-sequence-data.html"><a href="the-form-and-structure-of-sequence-data.html#barcoding-genes"><i class="fa fa-check"></i><b>3.5</b> barcoding genes</a></li>
<li class="chapter" data-level="3.6" data-path="the-form-and-structure-of-sequence-data.html"><a href="the-form-and-structure-of-sequence-data.html#functions-of-common-barcoding-genes"><i class="fa fa-check"></i><b>3.6</b> functions of common barcoding genes</a></li>
<li class="chapter" data-level="3.7" data-path="the-form-and-structure-of-sequence-data.html"><a href="the-form-and-structure-of-sequence-data.html#examples"><i class="fa fa-check"></i><b>3.7</b> examples?</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html"><i class="fa fa-check"></i><b>4</b> Biodiversity Databases</a><ul>
<li class="chapter" data-level="4.1" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#global-biodiversity-information-facility-gbif"><i class="fa fa-check"></i><b>4.1</b> Global Biodiversity Information Facility (GBIF)</a></li>
<li class="chapter" data-level="4.2" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#genbank"><i class="fa fa-check"></i><b>4.2</b> GenBank</a></li>
<li class="chapter" data-level="4.3" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#phylogatr"><i class="fa fa-check"></i><b>4.3</b> PhylogatR</a></li>
<li class="chapter" data-level="4.4" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#using-pylogatr"><i class="fa fa-check"></i><b>4.4</b> Using PylogatR</a></li>
<li class="chapter" data-level="4.5" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#practice"><i class="fa fa-check"></i><b>4.5</b> Practice</a><ul>
<li class="chapter" data-level="4.5.1" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#data-checking-and-cleaning"><i class="fa fa-check"></i><b>4.5.1</b> Data checking and cleaning</a></li>
<li class="chapter" data-level="4.5.2" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#outliers"><i class="fa fa-check"></i><b>4.5.2</b> Outliers</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="natural-history-collections.html"><a href="natural-history-collections.html"><i class="fa fa-check"></i><b>5</b> Natural History Collections</a></li>
<li class="chapter" data-level="6" data-path="genetic-diversity-measures.html"><a href="genetic-diversity-measures.html"><i class="fa fa-check"></i><b>6</b> Genetic diversity measures</a><ul>
<li class="chapter" data-level="6.1" data-path="genetic-diversity-measures.html"><a href="genetic-diversity-measures.html#difference"><i class="fa fa-check"></i><b>6.1</b> % difference</a></li>
<li class="chapter" data-level="6.2" data-path="genetic-diversity-measures.html"><a href="genetic-diversity-measures.html#model-corrected-difference-modeling-dna-variation"><i class="fa fa-check"></i><b>6.2</b> model corrected difference (modeling DNA variation)</a></li>
<li class="chapter" data-level="6.3" data-path="genetic-diversity-measures.html"><a href="genetic-diversity-measures.html#summary-statistics-pi-thetah-d-s"><i class="fa fa-check"></i><b>6.3</b> summary statistics (pi, thetaH, D, S)</a></li>
<li class="chapter" data-level="6.4" data-path="genetic-diversity-measures.html"><a href="genetic-diversity-measures.html#genetic-diversity-over-geographic-space-pie-charts"><i class="fa fa-check"></i><b>6.4</b> genetic diversity over geographic space pie charts</a></li>
<li class="chapter" data-level="6.5" data-path="genetic-diversity-measures.html"><a href="genetic-diversity-measures.html#practice-1"><i class="fa fa-check"></i><b>6.5</b> practice</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="population-genetic-structure.html"><a href="population-genetic-structure.html"><i class="fa fa-check"></i><b>7</b> Population genetic structure</a><ul>
<li class="chapter" data-level="7.1" data-path="population-genetic-structure.html"><a href="population-genetic-structure.html#genetic-drift-and-population-size"><i class="fa fa-check"></i><b>7.1</b> genetic drift and population size</a></li>
<li class="chapter" data-level="7.2" data-path="population-genetic-structure.html"><a href="population-genetic-structure.html#clustering-methods"><i class="fa fa-check"></i><b>7.2</b> clustering methods</a></li>
<li class="chapter" data-level="7.3" data-path="population-genetic-structure.html"><a href="population-genetic-structure.html#genetic-isolation-by-distance-geography"><i class="fa fa-check"></i><b>7.3</b> genetic isolation by distance (geography)</a></li>
<li class="chapter" data-level="7.4" data-path="population-genetic-structure.html"><a href="population-genetic-structure.html#genetic-isolation-by-environment-temperature-precipitation"><i class="fa fa-check"></i><b>7.4</b> genetic isolation by environment (temperature, precipitation)</a></li>
<li class="chapter" data-level="7.5" data-path="population-genetic-structure.html"><a href="population-genetic-structure.html#genetic-isolation-by-resistance"><i class="fa fa-check"></i><b>7.5</b> genetic isolation by resistance</a></li>
<li class="chapter" data-level="7.6" data-path="population-genetic-structure.html"><a href="population-genetic-structure.html#monmoniers-algorithm-and-phylogeographic-breaks"><i class="fa fa-check"></i><b>7.6</b> Monmonier's algorithm and phylogeographic breaks</a></li>
<li class="chapter" data-level="7.7" data-path="population-genetic-structure.html"><a href="population-genetic-structure.html#examples-phylogeographic-breaks-in-flying-vertebrates"><i class="fa fa-check"></i><b>7.7</b> examples, phylogeographic breaks in flying vertebrates</a></li>
<li class="chapter" data-level="7.8" data-path="population-genetic-structure.html"><a href="population-genetic-structure.html#practice-2"><i class="fa fa-check"></i><b>7.8</b> practice</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="historical-demography.html"><a href="historical-demography.html"><i class="fa fa-check"></i><b>8</b> Historical demography</a><ul>
<li class="chapter" data-level="8.1" data-path="historical-demography.html"><a href="historical-demography.html#demographic-bottlenecks"><i class="fa fa-check"></i><b>8.1</b> demographic bottlenecks</a></li>
<li class="chapter" data-level="8.2" data-path="historical-demography.html"><a href="historical-demography.html#simple-tests-for-genetic-bottlenecks"><i class="fa fa-check"></i><b>8.2</b> simple tests for genetic bottlenecks</a></li>
<li class="chapter" data-level="8.3" data-path="historical-demography.html"><a href="historical-demography.html#demographic-expansion"><i class="fa fa-check"></i><b>8.3</b> demographic expansion</a></li>
<li class="chapter" data-level="8.4" data-path="historical-demography.html"><a href="historical-demography.html#simple-tests-for-demographic-expansion-tajimas-d"><i class="fa fa-check"></i><b>8.4</b> simple tests for demographic expansion (Tajima's D)</a></li>
<li class="chapter" data-level="8.5" data-path="historical-demography.html"><a href="historical-demography.html#simulation-testing-using-abc"><i class="fa fa-check"></i><b>8.5</b> simulation testing using ABC</a></li>
<li class="chapter" data-level="8.6" data-path="historical-demography.html"><a href="historical-demography.html#example-from-pnw-salamanders"><i class="fa fa-check"></i><b>8.6</b> example from PNW salamanders</a></li>
<li class="chapter" data-level="8.7" data-path="historical-demography.html"><a href="historical-demography.html#practice-3"><i class="fa fa-check"></i><b>8.7</b> practice</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="phylogeny-inference.html"><a href="phylogeny-inference.html"><i class="fa fa-check"></i><b>9</b> Phylogeny inference</a><ul>
<li class="chapter" data-level="9.1" data-path="phylogeny-inference.html"><a href="phylogeny-inference.html#what-are-phylogenies"><i class="fa fa-check"></i><b>9.1</b> what are phylogenies?</a></li>
<li class="chapter" data-level="9.2" data-path="phylogeny-inference.html"><a href="phylogeny-inference.html#gene-trees-and-species-trees"><i class="fa fa-check"></i><b>9.2</b> gene trees and species trees</a></li>
<li class="chapter" data-level="9.3" data-path="phylogeny-inference.html"><a href="phylogeny-inference.html#inference-with-distance-methods"><i class="fa fa-check"></i><b>9.3</b> inference with distance methods</a></li>
<li class="chapter" data-level="9.4" data-path="phylogeny-inference.html"><a href="phylogeny-inference.html#modeling-dna-variation"><i class="fa fa-check"></i><b>9.4</b> modeling DNA variation</a></li>
<li class="chapter" data-level="9.5" data-path="phylogeny-inference.html"><a href="phylogeny-inference.html#inference-with-statistical-methods"><i class="fa fa-check"></i><b>9.5</b> inference with statistical methods</a></li>
<li class="chapter" data-level="9.6" data-path="phylogeny-inference.html"><a href="phylogeny-inference.html#example-from-tamias-chipmunks"><i class="fa fa-check"></i><b>9.6</b> example from Tamias chipmunks</a></li>
<li class="chapter" data-level="9.7" data-path="phylogeny-inference.html"><a href="phylogeny-inference.html#practice-4"><i class="fa fa-check"></i><b>9.7</b> practice</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="species-limits.html"><a href="species-limits.html"><i class="fa fa-check"></i><b>10</b> Species limits</a><ul>
<li class="chapter" data-level="10.1" data-path="species-limits.html"><a href="species-limits.html#cryptic-genetic-diversity"><i class="fa fa-check"></i><b>10.1</b> cryptic genetic diversity</a></li>
<li class="chapter" data-level="10.2" data-path="species-limits.html"><a href="species-limits.html#clustering-methods-revisited"><i class="fa fa-check"></i><b>10.2</b> clustering methods revisited</a></li>
<li class="chapter" data-level="10.3" data-path="species-limits.html"><a href="species-limits.html#abgd"><i class="fa fa-check"></i><b>10.3</b> ABGD</a></li>
<li class="chapter" data-level="10.4" data-path="species-limits.html"><a href="species-limits.html#gmyc"><i class="fa fa-check"></i><b>10.4</b> GMYC</a></li>
<li class="chapter" data-level="10.5" data-path="species-limits.html"><a href="species-limits.html#example-cryptic-diversity-in-sorex"><i class="fa fa-check"></i><b>10.5</b> example, cryptic diversity in Sorex</a></li>
<li class="chapter" data-level="10.6" data-path="species-limits.html"><a href="species-limits.html#practice-5"><i class="fa fa-check"></i><b>10.6</b> practice</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="species-distribution-modeling.html"><a href="species-distribution-modeling.html"><i class="fa fa-check"></i><b>11</b> Species distribution modeling</a><ul>
<li class="chapter" data-level="11.1" data-path="species-distribution-modeling.html"><a href="species-distribution-modeling.html#fundamental-vs.-realized-niches"><i class="fa fa-check"></i><b>11.1</b> fundamental vs. realized niches</a></li>
<li class="chapter" data-level="11.2" data-path="species-distribution-modeling.html"><a href="species-distribution-modeling.html#climatic-variables"><i class="fa fa-check"></i><b>11.2</b> climatic variables</a></li>
<li class="chapter" data-level="11.3" data-path="species-distribution-modeling.html"><a href="species-distribution-modeling.html#estimating-sdms-methods"><i class="fa fa-check"></i><b>11.3</b> estimating SDMs, methods</a></li>
<li class="chapter" data-level="11.4" data-path="species-distribution-modeling.html"><a href="species-distribution-modeling.html#estimating-sdms-ensemble-approaches"><i class="fa fa-check"></i><b>11.4</b> estimating SDMs, ensemble approaches</a></li>
<li class="chapter" data-level="11.5" data-path="species-distribution-modeling.html"><a href="species-distribution-modeling.html#hindcasting-sdms"><i class="fa fa-check"></i><b>11.5</b> hindcasting SDMs</a></li>
<li class="chapter" data-level="11.6" data-path="species-distribution-modeling.html"><a href="species-distribution-modeling.html#forecasting-sdms"><i class="fa fa-check"></i><b>11.6</b> forecasting SDMs</a></li>
<li class="chapter" data-level="11.7" data-path="species-distribution-modeling.html"><a href="species-distribution-modeling.html#example-sdms-for-pwn-slugs"><i class="fa fa-check"></i><b>11.7</b> example, SDMs for PWN slugs</a></li>
<li class="chapter" data-level="11.8" data-path="species-distribution-modeling.html"><a href="species-distribution-modeling.html#practice-6"><i class="fa fa-check"></i><b>11.8</b> practice</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html"><i class="fa fa-check"></i><b>12</b> Predictive Phylogeography</a><ul>
<li class="chapter" data-level="12.1" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#trait-data-response-variables"><i class="fa fa-check"></i><b>12.1</b> trait data &amp; response variables</a></li>
<li class="chapter" data-level="12.2" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#regression-based-approaches"><i class="fa fa-check"></i><b>12.2</b> regression-based approaches</a></li>
<li class="chapter" data-level="12.3" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#random-forest-and-machine-learning"><i class="fa fa-check"></i><b>12.3</b> random forest and machine learning</a></li>
<li class="chapter" data-level="12.4" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#example-predicting-cryptic-diversity-espindola-et-al.-2016"><i class="fa fa-check"></i><b>12.4</b> example, predicting cryptic diversity (Espindola et al. 2016)</a></li>
<li class="chapter" data-level="12.5" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#example-predicting-genetic-isolation-by-distance-pelletier-2018"><i class="fa fa-check"></i><b>12.5</b> example, predicting genetic isolation by distance (Pelletier 2018)</a></li>
<li class="chapter" data-level="12.6" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#practice-7"><i class="fa fa-check"></i><b>12.6</b> practice</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="conservation-genetics.html"><a href="conservation-genetics.html"><i class="fa fa-check"></i><b>13</b> Conservation genetics</a><ul>
<li class="chapter" data-level="13.1" data-path="conservation-genetics.html"><a href="conservation-genetics.html#subtopics"><i class="fa fa-check"></i><b>13.1</b> subtopics</a></li>
<li class="chapter" data-level="13.2" data-path="conservation-genetics.html"><a href="conservation-genetics.html#practice-8"><i class="fa fa-check"></i><b>13.2</b> Practice</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"><img src="Segoe1.png" title="fig:" alt="PhylogatR" style="width:75.0%" /></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="conservation-genetics" class="section level1">
<h1><span class="header-section-number">13</span> Conservation genetics</h1>
<div id="subtopics" class="section level2">
<h2><span class="header-section-number">13.1</span> subtopics</h2>
</div>
<div id="practice-8" class="section level2">
<h2><span class="header-section-number">13.2</span> Practice</h2>
<div id="purpose-1" class="section level4">
<h4><span class="header-section-number">13.2.0.1</span> Purpose</h4>
<p><strong>Genetic diversity</strong> refers to variation in the base pairs of DNA. This genetic diveristy is resposible for the diversity we observe within species (e.g., thermal tolerance, body size). This diversity is essential if species are going to survive environmental change and we can sequence DNA from individuals within a species to obtain measures of genetic diversity.The <a href="https://www.cbd.int/" target="_blank" rel="noopener">Convention on Biological Diversity (CBD)</a> included genetic diversity among its 2010 biodiversity targets.</p>
<p>The <a href="https://www.iucnredlist.org/" target="_blank" rel="noopener">IUCN red list</a> is an international effort to list species as either least concern (LC), near threatened (NT), vulnerable (VU), endangered (EN), or critically endangered (CR) in order to adequately protect species that are in need.</p>
<p>This exercise will use data from an open-source genetic database to answer the question: Is the IUCN red list designed to help protect species with low genetic diveristy that would put them at a higher risk of extinction?</p>
<div id="learning-objectives-1" class="section level5">
<h5><span class="header-section-number">13.2.0.1.1</span> Learning objectives:</h5>
<ol style="list-style-type: decimal">
<li>Explain why genetic diversity is an important concept in conservation biology.</li>
<li>Provide examples as to why open-source databases are valuable tools in conservation biology.</li>
<li>Conduct data analysis using the R statistical computing language.</li>
<li>Interpret plots in order to answer a research question.</li>
</ol>
</div>
<div id="packages-needed-for-this-module-1" class="section level5">
<h5><span class="header-section-number">13.2.0.1.2</span> Packages needed for this module</h5>
<ul>
<li>ggplot2</li>
<li>tools</li>
<li>ade</li>
<li>adegenet</li>
<li>ape</li>
<li>pegas</li>
<li>stringr</li>
<li>tidyr</li>
</ul>
<p>Load packages.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">library</span>(ggplot2)
<span class="kw">library</span>(tools)
<span class="kw">library</span>(adegenet)
<span class="kw">library</span>(pegas)
<span class="kw">library</span>(stringr)
<span class="kw">library</span>(tidyr)</code></pre></div>
</div>
</div>
<div id="download-genetic-data" class="section level4">
<h4><span class="header-section-number">13.2.0.2</span> Download genetic data</h4>
<p>Choose a taxonomic group to study for the above question. Navigate to the <a href="https://phylogatr.org/" target="_blank" rel="noopener">PhylogatR</a> website to download data.</p>
</div>
<div id="download-iucn-data" class="section level4">
<h4><span class="header-section-number">13.2.0.3</span> Download IUCN data</h4>
<p>Navigate to the <a href="https://www.iucnredlist.org/" target="_blank" rel="noopener">IUCN red list</a> website. You will need to create and account to download data.</p>
<ul>
<li>Click on <code>Advanced</code></li>
<li>On the right tool bar, click on <code>Taxonomy</code></li>
<li>Click the appropriate check boxes/arrows to narrow the selection down to your group.</li>
<li>Download the <code>Search Results</code></li>
</ul>
<p>Your download will be a zip file called <code>redlist_species_data_</code> followed by a long list of numbers and letters. If you are using a Mac the zip file will unzip when you double click on it. If you are using Windows, you must extract the files first.</p>
</div>
<div id="import-iucn-data" class="section level4">
<h4><span class="header-section-number">13.2.0.4</span> Import IUCN data</h4>
<p>This imports the csv file and creates and R object called assessments.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">assessments &lt;-<span class="st"> </span><span class="kw">read.csv</span>(<span class="st">&quot;assessments.csv&quot;</span>)</code></pre></div>
<p>In the upper right block of RStudio you will see an <code>Environment</code> tab. Click on it if it is not already chosen. Under <code>Global Environment</code> you will see the objects from your R session. You can click on the <code>assessments</code> object to view it.</p>
<p>You can use R to list the column headers.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">names</span>(assessments)</code></pre></div>
<pre><code>##  [1] &quot;assessmentId&quot;             &quot;internalTaxonId&quot;         
##  [3] &quot;scientificName&quot;           &quot;redlistCategory&quot;         
##  [5] &quot;redlistCriteria&quot;          &quot;yearPublished&quot;           
##  [7] &quot;assessmentDate&quot;           &quot;criteriaVersion&quot;         
##  [9] &quot;language&quot;                 &quot;rationale&quot;               
## [11] &quot;habitat&quot;                  &quot;threats&quot;                 
## [13] &quot;population&quot;               &quot;populationTrend&quot;         
## [15] &quot;range&quot;                    &quot;useTrade&quot;                
## [17] &quot;systems&quot;                  &quot;conservationActions&quot;     
## [19] &quot;realm&quot;                    &quot;yearLastSeen&quot;            
## [21] &quot;possiblyExtinct&quot;          &quot;possiblyExtinctInTheWild&quot;
## [23] &quot;scopes&quot;</code></pre>
<p>We are the most interested in columns 3 and 4, so lets clean it up a little so it is easier to use later.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">assessments &lt;-<span class="st"> </span>assessments[,<span class="dv">3</span><span class="op">:</span><span class="dv">4</span>]</code></pre></div>
<p>Create a plot to look at the distribution of red listings in this group.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">r &lt;-<span class="st"> </span><span class="kw">ggplot</span>(assessments, <span class="kw">aes</span>(<span class="dt">x=</span>redlistCategory, <span class="dt">fill=</span>redlistCategory)) <span class="op">+</span><span class="st"> </span><span class="kw">geom_bar</span>() <span class="op">+</span><span class="st"> </span><span class="kw">scale_fill_brewer</span>(<span class="dt">palette =</span> <span class="st">&quot;Paired&quot;</span>) <span class="op">+</span><span class="st"> </span><span class="kw">theme</span>(<span class="dt">axis.title.x=</span><span class="kw">element_blank</span>(), <span class="dt">legend.position =</span> <span class="st">&quot;none&quot;</span>) <span class="op">+</span><span class="st"> </span><span class="kw">theme</span>(<span class="dt">axis.text.x =</span> <span class="kw">element_text</span>(<span class="dt">angle =</span> <span class="dv">25</span>))

r</code></pre></div>
<p><img src="_main_files/figure-html/unnamed-chunk-13-1.png" width="672" /></p>
<p>We can also view these in a table.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">table</span>(assessments<span class="op">$</span>redlistCategory)</code></pre></div>
<pre><code>## 
##             Critically Endangered                    Data Deficient 
##                               682                              2229 
##                        Endangered                           Extinct 
##                               586                               299 
##               Extinct in the Wild                     Least Concern 
##                                14                              3291 
## Lower Risk/conservation dependent          Lower Risk/least concern 
##                                 4                                17 
##        Lower Risk/near threatened                   Near Threatened 
##                               118                               575 
##                        Vulnerable 
##                              1032</code></pre>
<p><strong>What do all the categories mean?</strong> Hint: Your dataset might not contain all these categories. Some of them are older categories that need to be updated in the database.</p>
<p><strong>Take a look at each category for your group. What is the most frequent category? Compare with your classmates.</strong></p>
</div>
<div id="import-and-summarize-sequence-data" class="section level4">
<h4><span class="header-section-number">13.2.0.5</span> Import and summarize sequence data</h4>
<p>To get the information we need from all the sequence alignments from our PhylogatR download, we'll first create a list of all the sequence alignments and put them in an object called <code>myfiles</code>.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">myfiles &lt;-<span class="st"> </span><span class="kw">list.files</span>(<span class="dt">pattern =</span> <span class="st">&quot;.afa&quot;</span>, <span class="dt">full.names=</span><span class="ot">TRUE</span>, <span class="dt">recursive =</span> <span class="ot">TRUE</span>)</code></pre></div>
<p>Here we loop through the list stored in <code>myfiles</code> to calculate genetic diversity for each sequence alignment. All data will be written to a csv file called <code>phylogatr-data.csv</code> and stored in the current working directory. This step will take a few minutes to run, and will take longer if you have a lot of species.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="cf">for</span> (f <span class="cf">in</span> myfiles) {

  <span class="co">#get species and gene name</span>
  sp &lt;-<span class="st"> </span><span class="kw">basename</span>(f)
  sp &lt;-<span class="st"> </span><span class="kw">file_path_sans_ext</span>(sp)
  
  <span class="co">#read in fasta file</span>
  seq &lt;-<span class="st"> </span><span class="kw">fasta2DNAbin</span>(f, <span class="dt">quiet=</span><span class="ot">TRUE</span>)
  
  <span class="co">#get number of sequences</span>
  n &lt;-<span class="st"> </span><span class="kw">nrow</span>(seq)

  <span class="co">#calculate nucleotide diversity</span>
  pi &lt;-<span class="st"> </span><span class="kw">nuc.div</span>(seq)
  
  <span class="co">#write data to file</span>
  <span class="kw">write.table</span>(<span class="kw">data.frame</span>(sp, pi, n), <span class="dt">file=</span><span class="st">&quot;phylogatr-data.csv&quot;</span>, <span class="dt">quote=</span><span class="ot">FALSE</span>, <span class="dt">row.names=</span><span class="ot">FALSE</span>, <span class="dt">col.names=</span><span class="op">!</span><span class="kw">file.exists</span>(<span class="st">&quot;phylogatr-data.csv&quot;</span>), <span class="dt">append=</span><span class="ot">TRUE</span>, <span class="dt">sep=</span><span class="st">&quot;,&quot;</span>)
}</code></pre></div>
</div>
<div id="merge-iucn-and-genetic-data-results" class="section level4">
<h4><span class="header-section-number">13.2.0.6</span> Merge IUCN and genetic data results</h4>
<p>First, import the output from the previous step, then look at the column headers.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">genetics &lt;-<span class="st"> </span><span class="kw">read.csv</span>(<span class="st">&quot;phylogatr-data.csv&quot;</span>)
<span class="kw">names</span>(genetics)</code></pre></div>
<pre><code>## [1] &quot;sp&quot; &quot;pi&quot; &quot;n&quot;</code></pre>
<p>sp = species</p>
<p>pi = nuceotide diversity</p>
<p>n = the number of sequences</p>
<p>Now, we can merge the data from both databases. We have two objects in our R environment. 1) <code>assessments</code> has the IUCN data and 2) <code>genetics</code> has the calculated genetic diversity.</p>
<p>We'll need to reformat the species names from PhylogatR so they can match those found on the IUCN database. The species names will be merged and if there is a species that has data from both databases, it will be saved in a new R object.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="co">#reformat PhylogatR species names</span>
genetics<span class="op">$</span>sp &lt;-<span class="st"> </span><span class="kw">sub</span>(<span class="st">&quot;-&quot;</span>, <span class="st">&quot; &quot;</span> ,genetics<span class="op">$</span>sp)
genetics &lt;-<span class="st"> </span><span class="kw">separate</span>(genetics, sp, <span class="dt">into=</span><span class="kw">c</span>(<span class="st">&quot;scientificName&quot;</span>, <span class="st">&quot;gene&quot;</span>), <span class="dt">sep=</span><span class="st">&quot;-&quot;</span>)

<span class="co">#merge</span>
dataset &lt;-<span class="st"> </span><span class="kw">merge</span>(assessments, genetics, <span class="dt">by =</span> <span class="st">&quot;scientificName&quot;</span>)
<span class="kw">names</span>(dataset)</code></pre></div>
<pre><code>## [1] &quot;scientificName&quot;  &quot;redlistCategory&quot; &quot;gene&quot;            &quot;pi&quot;             
## [5] &quot;n&quot;</code></pre>
<p>Remember, you can click on objects in your R Environment to view them.</p>
<p>This is what the first six lines of the mollusc data look like.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="kw">head</span>(dataset)</code></pre></div>
<pre><code>##         scientificName redlistCategory gene          pi  n
## 1 Alasmidonta varicosa      Vulnerable  COI 0.000000000  3
## 2    Alloteuthis media  Data Deficient  COI 0.004959925 21
## 3 Alloteuthis subulata  Data Deficient  COI 0.003960396  3
## 4        Alopia livida   Least Concern  COI 0.014712006 11
## 5       Alopia monacha   Least Concern  COI 0.020174482  8
## 6      Amblema plicata   Least Concern  COI 0.006259781  3</code></pre>
<p><strong>How many species had both genetic and IUCN red list data from your group? Compare with your classmates.</strong></p>
</div>
<div id="data-analysis" class="section level4">
<h4><span class="header-section-number">13.2.0.7</span> Data analysis</h4>
<p>First, we'll make a plot to visualize our data. This is a plot of mean nucleotide diversity found within each red list category.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r"><span class="co">#calculate mean nucleotide diversity for each group</span>
mean_pi &lt;-<span class="st"> </span><span class="kw">aggregate</span>(dataset<span class="op">$</span>pi, <span class="kw">list</span>(dataset<span class="op">$</span>redlistCategory), mean, <span class="dt">na.rm=</span>T)

r_pi &lt;-<span class="st"> </span><span class="kw">ggplot</span>(mean_pi, <span class="kw">aes</span>(<span class="dt">x =</span> Group.<span class="dv">1</span>, <span class="dt">y =</span> x, <span class="dt">fill =</span> Group.<span class="dv">1</span>)) <span class="op">+</span><span class="st"> </span><span class="kw">geom_bar</span>(<span class="dt">stat =</span> <span class="st">&quot;identity&quot;</span>) <span class="op">+</span><span class="st"> </span><span class="kw">scale_fill_brewer</span>(<span class="dt">palette =</span> <span class="st">&quot;Paired&quot;</span>) <span class="op">+</span><span class="st"> </span><span class="kw">theme</span>(<span class="dt">axis.title.x=</span><span class="kw">element_blank</span>(), <span class="dt">legend.position =</span> <span class="st">&quot;none&quot;</span>) <span class="op">+</span><span class="st"> </span><span class="kw">theme</span>(<span class="dt">axis.text.x =</span> <span class="kw">element_text</span>(<span class="dt">angle =</span> <span class="dv">25</span>)) <span class="op">+</span><span class="st"> </span><span class="kw">ylab</span>(<span class="st">&quot;nucleotide diversity (pi)&quot;</span>)

r_pi</code></pre></div>
<p><img src="_main_files/figure-html/unnamed-chunk-20-1.png" width="672" /></p>
<p>Now we can test whether there is a significant difference in nucleotide diversity between those species that are threatened and those that are not. A t-test compares two groups so first we'll group categories into threatened and not-threatened and call them as such. We'll remove the data deficient species (notice we are overwriting the <code>dataset</code> object and it will no longer contain the data deficient assessments).</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">dataset&lt;-dataset[<span class="op">!</span>(dataset<span class="op">$</span>redlistCategory<span class="op">==</span><span class="st">&quot;Data Deficient&quot;</span>),]

notthreatened&lt;-<span class="kw">c</span>(<span class="st">&#39;Near Threatened&#39;</span>, <span class="st">&#39;Least Concern&#39;</span>, <span class="st">&#39;Lower Risk/least concern&#39;</span>, <span class="st">&#39;Lower Risk/near threatened&#39;</span>)
<span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="op">:</span><span class="kw">nrow</span>(dataset)) {
  <span class="cf">if</span> (dataset<span class="op">$</span>redlistCategory[i] <span class="op">%in%</span><span class="st"> </span>notthreatened) {
    dataset<span class="op">$</span>category[i] &lt;-<span class="st">&#39;notthreatened&#39;</span> 
  }
  <span class="cf">else</span> {
    dataset<span class="op">$</span>category[i] &lt;-<span class="st">&#39;threatened&#39;</span>
   }
}</code></pre></div>
<p>Compare the two groups.</p>
<div class="sourceCode"><pre class="sourceCode r"><code class="sourceCode r">threatened &lt;-<span class="st"> </span>dataset<span class="op">$</span>pi[dataset<span class="op">$</span>category<span class="op">==</span><span class="st">&quot;threatened&quot;</span>]
notthreatened &lt;-<span class="st"> </span>dataset<span class="op">$</span>pi[dataset<span class="op">$</span>category<span class="op">==</span><span class="st">&quot;notthreatened&quot;</span>]

<span class="kw">t.test</span>(threatened, notthreatened)</code></pre></div>
<pre><code>## 
##  Welch Two Sample t-test
## 
## data:  threatened and notthreatened
## t = -2.2634, df = 50.678, p-value = 0.02793
## alternative hypothesis: true difference in means is not equal to 0
## 95 percent confidence interval:
##  -0.026176358 -0.001565606
## sample estimates:
##  mean of x  mean of y 
## 0.01434120 0.02821218</code></pre>
<p>In this case there is a significant difference in genetic diversity among those species that are listed as threatened vs. not threatened.</p>

</div>
</div>
</div>







            </section>

          </div>
        </div>
      </div>
<a href="predictive-phylogeography.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
