<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>3 DNA | PhylogatR</title>
  <meta name="description" content="" />
  <meta name="generator" content="bookdown 0.26.3 and GitBook 2.6.7" />

  <meta property="og:title" content="3 DNA | PhylogatR" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="3 DNA | PhylogatR" />
  
  
  

<meta name="author" content="Bryan Carstens, Sydney Decker, Anahi Espindola, Sarah Foltz, Abbi Green, Danielle Parsons, Tara Pelletier, Jordan Satler" />


<meta name="date" content="2022-08-01" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="introduction.html"/>
<link rel="next" href="biodiversity-databases.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Background</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#how-to-use-this-book"><i class="fa fa-check"></i><b>1.1</b> How to use this book</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#prerequisites-if-using-r"><i class="fa fa-check"></i><b>1.2</b> Prerequisites if using R</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="index.html"><a href="index.html#r-and-rstudio"><i class="fa fa-check"></i><b>1.2.1</b> R and RStudio</a></li>
<li class="chapter" data-level="1.2.2" data-path="index.html"><a href="index.html#r-packages"><i class="fa fa-check"></i><b>1.2.2</b> R packages</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>2</b> Introduction</a>
<ul>
<li class="chapter" data-level="2.1" data-path="introduction.html"><a href="introduction.html#data-in-the-biological-sciences"><i class="fa fa-check"></i><b>2.1</b> Data in the biological sciences</a></li>
<li class="chapter" data-level="2.2" data-path="introduction.html"><a href="introduction.html#why-species"><i class="fa fa-check"></i><b>2.2</b> Why species?</a></li>
<li class="chapter" data-level="2.3" data-path="introduction.html"><a href="introduction.html#the-scientific-method"><i class="fa fa-check"></i><b>2.3</b> The Scientific Method</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="dna.html"><a href="dna.html"><i class="fa fa-check"></i><b>3</b> DNA</a>
<ul>
<li class="chapter" data-level="3.1" data-path="dna.html"><a href="dna.html#the-genetic-code"><i class="fa fa-check"></i><b>3.1</b> The genetic code</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="dna.html"><a href="dna.html#practice-translate-the-following-dna-sequence-into-amino-acids"><i class="fa fa-check"></i><b>3.1.1</b> Practice: Translate the following DNA sequence into amino acids</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="dna.html"><a href="dna.html#mutation"><i class="fa fa-check"></i><b>3.2</b> Mutation</a></li>
<li class="chapter" data-level="3.3" data-path="dna.html"><a href="dna.html#synonymous-versus-non-synonymous-genetic-variation"><i class="fa fa-check"></i><b>3.3</b> Synonymous versus non-synonymous genetic variation</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="dna.html"><a href="dna.html#practice-introduce-a-mutation"><i class="fa fa-check"></i><b>3.3.1</b> Practice: Introduce a mutation</a></li>
<li class="chapter" data-level="3.3.2" data-path="dna.html"><a href="dna.html#practice-introduce-an-insertion"><i class="fa fa-check"></i><b>3.3.2</b> Practice: Introduce an insertion</a></li>
<li class="chapter" data-level="3.3.3" data-path="dna.html"><a href="dna.html#practice-calculate-the-proportion-of-single-base-pair-mutations-that-are-synonymous"><i class="fa fa-check"></i><b>3.3.3</b> Practice: Calculate the proportion of single base pair mutations that are synonymous</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="dna.html"><a href="dna.html#probabilities-snps-and-genetic-variation"><i class="fa fa-check"></i><b>3.4</b> Probabilities, SNPs, and genetic variation</a></li>
<li class="chapter" data-level="3.5" data-path="dna.html"><a href="dna.html#modeling-sequence-variation"><i class="fa fa-check"></i><b>3.5</b> Modeling sequence variation</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="dna.html"><a href="dna.html#practice-estimate-base-pair-composition"><i class="fa fa-check"></i><b>3.5.1</b> Practice: Estimate base pair composition</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="dna.html"><a href="dna.html#organellar-genomes-and-barcoding-genes"><i class="fa fa-check"></i><b>3.6</b> Organellar genomes and barcoding genes</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html"><i class="fa fa-check"></i><b>4</b> Biodiversity Databases</a>
<ul>
<li class="chapter" data-level="4.1" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#global-biodiversity-information-facility-gbif"><i class="fa fa-check"></i><b>4.1</b> Global Biodiversity Information Facility (GBIF)</a></li>
<li class="chapter" data-level="4.2" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#genbank"><i class="fa fa-check"></i><b>4.2</b> GenBank</a></li>
<li class="chapter" data-level="4.3" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#bold"><i class="fa fa-check"></i><b>4.3</b> BOLD</a></li>
<li class="chapter" data-level="4.4" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#phylogatr"><i class="fa fa-check"></i><b>4.4</b> <em>PhylogatR</em></a></li>
<li class="chapter" data-level="4.5" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#using-phylogatr"><i class="fa fa-check"></i><b>4.5</b> Using <em>phylogatR</em></a></li>
<li class="chapter" data-level="4.6" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#practice"><i class="fa fa-check"></i><b>4.6</b> Practice</a>
<ul>
<li class="chapter" data-level="4.6.1" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#downloading-data-from-phylogatr"><i class="fa fa-check"></i><b>4.6.1</b> Downloading data from <em>phylogatR</em></a></li>
<li class="chapter" data-level="4.6.2" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#check-for-outliers"><i class="fa fa-check"></i><b>4.6.2</b> Check for outliers</a></li>
<li class="chapter" data-level="4.6.3" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#check-sequence-alignments"><i class="fa fa-check"></i><b>4.6.3</b> Check sequence alignments</a></li>
<li class="chapter" data-level="4.6.4" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#visualizing-geographic-coordinates"><i class="fa fa-check"></i><b>4.6.4</b> Visualizing geographic coordinates</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html"><i class="fa fa-check"></i><b>5</b> Population genetics and conservation</a>
<ul>
<li class="chapter" data-level="5.1" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#loss-of-genetic-diversity"><i class="fa fa-check"></i><b>5.1</b> Loss of genetic diversity</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#small-population-size"><i class="fa fa-check"></i><b>5.1.1</b> Small population size</a></li>
<li class="chapter" data-level="5.1.2" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#genetic-drift"><i class="fa fa-check"></i><b>5.1.2</b> Genetic drift</a></li>
<li class="chapter" data-level="5.1.3" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#gene-flow"><i class="fa fa-check"></i><b>5.1.3</b> Gene flow</a></li>
<li class="chapter" data-level="5.1.4" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#inbreeding"><i class="fa fa-check"></i><b>5.1.4</b> Inbreeding</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#units-of-conservation"><i class="fa fa-check"></i><b>5.2</b> Units of conservation</a></li>
<li class="chapter" data-level="5.3" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#practice-1"><i class="fa fa-check"></i><b>5.3</b> Practice</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#learning-objectives-1"><i class="fa fa-check"></i><b>5.3.1</b> Learning objectives:</a></li>
<li class="chapter" data-level="5.3.2" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#data"><i class="fa fa-check"></i><b>5.3.2</b> Data</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#click-and-play"><i class="fa fa-check"></i><b>5.4</b> Click and play!</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="species-limits.html"><a href="species-limits.html"><i class="fa fa-check"></i><b>6</b> Species limits</a>
<ul>
<li class="chapter" data-level="6.1" data-path="species-limits.html"><a href="species-limits.html#cryptic-genetic-diversity"><i class="fa fa-check"></i><b>6.1</b> Cryptic genetic diversity</a></li>
<li class="chapter" data-level="6.2" data-path="species-limits.html"><a href="species-limits.html#genetic-clustering"><i class="fa fa-check"></i><b>6.2</b> Genetic clustering</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="species-limits.html"><a href="species-limits.html#structure"><i class="fa fa-check"></i><b>6.2.1</b> Structure</a></li>
<li class="chapter" data-level="6.2.2" data-path="species-limits.html"><a href="species-limits.html#structurama"><i class="fa fa-check"></i><b>6.2.2</b> Structurama</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="species-limits.html"><a href="species-limits.html#tree-based-methods"><i class="fa fa-check"></i><b>6.3</b> Tree-based methods</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="species-limits.html"><a href="species-limits.html#general-mixed-yule-coalescent-gmyc"><i class="fa fa-check"></i><b>6.3.1</b> General mixed Yule Coalescent (GMYC)</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="species-limits.html"><a href="species-limits.html#practice-2"><i class="fa fa-check"></i><b>6.4</b> Practice</a>
<ul>
<li class="chapter" data-level="6.4.1" data-path="species-limits.html"><a href="species-limits.html#data-preparation"><i class="fa fa-check"></i><b>6.4.1</b> Data Preparation</a></li>
<li class="chapter" data-level="6.4.2" data-path="species-limits.html"><a href="species-limits.html#generate-a-phylogenetic-tree"><i class="fa fa-check"></i><b>6.4.2</b> Generate a Phylogenetic Tree</a></li>
<li class="chapter" data-level="6.4.3" data-path="species-limits.html"><a href="species-limits.html#create-a-summary-tree"><i class="fa fa-check"></i><b>6.4.3</b> Create a summary tree</a></li>
<li class="chapter" data-level="6.4.4" data-path="species-limits.html"><a href="species-limits.html#species-delimitation"><i class="fa fa-check"></i><b>6.4.4</b> Species Delimitation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html"><i class="fa fa-check"></i><b>7</b> Predictive Phylogeography</a>
<ul>
<li class="chapter" data-level="7.0.1" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#trait-data-and-response-variables"><i class="fa fa-check"></i><b>7.0.1</b> Trait data and response variables</a></li>
<li class="chapter" data-level="7.1" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#machine-learning-and-random-forest"><i class="fa fa-check"></i><b>7.1</b> Machine learning and random forest</a></li>
<li class="chapter" data-level="7.2" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#real-life-example"><i class="fa fa-check"></i><b>7.2</b> Real life example</a></li>
<li class="chapter" data-level="7.3" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#practice-3"><i class="fa fa-check"></i><b>7.3</b> Practice</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>8</b> References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"><img src="Segoe1.png" title="fig:" style="width:75.0%" alt="PhylogatR" /></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="dna" class="section level1 hasAnchor" number="3">
<h1><span class="header-section-number">3</span> DNA<a href="dna.html#dna" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p><strong>Deoxyribonucleic acid (DNA)</strong> is a molecule that consists of two chains that coil around each other to form a structure that looks something like a twisted ladder and is known as a double helix. The sides of this ladder are composed of sugars and phosphates, with the rungs of the ladder composed of nucleotides that occur in one of four forms: <strong>Cytosine</strong> and <strong>Thymine</strong>, are known as Pyrimidines because their molecules are made of one ring; <strong>Guanine</strong> and <strong>Adenine</strong>, are known as Purines because their molecules are made of two rings. These nucleotide bases (Adenine, Cytosine, Guanine, and Thymine) form <strong>hydrogen bonds</strong> at the center of each rung of the DNA ladder to hold the whole structure together. These bonds occur in a <strong><em>complementary</em></strong> manner, with Purines always bonding to Pyrimidines. Because Adenine and Thymine have two available spots for hydrogen bonds these nucleotide bases always bond to each other; similarly, Guanine and Cytosine both have three available spots for hydrogen bonds and thus always bond to each other. This complementary bonding in combination with the double helix structure has several important ramifications: it allows an almost endless variation in the sequence of individual nucleotide bases while simultaneously making the molecule very stable, and it enables the sequence of these bases to be repeated on each side of the ladder. The complementary bonding (i.e., <strong>A—T</strong> and <strong>C—G</strong>) allows the total sequence to be a reverse complement on either side of the ladder. However, the ladder metaphor is not perfect, as DNA shares some important characteristics with a zipper as well because its reverse complementarity allows it to be unzipped for two different purposes: to serve as the instructions for making proteins (see section 3.1) and during the process of cell division. During cell division, the set of DNA in the nucleus of the parent cell is replicated in order to provide an exact copy in each of the daughter cells via the process of mitosis. Some cells, which are known as gametes, undergo a special process of incomplete replication referred to as meiosis in species that reproduce sexually. This allows the gametes from the mother and father to be combined in their offspring.</p>
<p><img src="genetics-gd76030a9c_1280.png" style="width:75.0%" /></p>
   <font size="2"><strong> Figure 3A: </strong>Structure of DNA. Image from pixaby.com.</font></p>
<p>In practice, when geneticists sample DNA from a natural population they mostly do not have information about the relationships among individuals that are sampled. They are, however, able to take tissue samples, extract DNA for <strong>sequencing</strong> or <strong>genotyping</strong>, and derive information from these samples. On a basic level, every sample contains DNA sequences of some type, and information such as the length of the sequence, the DNA base composition (i.e., the proportion of A’s and C’s and G’s and T’s) and the variation within this sample are useful. In order to learn how this information is used, a brief review of the ultimate source of genetic variation is required.</p>
<div id="the-genetic-code" class="section level2 hasAnchor" number="3.1">
<h2><span class="header-section-number">3.1</span> The genetic code<a href="dna.html#the-genetic-code" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>DNA provides the instructions for making the proteins that make up the cells of all organisms and the bodies of plants, animals, and fungi. This process occurs via a process where DNA is <strong>translated</strong> into RNA (with Uracil substituting for Thymine) from which chains of <strong>amino acid</strong> are <strong>transcribed</strong> in the cell into the three-dimensional structures of proteins. Ultimately, the instructions for making a given protein flows from triplets of nucleotide bases called <strong>codons</strong>, where three bases describe the recipe for one of 20 amino acids, for the entire length of the gene. Since there are four different nucleotide bases assembled into sets of three bases that code for a particular amino acid, there are more combinations of three bases (4<sup>3</sup> = 64, to be exact) than amino acids (20). For this reason, the mapping of codons to amino acids has inherent redundancy, where different triplets of nucleotides can code for the same amino acid. When different codon triplets can code for the same amino acid, the first and second nucleotides in the codon will always correspond to a particular amino acid but the third position may be redundant. In addition, there are special codons such as ATG and AGA that start and stop protein synthesis. This correspondence between nucleotide codons and the amino acids that are ultimately produced is known as the <strong>genetic code</strong>.</p>
<p><img src="dna-152135_1280.png" style="width:75.0%" /></p>
   <font size="2"><strong> Figure 3B: </strong>The genetic code describes the correspondence between nucleotide codons and the amino acids which are ultimately generated from their sequences. Image from pixaby.com.</font></p>
<div id="practice-translate-the-following-dna-sequence-into-amino-acids" class="section level3 hasAnchor" number="3.1.1">
<h3><span class="header-section-number">3.1.1</span> Practice: Translate the following DNA sequence into amino acids<a href="dna.html#practice-translate-the-following-dna-sequence-into-amino-acids" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>First, load the packages required for the following exercises. You might get some warning messages, but these are usually ok, so we’ll ignore them for now. Make sure all the packages are installed first, and if R asks if you’d like to update other packages, it’s usually ok to click yes.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="dna.html#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(seqinr)</span>
<span id="cb1-2"><a href="dna.html#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ape)</span>
<span id="cb1-3"><a href="dna.html#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(apex)</span></code></pre></div>
<p>In the code below, we will first generate a DNA sequence in R using the <code>as.vector</code> function. Then, we will translate the DNA sequence into the corresponding sequence of amino acids. You can change the nucleotide bases (i.e., <code>a</code>, <code>t</code>, <code>c</code>, and <code>g</code>) in the first line of code below to see what (if any) changes occur to the amino acid sequence.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="dna.html#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="co">#create DNA sequence by creating an object in R called &#39;dna&#39;</span></span>
<span id="cb2-2"><a href="dna.html#cb2-2" aria-hidden="true" tabindex="-1"></a>dna <span class="ot">&lt;-</span> <span class="fu">as.vector</span>(<span class="fu">c</span>(<span class="st">&quot;c&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;a&quot;</span>))</span>
<span id="cb2-3"><a href="dna.html#cb2-3" aria-hidden="true" tabindex="-1"></a><span class="co">#print DNA sequence to screen</span></span>
<span id="cb2-4"><a href="dna.html#cb2-4" aria-hidden="true" tabindex="-1"></a>dna</span></code></pre></div>
<pre><code>##  [1] &quot;c&quot; &quot;c&quot; &quot;g&quot; &quot;c&quot; &quot;a&quot; &quot;t&quot; &quot;t&quot; &quot;a&quot; &quot;c&quot; &quot;c&quot; &quot;t&quot; &quot;a&quot; &quot;c&quot; &quot;a&quot; &quot;g&quot; &quot;g&quot; &quot;g&quot; &quot;t&quot; &quot;g&quot;
## [20] &quot;c&quot; &quot;a&quot; &quot;a&quot; &quot;c&quot; &quot;t&quot; &quot;c&quot; &quot;g&quot; &quot;a&quot;</code></pre>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="dna.html#cb4-1" aria-hidden="true" tabindex="-1"></a><span class="co">#translate DNA sequence using the R function &#39;translate&#39;</span></span>
<span id="cb4-2"><a href="dna.html#cb4-2" aria-hidden="true" tabindex="-1"></a>aminoseq1 <span class="ot">&lt;-</span> <span class="fu">noquote</span>(<span class="fu">translate</span>(dna))</span>
<span id="cb4-3"><a href="dna.html#cb4-3" aria-hidden="true" tabindex="-1"></a><span class="co">#print amino acid sequence to screen</span></span>
<span id="cb4-4"><a href="dna.html#cb4-4" aria-hidden="true" tabindex="-1"></a>aminoseq1</span></code></pre></div>
<pre><code>## [1] P H Y L Q G A T R</code></pre>
</div>
</div>
<div id="mutation" class="section level2 hasAnchor" number="3.2">
<h2><span class="header-section-number">3.2</span> Mutation<a href="dna.html#mutation" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>While the DNA molecule is very stable, occasionally there are mistakes in the process of replication when DNA is being copied. When these occur during <strong>mitosis</strong> in <strong>somatic cells</strong>, the result can sometimes lead to cancerous growth, but most often do not cause any big changes. Mistakes that occur during <strong>meiosis</strong> in the <strong>gametic cells</strong> are passed on to the next generation and can sometimes impact the offspring. These mistakes, known as <strong>mutations</strong>, are exceedingly rare – one estimate from humans ([Nachman &amp; Crowell 2000](/phylogatr/book/references.html)) suggests that they occur on average at a rate of 2.5 x 10<sup>-9</sup> (0.000000025) mutations per nucleotide per site per <strong>generation</strong>. This means that on average, a particular site in the human genome has a mutation less than once in every 10,000,000 generations. However, mutations occur in a random manner <em>and</em> there are a lot of sites in the human genome – 3,200,000,000 x 2 (because there are two copies) in total. This leads to a bit of a paradox: even though the odds of a mutation occurring at a particular site in the genome for any given individual are extremely low, there are so many sites that all individuals have a small number of mutations in comparison to their parents. For example, a typical human baby has a couple of dozen sites that differ from the corresponding site in their parent due to a mutation that occurred in that parent’s gamete. Most of the time these mutations are benign because they either occur in parts of the genome that are not associated with function (producing proteins or regulating gene expression), or because if they occur in a gene that makes a protein, they occur in a redundant nucleotide base that does not change the amino acid produced. Rarely, these mutations do occur in a protein coding gene and do change the amino acid that is produced by the protein and natural selection may act to change its <strong>frequency</strong>.</p>
</div>
<div id="synonymous-versus-non-synonymous-genetic-variation" class="section level2 hasAnchor" number="3.3">
<h2><span class="header-section-number">3.3</span> Synonymous versus non-synonymous genetic variation<a href="dna.html#synonymous-versus-non-synonymous-genetic-variation" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Mutations that introduce new <strong>alleles</strong> into a population are the raw material from which natural selection acts upon to produce evolutionary change. For most populations of a given species, the process of mutation acting over time has produced more than one sequence of DNA nucleotide bases at a given genetic <strong>locus</strong>, a location in the genome that consists of some number of nucleotides. These variants are referred to as alleles; each allele is a slightly different version of a given gene. Many times, the variants are not functionally different because the mutations occur in redundant positions in the codon. Such variants are referred to as <strong>synonymous</strong> variants because the amino acids that are produced are ultimately the <em>same</em>. Conversely, if a mutation occurs in a nucleotide base that does lead to a new amino acid, such changes are known as <strong>non-synonymous</strong>. In these cases, the resulting protein will be <em>different</em>. The likely result of non-synonymous change is a version of the protein that is no better, and usually worse, than the version that exists in the population, although rarely the random process of mutation will lead to an improved version of the protein. An individual that lucks into an improved version of a protein may ultimately be more fit and have more offspring than it otherwise would have had because it confers some advantages to the individual. For example, the improved version of the protein could enable that individual to extract more energy from a food resource. Over many generations, the increase in reproductive success for individuals that have the improved version of the protein would lead to an increase in the frequency of the allele that produced this version of the protein. Over time, evolutionary change will occur.</p>
<div id="practice-introduce-a-mutation" class="section level3 hasAnchor" number="3.3.1">
<h3><span class="header-section-number">3.3.1</span> Practice: Introduce a mutation<a href="dna.html#practice-introduce-a-mutation" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>By changing the nucleotide bases we are introducing a mutation. In the code below, we have switched the 6th nucleotide base from a <code>t</code> to a <code>c</code>. Run the following code to determine how this single nucelotide mutation affects the amino acid sequence.</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="dna.html#cb6-1" aria-hidden="true" tabindex="-1"></a>dna <span class="ot">&lt;-</span> <span class="fu">as.vector</span>(<span class="fu">c</span>(<span class="st">&quot;c&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;a&quot;</span>))</span>
<span id="cb6-2"><a href="dna.html#cb6-2" aria-hidden="true" tabindex="-1"></a><span class="fu">noquote</span>(<span class="fu">translate</span>(dna))</span></code></pre></div>
<pre><code>## [1] P H Y L Q G A T R</code></pre>
<p>Compare this amino acid to the sequence from the previous example in section 3.1.1. Is this a synonymous or non-synonymous mutation?</p>
<p>Now let’s switch the 17th nucleotide base from a <code>g</code> to a <code>c</code>. Run the following code to determine how this single nuceltoide mutation affects the amino acid sequence.</p>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="dna.html#cb8-1" aria-hidden="true" tabindex="-1"></a>dna <span class="ot">&lt;-</span> <span class="fu">as.vector</span>(<span class="fu">c</span>(<span class="st">&quot;c&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;a&quot;</span>))</span>
<span id="cb8-2"><a href="dna.html#cb8-2" aria-hidden="true" tabindex="-1"></a><span class="fu">noquote</span>(<span class="fu">translate</span>(dna))</span></code></pre></div>
<pre><code>## [1] P H Y L Q D A T R</code></pre>
<p>Compare this amino acid to the sequence to the one above. Is this a synonymous or non-synonymous mutation? How does the new amino acid sequence differ from that resulting from the previous mutation?</p>
</div>
<div id="practice-introduce-an-insertion" class="section level3 hasAnchor" number="3.3.2">
<h3><span class="header-section-number">3.3.2</span> Practice: Introduce an insertion<a href="dna.html#practice-introduce-an-insertion" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Now let’s see what happens when we insert an additional nucleotide base into our DNA sequence. We will insert a <code>t</code> after the fourth base and translate the resulting DNA sequence into the corresponding sequence of amino acids.</p>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="dna.html#cb10-1" aria-hidden="true" tabindex="-1"></a>dna <span class="ot">&lt;-</span> <span class="fu">as.vector</span>(<span class="fu">c</span>(<span class="st">&quot;c&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;a&quot;</span>))</span>
<span id="cb10-2"><a href="dna.html#cb10-2" aria-hidden="true" tabindex="-1"></a><span class="fu">noquote</span>(<span class="fu">translate</span>(dna))</span></code></pre></div>
<pre><code>## [1] P L S I P G C N S</code></pre>
<p>Compare the resulting amino acid sequence to those you generated through the mutations shown above. How do insertions compare to single-point mutations when it comes to the process of translation?</p>
</div>
<div id="practice-calculate-the-proportion-of-single-base-pair-mutations-that-are-synonymous" class="section level3 hasAnchor" number="3.3.3">
<h3><span class="header-section-number">3.3.3</span> Practice: Calculate the proportion of single base pair mutations that are synonymous<a href="dna.html#practice-calculate-the-proportion-of-single-base-pair-mutations-that-are-synonymous" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>This chunk of code runs a <strong>for loop</strong> that randomly samples <code>a</code>, <code>c</code>, <code>t</code>, and <code>g</code> base pairs one million times. A for loop is coding statement that allows you to repeat a process some number of times given a condition. The loop then introduces a random mutation for each replication, and translates the DNA sequence to its amino acid sequence. It then checks if the <em>new</em> amino acid sequence is identical to the original amino acid sequence.</p>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="dna.html#cb12-1" aria-hidden="true" tabindex="-1"></a><span class="co">#create an original DNA and amino acid sequence, same as above</span></span>
<span id="cb12-2"><a href="dna.html#cb12-2" aria-hidden="true" tabindex="-1"></a>dna <span class="ot">&lt;-</span> <span class="fu">as.vector</span>(<span class="fu">c</span>(<span class="st">&quot;c&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;a&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;t&quot;</span>,<span class="st">&quot;c&quot;</span>,<span class="st">&quot;g&quot;</span>,<span class="st">&quot;a&quot;</span>))</span>
<span id="cb12-3"><a href="dna.html#cb12-3" aria-hidden="true" tabindex="-1"></a>aminoseq1 <span class="ot">&lt;-</span> <span class="fu">noquote</span>(<span class="fu">translate</span>(dna))</span>
<span id="cb12-4"><a href="dna.html#cb12-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb12-5"><a href="dna.html#cb12-5" aria-hidden="true" tabindex="-1"></a><span class="co">#create an empty matrix for 10000000 replications</span></span>
<span id="cb12-6"><a href="dna.html#cb12-6" aria-hidden="true" tabindex="-1"></a>mutations <span class="ot">&lt;-</span> <span class="fu">as.matrix</span>(<span class="fu">rep</span>(<span class="cn">NA</span>, <span class="dv">1000000</span>))</span>
<span id="cb12-7"><a href="dna.html#cb12-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb12-8"><a href="dna.html#cb12-8" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(mutations)) {</span>
<span id="cb12-9"><a href="dna.html#cb12-9" aria-hidden="true" tabindex="-1"></a>values1 <span class="ot">&lt;-</span> <span class="fu">sample</span>(<span class="fu">c</span>(<span class="st">&quot;a&quot;</span>, <span class="st">&quot;c&quot;</span>, <span class="st">&quot;t&quot;</span>, <span class="st">&quot;g&quot;</span>), <span class="dv">1</span>)</span>
<span id="cb12-10"><a href="dna.html#cb12-10" aria-hidden="true" tabindex="-1"></a>dnanew <span class="ot">&lt;-</span> <span class="fu">replace</span>(dna, <span class="fu">sample</span>(<span class="dv">1</span><span class="sc">:</span><span class="fu">length</span>(dna),<span class="dv">1</span>), values1)</span>
<span id="cb12-11"><a href="dna.html#cb12-11" aria-hidden="true" tabindex="-1"></a>aminoseqnew <span class="ot">&lt;-</span> <span class="fu">noquote</span>(<span class="fu">translate</span>(dnanew))</span>
<span id="cb12-12"><a href="dna.html#cb12-12" aria-hidden="true" tabindex="-1"></a>mutations[i] <span class="ot">&lt;-</span> <span class="fu">identical</span>(aminoseq1, aminoseqnew)</span>
<span id="cb12-13"><a href="dna.html#cb12-13" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb12-14"><a href="dna.html#cb12-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb12-15"><a href="dna.html#cb12-15" aria-hidden="true" tabindex="-1"></a><span class="co">#calculate the proportion of mutations that are synonymous i.e., the proportion of single base pair mutations did not change the resulting amino acid sequence </span></span>
<span id="cb12-16"><a href="dna.html#cb12-16" aria-hidden="true" tabindex="-1"></a><span class="fu">length</span>(mutations[mutations <span class="sc">==</span> <span class="st">&quot;TRUE&quot;</span>])<span class="sc">/</span><span class="dv">1000000</span></span></code></pre></div>
<pre><code>## [1] 0.462907</code></pre>
<p>How often does the mutation lead to a synonymous change? What happens if you introduce more than one DNA mutation?</p>
</div>
</div>
<div id="probabilities-snps-and-genetic-variation" class="section level2 hasAnchor" number="3.4">
<h2><span class="header-section-number">3.4</span> Probabilities, SNPs, and genetic variation<a href="dna.html#probabilities-snps-and-genetic-variation" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>The <strong>genetic variation</strong> of any species describes how many copies of alleles are present in a particular locus, averaged over all of the loci in the genome. Geneticists have developed many statistics that quantify genetic variation. For example, average <strong>heterozygosity</strong> describes the proportion of individuals in a population that have two copies of an allele averaged over loci in their genome. Since each individual (at least in a <strong>diploid</strong> species) gets one copy of an allele from their mother and one from their father, individuals who inherit different versions from each parent are referred to as <strong>heterozygotes</strong> because they get different copies from each parent. An individual who inherits the same copy is referred to as a <strong>homozygote</strong>. In addition to heterozygosity, there are other measures of population variation. The most basic is simply to count the number of nucelotide changes in the aligned set of DNA sequences sampled from a population. These sites, known as <strong>Single Nucleotide Polymorphisms</strong>, or SNPs for short, can be reported either as a count or as a proportion of sites in the sequence that are variable. Other metrics, such as nucleotide polymorphism, measure the expected proportion of variable sites in a sequence of a particular length sampled as a function of sample size. These ideas will be covered more in Chapter 5 Population genetics and conservation.</p>
<p>Regardless of the metric used, all measures of genetic variation are influenced by a <em>relationship between the rate of mutation and the number of individuals in the population</em>. Essentially, larger populations are expected to have more genetic variation than smaller populations because there are more opportunities for mutations to occur. However, changes in population size influence this amount of genetic diversity. Populations that start small and grow rapidly often have less genetic diversity than we might expect due to census sizes alone. For example, human genetic diversity is more characteristic of a population of size 10,000 than our actual size of seven+ billion individuals, because the number of humans on the planet was very small prior to the agricultural revolution. On the other hand, large populations that are suddenly reduced in size will lose genetic variation if individuals carrying a given variant do not leave offspring.</p>
<p>In section 3.2 we introduced a concept known as the mutation rate to describe the average rate at which mutations lead to the introduction of new alleles in the population. In most species this rate is quite low, for example 2.5 x 10<sup>-9</sup> mutations per site per generation estimated for humans. When a natural population is sampled for genetic variation, it may be the case that two or more individuals share a particular SNP at a given location in the nucleotide sequence. For example, in a codon where most of the population has a GCC, perhaps two individuals have a GAC. In such cases there are two possible explanations. It could be the case that these individuals each had a parent that independently had a mutation that changed the C to an A in the second position of the codon. Given a mutation rate of 2.5 x 10<sup>-9</sup>, the odds that this happens independently is (2.5 x 10<sup>-9</sup>)<sup>2</sup>, or 6.25 x 10<sup>-18</sup>. Your odds of winning the lottery on the same day that you are struck by lightning and attacked by a shark are much higher than this! Alternatively, these individuals could share the SNP because they each inherited this variant from a parent, and their parents inherited the same variant from their parents, etcetera until there is one individual who inherited this variant from a parent and passed it along to multiple offspring. This is far more likely to occur, in fact the rough probability is equal to 1 - 6.25 x 10<sup>-18</sup>, so much so that we interpret the presence of shared SNP variants as evidence of relatedness when they occur among individuals within species and of common ancestry when they occur between species. Understanding how individuals are related allows us to understand patterns in biodiversity!</p>
<p><img src="snps.png" style="width:75.0%" /></p>
   <font size="2"><strong> Figure 3C: </strong>An example of single nucleotide polymorphisms (SNPs) in an aligned set of DNA sequences. Figure by Danielle J Parsons.</font></p>
</div>
<div id="modeling-sequence-variation" class="section level2 hasAnchor" number="3.5">
<h2><span class="header-section-number">3.5</span> Modeling sequence variation<a href="dna.html#modeling-sequence-variation" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Genetic variation can be used for a wide range of applications, including many that you will be introduced to in later chapters. In turn, many of these applications rely on some quantification of the genetic differences among individuals, for example by counting the proportion of different bases between any pair of individuals. However, because some types of changes are less likely than others it may be the case that a simple count does not represent the actual degree of difference between individuals accurately. For example, due to the chemical structure of the DNA molecule, mutations that replace one purine with another purine or one pyrimidine with another pyrimidine are about twice as likely as a mutation that replaces a purine with a pyrimidine or a pyrimidine with a purine. The former (<em>i.e.</em>, like replaced with like) are known as <strong>transitions</strong> while the latter (<em>i.e.</em>, purine replaced with pyrimidine) are known as <strong>transversions</strong>. In addition, there are some locations in the sequence that are far more important to the functionality of the synthesized protein than others. For example, mutations in the first codon position are rarer than those in the third codon position because the former always lead to a new amino acid. In addition, the three-dimensional structure of the protein can influence how important a given position in the amino acid sequence is to the overall function of the protein. For this reason, the nucleotide sites in a gene sequence effectively have varying rates of one nucleotide base being substituted for another.</p>
<p><img src="trans_travs.jpg" style="width:75.0%" /></p>
  <font size="2"><strong> Figure 3D: </strong>Types of mutations: transitions vs transversions. Figure by Danielle J Parsons. </font></p>
<p>In order to measure genetic distances among individuals accurately, geneticists have developed models of DNA substitution that should be applied when analyzing sequence variation. For example, many models start with estimates of the DNA base composition, the ratio of transitions to transversions (Ti/Tv ratio), and the proportion of sites in the sequence that do not vary (proportion of invariant sites or ‘pinvar’).</p>
<div id="practice-estimate-base-pair-composition" class="section level3 hasAnchor" number="3.5.1">
<h3><span class="header-section-number">3.5.1</span> Practice: Estimate base pair composition<a href="dna.html#practice-estimate-base-pair-composition" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>We can look at the composition of nucleotide bases in genetic data that have been collected from a number of individuals from the same species. First, we will load data from <em>phylogatR</em> into R (more details about these sequences and how to do this yourself will follow in later chapters). Here, we’re using data from four shrew species that are stored in a folder named “phylogatr-results-DNA”.</p>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb14-1"><a href="dna.html#cb14-1" aria-hidden="true" tabindex="-1"></a><span class="co">#load data</span></span>
<span id="cb14-2"><a href="dna.html#cb14-2" aria-hidden="true" tabindex="-1"></a>S.pal <span class="ot">&lt;-</span> <span class="fu">read.multiFASTA</span>(<span class="at">files=</span><span class="st">&quot;phylogatr-results-DNA/Sorex-palustris/Sorex-palustris-CYTB.afa&quot;</span>)</span>
<span id="cb14-3"><a href="dna.html#cb14-3" aria-hidden="true" tabindex="-1"></a>S.mon <span class="ot">&lt;-</span> <span class="fu">read.multiFASTA</span>(<span class="at">files=</span><span class="st">&quot;phylogatr-results-DNA/Sorex-monticolus/Sorex-monticolus-CYTB.afa&quot;</span>)</span>
<span id="cb14-4"><a href="dna.html#cb14-4" aria-hidden="true" tabindex="-1"></a>S.cin <span class="ot">&lt;-</span> <span class="fu">read.multiFASTA</span>(<span class="at">files=</span><span class="st">&quot;phylogatr-results-DNA/Sorex-cinereus/Sorex-cinereus-CYTB.afa&quot;</span>)</span>
<span id="cb14-5"><a href="dna.html#cb14-5" aria-hidden="true" tabindex="-1"></a>S.ben <span class="ot">&lt;-</span> <span class="fu">read.multiFASTA</span>(<span class="at">files=</span><span class="st">&quot;phylogatr-results-DNA/Sorex-bendirii/Sorex-bendirii-CYTB.afa&quot;</span>)</span></code></pre></div>
<p>All the sequences for each individual for a location in the genome are aligned so they can be compared. We can plot the data from the DNA sequence alignments for visualization using the code below. The different colors represent different bases (A,T,G, and C) where each column is a different nucleotide and each individual is on a different row.</p>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb15-1"><a href="dna.html#cb15-1" aria-hidden="true" tabindex="-1"></a><span class="co">#plot data</span></span>
<span id="cb15-2"><a href="dna.html#cb15-2" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(S.pal)</span></code></pre></div>
<p><img src="images/unnamed-chunk-8-1.png" width="672" /></p>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb16-1"><a href="dna.html#cb16-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(S.mon)</span></code></pre></div>
<p><img src="images/unnamed-chunk-8-2.png" width="672" /></p>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb17-1"><a href="dna.html#cb17-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(S.cin)</span></code></pre></div>
<p><img src="images/unnamed-chunk-8-3.png" width="672" /></p>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="dna.html#cb18-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(S.ben)</span></code></pre></div>
<p><img src="images/unnamed-chunk-8-4.png" width="672" /></p>
<p>Finally, let’s estimate the composition of base pairs present in each of our alignments. This will tell us the frequency at which each base is present in the genetic data for our chosen species. Then, we can plot these values using a bar chart, and visually compare base pair frequencies for each species.</p>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb19-1"><a href="dna.html#cb19-1" aria-hidden="true" tabindex="-1"></a><span class="co">#Estimate base pair composition</span></span>
<span id="cb19-2"><a href="dna.html#cb19-2" aria-hidden="true" tabindex="-1"></a>S.paldnabin <span class="ot">&lt;-</span> <span class="fu">concatenate</span>(S.pal)</span>
<span id="cb19-3"><a href="dna.html#cb19-3" aria-hidden="true" tabindex="-1"></a>S.palvar <span class="ot">&lt;-</span> <span class="fu">base.freq</span>(S.paldnabin, <span class="at">freq=</span><span class="cn">FALSE</span>, <span class="at">all=</span><span class="cn">TRUE</span>)</span>
<span id="cb19-4"><a href="dna.html#cb19-4" aria-hidden="true" tabindex="-1"></a>S.palvar[<span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">3</span>,<span class="dv">2</span>,<span class="dv">4</span>,<span class="dv">15</span>,<span class="dv">16</span>)]</span></code></pre></div>
<pre><code>##          a          g          c          t          n          - 
## 0.26784539 0.12083333 0.24451754 0.27513706 0.03843202 0.05323465</code></pre>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb21-1"><a href="dna.html#cb21-1" aria-hidden="true" tabindex="-1"></a>S.mondnabin <span class="ot">&lt;-</span> <span class="fu">concatenate</span>(S.mon)</span>
<span id="cb21-2"><a href="dna.html#cb21-2" aria-hidden="true" tabindex="-1"></a>S.monvar <span class="ot">&lt;-</span> <span class="fu">base.freq</span>(S.mondnabin, <span class="at">freq=</span><span class="cn">FALSE</span>, <span class="at">all=</span><span class="cn">TRUE</span>)</span>
<span id="cb21-3"><a href="dna.html#cb21-3" aria-hidden="true" tabindex="-1"></a>S.monvar[<span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">3</span>,<span class="dv">2</span>,<span class="dv">4</span>,<span class="dv">15</span>,<span class="dv">16</span>)]</span></code></pre></div>
<pre><code>##           a           g           c           t           n           - 
## 0.265947452 0.125442754 0.255080236 0.272054544 0.005807766 0.075667249</code></pre>
<div class="sourceCode" id="cb23"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb23-1"><a href="dna.html#cb23-1" aria-hidden="true" tabindex="-1"></a>S.cindnabin <span class="ot">&lt;-</span> <span class="fu">concatenate</span>(S.cin)</span>
<span id="cb23-2"><a href="dna.html#cb23-2" aria-hidden="true" tabindex="-1"></a>S.cinvar <span class="ot">&lt;-</span> <span class="fu">base.freq</span>(S.cindnabin, <span class="at">freq=</span><span class="cn">FALSE</span>, <span class="at">all=</span><span class="cn">TRUE</span>)</span>
<span id="cb23-3"><a href="dna.html#cb23-3" aria-hidden="true" tabindex="-1"></a>S.cinvar[<span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">3</span>,<span class="dv">2</span>,<span class="dv">4</span>,<span class="dv">15</span>,<span class="dv">16</span>)]</span></code></pre></div>
<pre><code>##            a            g            c            t            n            - 
## 0.2856485476 0.1294974116 0.2667888985 0.2869427668 0.0001869428 0.0309354328</code></pre>
<div class="sourceCode" id="cb25"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb25-1"><a href="dna.html#cb25-1" aria-hidden="true" tabindex="-1"></a>S.bendnabin <span class="ot">&lt;-</span> <span class="fu">concatenate</span>(S.ben)</span>
<span id="cb25-2"><a href="dna.html#cb25-2" aria-hidden="true" tabindex="-1"></a>S.benvar <span class="ot">&lt;-</span> <span class="fu">base.freq</span>(S.bendnabin, <span class="at">freq=</span><span class="cn">FALSE</span>, <span class="at">all=</span><span class="cn">TRUE</span>)</span>
<span id="cb25-3"><a href="dna.html#cb25-3" aria-hidden="true" tabindex="-1"></a>S.benvar[<span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">3</span>,<span class="dv">2</span>,<span class="dv">4</span>,<span class="dv">15</span>,<span class="dv">16</span>)]</span></code></pre></div>
<pre><code>##         a         g         c         t         n         - 
## 0.2333333 0.1134615 0.2294872 0.2339744 0.0000000 0.1897436</code></pre>
<div class="sourceCode" id="cb27"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb27-1"><a href="dna.html#cb27-1" aria-hidden="true" tabindex="-1"></a><span class="co">#Plot base pair frequencies</span></span>
<span id="cb27-2"><a href="dna.html#cb27-2" aria-hidden="true" tabindex="-1"></a>colors <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">&quot;red&quot;</span>, <span class="st">&quot;green&quot;</span>, <span class="st">&quot;yellow&quot;</span>, <span class="st">&quot;blue&quot;</span>, <span class="st">&quot;gray&quot;</span>, <span class="st">&quot;black&quot;</span>)</span>
<span id="cb27-3"><a href="dna.html#cb27-3" aria-hidden="true" tabindex="-1"></a><span class="fu">par</span>(<span class="at">mfrow =</span> <span class="fu">c</span>(<span class="dv">2</span>, <span class="dv">2</span>))</span>
<span id="cb27-4"><a href="dna.html#cb27-4" aria-hidden="true" tabindex="-1"></a><span class="fu">barplot</span>(S.palvar[<span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">3</span>,<span class="dv">2</span>,<span class="dv">4</span>,<span class="dv">15</span>,<span class="dv">16</span>)], <span class="at">col=</span>colors, <span class="at">xlab=</span><span class="st">&quot;Nucleotide&quot;</span>, <span class="at">ylab=</span><span class="st">&quot;Frequency&quot;</span>, <span class="at">main =</span><span class="fu">names</span>(S.pal<span class="sc">@</span>dna))</span>
<span id="cb27-5"><a href="dna.html#cb27-5" aria-hidden="true" tabindex="-1"></a><span class="fu">barplot</span>(S.monvar[<span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">3</span>,<span class="dv">2</span>,<span class="dv">4</span>,<span class="dv">15</span>,<span class="dv">16</span>)], <span class="at">col=</span>colors, <span class="at">xlab=</span><span class="st">&quot;Nucleotide&quot;</span>, <span class="at">ylab=</span><span class="st">&quot;Frequency&quot;</span>, <span class="at">main =</span><span class="fu">names</span>(S.mon<span class="sc">@</span>dna))</span>
<span id="cb27-6"><a href="dna.html#cb27-6" aria-hidden="true" tabindex="-1"></a><span class="fu">barplot</span>(S.cinvar[<span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">3</span>,<span class="dv">2</span>,<span class="dv">4</span>,<span class="dv">15</span>,<span class="dv">16</span>)], <span class="at">col=</span>colors, <span class="at">xlab=</span><span class="st">&quot;Nucleotide&quot;</span>, <span class="at">ylab=</span><span class="st">&quot;Frequency&quot;</span>, <span class="at">main =</span><span class="fu">names</span>(S.cin<span class="sc">@</span>dna))</span>
<span id="cb27-7"><a href="dna.html#cb27-7" aria-hidden="true" tabindex="-1"></a><span class="fu">barplot</span>(S.benvar[<span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">3</span>,<span class="dv">2</span>,<span class="dv">4</span>,<span class="dv">15</span>,<span class="dv">16</span>)], <span class="at">col=</span>colors, <span class="at">xlab=</span><span class="st">&quot;Nucleotide&quot;</span>, <span class="at">ylab=</span><span class="st">&quot;Frequency&quot;</span>, <span class="at">main =</span><span class="fu">names</span>(S.ben<span class="sc">@</span>dna))</span></code></pre></div>
<p><img src="images/unnamed-chunk-9-1.png" width="672" /></p>
</div>
</div>
<div id="organellar-genomes-and-barcoding-genes" class="section level2 hasAnchor" number="3.6">
<h2><span class="header-section-number">3.6</span> Organellar genomes and barcoding genes<a href="dna.html#organellar-genomes-and-barcoding-genes" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>By far the vast majority of DNA in <strong>Eukaryotic</strong> organisms is contained in the <strong>nuclear genome</strong>. For example, most humans have 23 pairs of chromosomes that collectively contain the approximately 6.4 billion nucleotide base pairs. However, Eukaryotes also contain small amounts of DNA in extranuclear organelles. For example, green plants have <strong>chloroplasts</strong> which are vital in the production of energy via photosynthesis, and plants, animals, and fungi contain <strong>mitochondria</strong> which are essential for respiration and energy production. In many ways, organellar DNA are easier for geneticists to work with because there can be dozens of the same organelle in a single cell, each with its own copy of DNA. When DNA is extracted from a tissue sample that contain some number of cells, there are thus many more copies of a given organellar gene than there would be for a given gene from the nuclear genome. This makes these genes easier to work with. In addition, because these organelles are the parts of the cell that are responsible for respiration, oxidative damage can lead to DNA mutation rates that may be an order of magnitude higher than in the nuclear genome. For certain types of questions, having more genetic variation in a sample can be useful because it gives us more information about a species. As a result of the fact that organellar genes were easier to sequence with the original sequencing technologies and often have more genetic variation (higher mutation rate), the <em>phylogatR</em> database has a disproportional number of genes from the mitochondria in animals and the chloroplast in plants.</p>
<p>Some genes in organellar genomes evolve so quickly that each species is likely to have a unique sequence of nucleotide bases. This fact has led to the proposal that species can be identified by using this unique sequence as a sort of DNA-based <strong>barcode</strong>, similar to how every item at the grocery store has a unique barcode that is used for identification at the checkout. Notice how the sequence alignments above kind of look like barcodes? While academic debate is ongoing about the degree to which DNA barcodes can be applied to important biodiversity studies, the possibility is tantalizing. For example, a comprehensive database of DNA barcodes could be invaluable for species identification used in the tracking of invasive species or to detect smuggling of endangered species or for use with biodiversity monitoring using environmental DNA.</p>
<p><img src="Mitochondrial_DNA_lg.jpg" style="width:60.0%" /></p>
  <font size="2"><strong> Figure 3E: </strong>Mitochondrial DNA (mtDNA). Image from the National Human Genome Research Institute.</font></p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="introduction.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="biodiversity-databases.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
