<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>5 Population genetics and conservation | PhylogatR</title>
  <meta name="description" content="" />
  <meta name="generator" content="bookdown 0.26.3 and GitBook 2.6.7" />

  <meta property="og:title" content="5 Population genetics and conservation | PhylogatR" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="5 Population genetics and conservation | PhylogatR" />
  
  
  

<meta name="author" content="Bryan Carstens, Sydney Decker, Anahi Espindola, Sarah Foltz, Abbi Green, Danielle Parsons, Tara Pelletier, Jordan Satler" />


<meta name="date" content="2022-06-02" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="biodiversity-databases.html"/>
<link rel="next" href="species-limits.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Background</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#how-to-use-this-book"><i class="fa fa-check"></i><b>1.1</b> How to use this book</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#prerequisites-if-using-r"><i class="fa fa-check"></i><b>1.2</b> Prerequisites if using R</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="index.html"><a href="index.html#r-and-rstudio"><i class="fa fa-check"></i><b>1.2.1</b> R and RStudio</a></li>
<li class="chapter" data-level="1.2.2" data-path="index.html"><a href="index.html#r-packages"><i class="fa fa-check"></i><b>1.2.2</b> R packages</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>2</b> Introduction</a>
<ul>
<li class="chapter" data-level="2.1" data-path="introduction.html"><a href="introduction.html#data-in-the-biological-sciences"><i class="fa fa-check"></i><b>2.1</b> Data in the biological sciences</a></li>
<li class="chapter" data-level="2.2" data-path="introduction.html"><a href="introduction.html#why-species"><i class="fa fa-check"></i><b>2.2</b> Why species?</a></li>
<li class="chapter" data-level="2.3" data-path="introduction.html"><a href="introduction.html#the-scientific-method"><i class="fa fa-check"></i><b>2.3</b> The Scientific Method</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="introduction.html"><a href="introduction.html#an-image-demonstrating-what-we-mean-by-model-and-testing-them"><i class="fa fa-check"></i><b>2.3.1</b> an image demonstrating what we mean by model and testing them???</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="dna.html"><a href="dna.html"><i class="fa fa-check"></i><b>3</b> DNA</a>
<ul>
<li class="chapter" data-level="3.1" data-path="dna.html"><a href="dna.html#the-genetic-code"><i class="fa fa-check"></i><b>3.1</b> The genetic code</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="dna.html"><a href="dna.html#practice-translate-the-following-dna-sequence-into-amino-acids"><i class="fa fa-check"></i><b>3.1.1</b> Practice: Translate the following DNA sequence into amino acids</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="dna.html"><a href="dna.html#mutation"><i class="fa fa-check"></i><b>3.2</b> Mutation</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="dna.html"><a href="dna.html#practice-this-section-and-code-needs-descriptions"><i class="fa fa-check"></i><b>3.2.1</b> Practice: this section and code needs descriptions</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="dna.html"><a href="dna.html#synonymous-versus-non-synonymous-genetic-variation"><i class="fa fa-check"></i><b>3.3</b> Synonymous versus non-synonymous genetic variation</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="dna.html"><a href="dna.html#practice-introduce-a-mutation"><i class="fa fa-check"></i><b>3.3.1</b> Practice: Introduce a mutation</a></li>
<li class="chapter" data-level="3.3.2" data-path="dna.html"><a href="dna.html#practice-introduce-an-insertion"><i class="fa fa-check"></i><b>3.3.2</b> Practice: Introduce an insertion</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="dna.html"><a href="dna.html#probabilities-snps-and-genetic-variation"><i class="fa fa-check"></i><b>3.4</b> Probabilities, SNPs, and genetic variation</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="dna.html"><a href="dna.html#this-is-a-lot-of-text---maybe-some-kind-of-graphic"><i class="fa fa-check"></i><b>3.4.1</b> this is a lot of text - maybe some kind of graphic?</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="dna.html"><a href="dna.html#modeling-sequence-variation"><i class="fa fa-check"></i><b>3.5</b> Modeling sequence variation</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="dna.html"><a href="dna.html#transitiontranservion-image"><i class="fa fa-check"></i><b>3.5.1</b> transition/transervion image?</a></li>
<li class="chapter" data-level="3.5.2" data-path="dna.html"><a href="dna.html#practice-estimate-base-pair-composition"><i class="fa fa-check"></i><b>3.5.2</b> Practice: Estimate base pair composition</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="dna.html"><a href="dna.html#organellar-genomes-and-barcoding-genes"><i class="fa fa-check"></i><b>3.6</b> Organellar genomes and barcoding genes</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="dna.html"><a href="dna.html#image-of-mt-genome---need-to-find-one-that-is-open-access"><i class="fa fa-check"></i><b>3.6.1</b> image of mt genome - need to find one that is open access</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="dna.html"><a href="dna.html#shinyr-app"><i class="fa fa-check"></i><b>3.7</b> ShinyR app???</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html"><i class="fa fa-check"></i><b>4</b> Biodiversity Databases</a>
<ul>
<li class="chapter" data-level="4.1" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#global-biodiversity-information-facility-gbif"><i class="fa fa-check"></i><b>4.1</b> Global Biodiversity Information Facility (GBIF)</a></li>
<li class="chapter" data-level="4.2" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#genbank"><i class="fa fa-check"></i><b>4.2</b> GenBank</a></li>
<li class="chapter" data-level="4.3" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#bold"><i class="fa fa-check"></i><b>4.3</b> BOLD</a></li>
<li class="chapter" data-level="4.4" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#phylogatr"><i class="fa fa-check"></i><b>4.4</b> <em>PhylogatR</em></a></li>
<li class="chapter" data-level="4.5" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#using-phylogatr"><i class="fa fa-check"></i><b>4.5</b> Using <em>phylogatR</em></a></li>
<li class="chapter" data-level="4.6" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#practice"><i class="fa fa-check"></i><b>4.6</b> Practice</a>
<ul>
<li class="chapter" data-level="4.6.1" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#downloading-data-from-phylogatr"><i class="fa fa-check"></i><b>4.6.1</b> Downloading data from <em>phylogatR</em></a></li>
<li class="chapter" data-level="4.6.2" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#check-for-outliers"><i class="fa fa-check"></i><b>4.6.2</b> Check for outliers</a></li>
<li class="chapter" data-level="4.6.3" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#check-sequence-alignments"><i class="fa fa-check"></i><b>4.6.3</b> Check sequence alignments</a></li>
<li class="chapter" data-level="4.6.4" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#visualizing-geographic-coordinates"><i class="fa fa-check"></i><b>4.6.4</b> Visualizing geographic coordinates</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html"><i class="fa fa-check"></i><b>5</b> Population genetics and conservation</a>
<ul>
<li class="chapter" data-level="5.1" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#loss-of-genetic-diversity"><i class="fa fa-check"></i><b>5.1</b> Loss of genetic diversity</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#small-population-size"><i class="fa fa-check"></i><b>5.1.1</b> Small population size</a></li>
<li class="chapter" data-level="5.1.2" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#genetic-drift"><i class="fa fa-check"></i><b>5.1.2</b> Genetic drift</a></li>
<li class="chapter" data-level="5.1.3" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#gene-flow"><i class="fa fa-check"></i><b>5.1.3</b> Gene flow</a></li>
<li class="chapter" data-level="5.1.4" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#inbreeding"><i class="fa fa-check"></i><b>5.1.4</b> Inbreeding</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#units-of-conservation"><i class="fa fa-check"></i><b>5.2</b> Units of conservation</a></li>
<li class="chapter" data-level="5.3" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#practice-1"><i class="fa fa-check"></i><b>5.3</b> Practice</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#learning-objectives-1"><i class="fa fa-check"></i><b>5.3.1</b> Learning objectives:</a></li>
<li class="chapter" data-level="5.3.2" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#data"><i class="fa fa-check"></i><b>5.3.2</b> Data</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#click-and-play"><i class="fa fa-check"></i><b>5.4</b> Click and play!</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="species-limits.html"><a href="species-limits.html"><i class="fa fa-check"></i><b>6</b> Species limits</a>
<ul>
<li class="chapter" data-level="6.1" data-path="species-limits.html"><a href="species-limits.html#cryptic-genetic-diversity"><i class="fa fa-check"></i><b>6.1</b> Cryptic genetic diversity</a></li>
<li class="chapter" data-level="6.2" data-path="species-limits.html"><a href="species-limits.html#genetic-clustering"><i class="fa fa-check"></i><b>6.2</b> Genetic clustering</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="species-limits.html"><a href="species-limits.html#structure"><i class="fa fa-check"></i><b>6.2.1</b> Structure</a></li>
<li class="chapter" data-level="6.2.2" data-path="species-limits.html"><a href="species-limits.html#structurama"><i class="fa fa-check"></i><b>6.2.2</b> Structurama</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="species-limits.html"><a href="species-limits.html#tree-based-methods"><i class="fa fa-check"></i><b>6.3</b> Tree-based methods</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="species-limits.html"><a href="species-limits.html#general-mixed-yule-coalescent-gmyc"><i class="fa fa-check"></i><b>6.3.1</b> General mixed Yule Coalescent (GMYC)</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="species-limits.html"><a href="species-limits.html#practice-2"><i class="fa fa-check"></i><b>6.4</b> Practice</a>
<ul>
<li class="chapter" data-level="6.4.1" data-path="species-limits.html"><a href="species-limits.html#data-preparation"><i class="fa fa-check"></i><b>6.4.1</b> Data Preparation</a></li>
<li class="chapter" data-level="6.4.2" data-path="species-limits.html"><a href="species-limits.html#generate-a-phylogenetic-tree"><i class="fa fa-check"></i><b>6.4.2</b> Generate a Phylogenetic Tree</a></li>
<li class="chapter" data-level="6.4.3" data-path="species-limits.html"><a href="species-limits.html#create-a-summary-tree"><i class="fa fa-check"></i><b>6.4.3</b> Create a summary tree</a></li>
<li class="chapter" data-level="6.4.4" data-path="species-limits.html"><a href="species-limits.html#species-delimitation"><i class="fa fa-check"></i><b>6.4.4</b> Species Delimitation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html"><i class="fa fa-check"></i><b>7</b> Predictive Phylogeography</a>
<ul>
<li class="chapter" data-level="7.0.1" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#trait-data-and-response-variables"><i class="fa fa-check"></i><b>7.0.1</b> Trait data and response variables</a></li>
<li class="chapter" data-level="7.1" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#machine-learning-and-random-forest"><i class="fa fa-check"></i><b>7.1</b> Machine learning and random forest</a></li>
<li class="chapter" data-level="7.2" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#real-life-example"><i class="fa fa-check"></i><b>7.2</b> Real life example</a></li>
<li class="chapter" data-level="7.3" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#practice-3"><i class="fa fa-check"></i><b>7.3</b> Practice</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>8</b> References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"><img src="Segoe1.png" title="fig:" style="width:75.0%" alt="PhylogatR" /></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="population-genetics-and-conservation" class="section level1 hasAnchor" number="5">
<h1><span class="header-section-number">5</span> Population genetics and conservation<a href="population-genetics-and-conservation.html#population-genetics-and-conservation" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>Population genetics is the study of the genetic variation (remember what we learned in Chapter 3) within a population of a species, which we will refer to as genetic diversity. This includes the distributions of genetic diversity over geographic space and changes in <strong>genotype</strong> and <strong>phenotype</strong> frequencies in response to the processes of genetic drift, gene flow, and natural selection. See <a href="https://evolution.berkeley.edu/evolution-101/mechanisms-the-processes-of-evolution/mechanisms-of-change/" target="_blank">here</a> for more information. <u><strong>Conservation genetics</strong></u> is the use of this genetic information to inform management decisions to preserve biodiversity.</p>
<p><img src="images/genphen.png" width="672" /></p>
</div>
<div id="loss-of-genetic-diversity" class="section level2 hasAnchor" number="5.1">
<h2><span class="header-section-number">5.1</span> Loss of genetic diversity<a href="population-genetics-and-conservation.html#loss-of-genetic-diversity" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Loss of genetic diversity is a common concern for conservation biologists. Many populations of threatened and endangered species suffer from low diversity of genes at the individual and population levels. But how is genetic diversity lost? There are several factors that can contribute, singly or in concert together, to the loss of genetic variation both in individuals and populations.</p>
<div id="small-population-size" class="section level3 hasAnchor" number="5.1.1">
<h3><span class="header-section-number">5.1.1</span> Small population size<a href="population-genetics-and-conservation.html#small-population-size" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Endangered species typically have declining numbers of individuals. As individuals are lost, so is the variation of the DNA that they carry. Factors that increase mortality or reduce birth rates, such as disease, pollution, overhunting, or low resource availability due to habitat destruction all cause populations to shrink. Additionally, destruction of a species habitat in particular places can create smaller populations by breaking up the remaining habitat areas that are suitable for the species. This <strong>habitat fragmentation</strong> also isolates populations from each other; populations are trapped in pockets of remaining habitat but are unable to interact with other populations due to inhospitable habitat or barriers like roads or fences blocking their movement.</p>
<p>The total number of individuals within a population is not the only factor that needs to be considered when thinking about how population size affects genetic diversity. Within any population, only a subset of the individuals will be capable of reproducing at a given time; others may be too young, too old, or too unhealthy to reproduce. The number of breeding individuals in a population is called the <strong>effective population size</strong>, and is necessarily smaller than the total population size. Because it is only these breeding individuals who will pass their DNA on to the next generation, the effective population size directly impacts genetic diversity. Certain factors, such as selective hunting or poaching, can disproportionately reduce effective population size. For example, older adult African elephants have been the primary target of poachers in search of ivory because elephant tusk size increases with age. This targeting of adults has affected elephant population demographics, skewing populations younger overall and inhibiting reproduction in heavily poached populations.</p>
<p>Because each individual in a population can only carry a certain number of alleles for each gene, small populations tend to have fewer alleles overall and the alleles that are present may only be carried by a handful of individuals, making them rare. These <strong>rare alleles</strong> are vulnerable to loss; if anything prevents those individuals from breeding, they will not be present in the next generation of that population. However, this is not the only reason that small population size is concerning. For <strong>sexually-reproducing</strong> species, smaller populations also means fewer potential mates; lack of options for mates may result in fewer individuals breeding (further reducing effective population size) or reduced offspring quality due to poor compatibility of parents who might not have paired if they had had more choices of mates.</p>
</div>
<div id="genetic-drift" class="section level3 hasAnchor" number="5.1.2">
<h3><span class="header-section-number">5.1.2</span> Genetic drift<a href="population-genetics-and-conservation.html#genetic-drift" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><strong>Genetic drift</strong> is a process in which populations diverge genetically from other populations of the same species. This occurs partly through the loss of rare alleles due to random occurrences that have nothing to do with the function or adaptive value of the allele in question. When populations are small, the loss of genetic diversity due to random processes is bigger.</p>
<p>Conservation biologists must consider the negative impacts of genetic drift not only in existing populations, but also in captive breeding programs and in newly-established populations created during reintroduction efforts. <strong>Founder effects</strong> occur whenever a subset of an existing population is used to create a new one. These founding individuals may not carry a representative proportion of the genetic diversity in the existing population, especially if only a small number of founders are used. This in turn leads to the new population having lower genetic diversity than the previous population it was derived from. Conservationists selecting individuals for use in captive breeding and reintroduction efforts must pay careful attention to the genotypes of these individuals to ensure that as many alleles as possible from the parent population are carried forward into the new populations, minimizing loss of genetic diversity in this process.</p>
<p>A slightly different example of a random loss of alleles is a <strong>population bottleneck</strong>. A bottleneck is an event that suffers a fast (evolutionary speaking) reduction in population size where may alleles are lost. For example, if a hiker accidentally stepping on a group of brown and green beetles; if the hiker’s foot happens to fall on all of the green beetles in the group, the allele that causes this green coloration may be lost in this beetle population, even though an individual beetle’s color did not make it more or less likely to be stepped on. This example also highlights a connection between small population size and genetic drift; if the beetle population were larger, it is unlikely that the hiker’s foot would randomly land on all of the green beetles. Although genetic drift occurs in all populations, larger populations are less likely to suffer extreme effects such as total loss of an allele than small populations, due to increased redundancy in the larger populations; larger population size simply means there are probably more copies of any given allele. Small populations are more likely to have alleles that are carried by only a few individuals, and these rare alleles are vulnerable to loss by random chance.</p>
</div>
<div id="gene-flow" class="section level3 hasAnchor" number="5.1.3">
<h3><span class="header-section-number">5.1.3</span> Gene flow<a href="population-genetics-and-conservation.html#gene-flow" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Isolation of populations also makes them more vulnerable to the effects of genetic drift. Isolation prevents <strong>gene flow</strong>, the sharing of alleles between populations through the dispersal of individuals into and out of these different breeding groups. A population that frequently trades individuals with other nearby populations of the species may lose a particular allele through genetic drift, but then regain it through gene flow when a new member carrying that allele joins the population. Isolated populations do not have this outside source of alleles; once an allele is lost through drift, it remains gone. Conservation biologists are particularly concerned about small, isolated populations because these populations drift faster than larger, better-connected populations. Habitat fragmentation contributes to population isolation as well as reducing population size, and so is a major factor contributing to loss of genetic diversity. A reduction in gene flow leads to population differentiation and can be measured using genetic data.</p>
</div>
<div id="inbreeding" class="section level3 hasAnchor" number="5.1.4">
<h3><span class="header-section-number">5.1.4</span> Inbreeding<a href="population-genetics-and-conservation.html#inbreeding" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>When two individuals with many of the same alleles breed, their offspring are likely to have lower genetic diversity than the offspring of individuals with very different alleles. They are also more likely to be <strong>homozygous</strong> at many loci; in other words, they are likely to carry two copies of the same allele for a given gene, because that was the allele carried by both of their parents. In contrast, an individual with two different alleles for a gene is considered <strong>heterozygous</strong>. Breeding of genetically similar individuals who produce highly homozygous offspring is referred to as <strong>inbreeding</strong>, and is often a result of matings between close relatives such as siblings.</p>
<p>Inbreeding occurs when mate choice is limited, such as when overall or effective population size is very small. In these situations, relatives may be the only available mate options. This is especially common in isolated populations, where dispersal to a new population in search of mates is not possible and unrelated individuals are not immigrating in.</p>
<p>In some cases, inbreeding can be exacerbated by population-specific changes in behaviors that affect mate choice. In these situations, individuals may prefer to breed with members of their own population, who are more genetically similar to them, even when presented with potential mates from outside populations. For example, a study of rufous-collared sparrows found that female sparrows preferred the songs of males from their own population to those from a neighboring population that sang slightly differently, a preference that likely limits gene flow between these two populations (Danner et al. 2011)</p>
<p>Inbreeding is of concern in conservation biology because of its potential to reduce genetic diversity at both the individual and population levels. One reason for concern is inbreeding depression, a situation in which homozygosity at one or more loci produces individuals who are lower quality and less fit. This happens when certain alleles are maladaptive. For example, a gene may code for a particular protein, but a mutation in one allele for that gene causes that allele to produce a malformed protein that cannot be used. Such ‘broken’ alleles can be compensated for by the presence of a different allele for the same gene; in this example, a heterozygous individual could carry one copy of the broken allele but still have a working version of the protein. However, homozygous individuals are at risk of carrying two broken versions, and thus being unable to produce the necessary protein. While natural selection often removes deleterious mutations from populations over time, alleles with negative mutations can remain in populations indefinitely if they are recessive and so their maladaptive effects only appear in individuals that are homozygous.</p>
<p>Heterozygosity is also beneficial at the population level. Different combinations of alleles can produce unique phenotypes not created by any allele paired with itself. This increased diversity of phenotypes in a population produces greater flexibility when faced with environmental challenges. A diverse population is more likely to have individuals within it whose combination of traits is adaptive in the current environmental conditions and so are able to survive the challenge and pass on those alleles. Therefore, populations that are more heterozygous overall have greater potential to adapt to environmental changes, rather than be driven extinct by them.</p>
<div id="measuring-heterozygosity-within-and-between-populations" class="section level4 hasAnchor" number="5.1.4.1">
<h4><span class="header-section-number">5.1.4.1</span> Measuring heterozygosity within and between populations<a href="population-genetics-and-conservation.html#measuring-heterozygosity-within-and-between-populations" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Heterozygosity is a measure of genetic variation that can be estimated within an individual, within a population of individuals of the same species, and between populations of the same species. These measurements of genetic variation can inform conservation management decisions by guiding the development of dispersal corridors and/or translocation events to restore levels of genetic variation within species.</p>
<p>Heterozygosity in a population is the proportion of individuals that are heterozygous at that locus, and can be averaged across many loci (H). This estimate is used as a measure of genetic variation within a population to assess population stability, in other words, is there enough genetic variation among individuals within a population to avoid a risk of extinction?</p>
<p>Image that represents heterozygosity at each level and includes fragmentation</p>
<div id="heterozygosity-example" class="section level5 hasAnchor" number="5.1.4.1.1">
<h5><span class="header-section-number">5.1.4.1.1</span> Heterozygosity example<a href="population-genetics-and-conservation.html#heterozygosity-example" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<p>In the 1800s and most of the 1900s black bears in FL were hunted without regulation and much of their habitat was destroyed due to human development. By the 1970s, only 300-400 bears remained. This type of reduction in population can severely reduce genetic diversity and black bears were subsequently listed as threatened in FL (cite). Cryptorchidism (undescended tesicles, a common phenotype when mammals suffer from inbreeding depression) was observed in black bears of FL in the mid-nineties (Dunbar et al. 1996). Not only did the FL black bears suffer a decrease in population size and a loss of genetic diversity, their habitat was severely fragmented. Researchers decided to use genetic data to assess the levels of genetic variation within and between populations (Dixon et al. 2007). The used 12 genetic loci to estimate heterozygosity within and between populations. They found that genetic diversity in the FL balck bears was lower in populations that are smaller and less connected to other populations, and that populations with at least 200 individuals is needed to maintain levels of genetic variation to match that of unaffected balck bears in other parts of the country. Finally, even though many of these populations are close enough for bears to disperse, the ability to cross highways is limited (find recent numbers), so the researchers recommend creating dispersal corridors, either naturally, or with the use of crossing structures. In this case, considering populations as units of conservation is important, rather than considering the whole species. For example, if one small population is particularly vulnerable, it is much more cost-effective to focus management decisions on this population, rather than the entire geographic region.</p>
</div>
</div>
</div>
</div>
<div id="units-of-conservation" class="section level2 hasAnchor" number="5.2">
<h2><span class="header-section-number">5.2</span> Units of conservation<a href="population-genetics-and-conservation.html#units-of-conservation" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Throughout this book we have defined species as one of the fundamental levels of biological organization. The designation of species is a useful tool in conservation because it provides a basic unit for management. However, the use of species can be problematic for several different species. First, defining species is not as straightforward as it may seem (see chapter species delimitation). Second, oftentimes, populations within a species might need special consideration (recall FL black bear example from above). The use of genetic tools can aid in the identification of conservation units. Species as units of conservation works by… For example, DNA sequence data was used to estimate the number of species in giraffes several times in order to determine how many units exist and which ones are in most need of conservation. Put short description of studies here.</p>
<p>On the other hand, simply defining species will not always offer the best guidance for management decisions. Evolutionary significant units (ESUs) and Management Units (MUs) are also used to make important management decisions. And ESU is a group of individuals that share an evolutionary history that is unique to other groups within that species (see chapter phylogenetics) and thus, deserve some independent recognition even though they would not be considered a separate species. Considering these units as important can aid in preserving important locally adaptive genetic variation within a species. MUs represent populations that display some accumulation of unique alleles, but are not necessarily phylogenetically distinct. In other words, one considers the evolutionary history of the group as important (ESU), while the other only considers uniqueness of the population, regardless of evolutionary history (MU). This genetic information can be put to use by incorporation geographic information. If a population exists that contains unique genetic variation for that species, management efforts can be made to preserve the geographic species occupied by this specific population, when most of the other genetic variation within the species is likely to be maintained naturally without any intervention.</p>
<div id="include-image-with-phylogatr-data-that-shows-genetic-groups-that-are-clustered-geographically" class="section level4 hasAnchor" number="5.3">
<div id="practice-1" class="section level2 hasAnchor" number="5.3">
<h2><span class="header-section-number">5.3</span> Practice<a href="population-genetics-and-conservation.html#practice-1" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>This genetic variation we have been discussing is resposible for the diversity we observe within species (e.g., thermal tolerance, body size). This diversity is essential if species are going to survive environmental change. The <a href="https://www.cbd.int/" target="_blank" rel="noopener">Convention on Biological Diversity (CBD)</a> included genetic diversity among its 2010 biodiversity targets.</p>
<p>The <a href="https://www.iucnredlist.org/" target="_blank" rel="noopener">IUCN red list</a> is an international effort to list species as either least concern (LC), near threatened (NT), vulnerable (VU), endangered (EN), or critically endangered (CR) in order to adequately protect species that are in need.</p>
<p>This exercise will use data from an open-source genetic database to answer the question: Is the IUCN red list designed to help protect species with low genetic diveristy that would put them at a higher risk of extinction?</p>
<div id="learning-objectives-1" class="section level3 hasAnchor" number="5.3.1">
<h3><span class="header-section-number">5.3.1</span> Learning objectives:<a href="population-genetics-and-conservation.html#learning-objectives-1" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ol style="list-style-type: decimal">
<li>Explain why genetic diversity is an important concept in conservation biology.</li>
<li>Provide examples as to why open-source databases are valuable tools in conservation biology.</li>
<li>Interpret plots in order to answer a research question.</li>
</ol>
<p>If you would like to skip the R code version of this exercise, jump ahead to the click and play section.</p>
</div>
<div id="data" class="section level3 hasAnchor" number="5.3.2">
<h3><span class="header-section-number">5.3.2</span> Data<a href="population-genetics-and-conservation.html#data" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Please see chapter 4 for details on downloading data from <em>phylogatR</em>.</p>
<p>Navigate to the <a href="https://www.iucnredlist.org/" target="_blank" rel="noopener">IUCN red list</a> website.
You will need to create an account to download data.</p>
<ul>
<li>Click on <code>Advanced</code></li>
<li>On the left tool bar, click on <code>Taxonomy</code></li>
<li>Click the appropriate check boxes/arrows to narrow the selection down to your group</li>
<li>Download the <code>Search Results</code></li>
</ul>
<p>Your download will be a zip file called <code>redlist_species_data_</code> followed by a long list of numbers and letters. If you are using a Mac the zip file will unzip when you double click on it. If you are using Windows, you must extract the files first.</p>
<p><strong>IMPORTANT</strong>: Tell R which working directory (i.e., folder) you would like to work from.</p>
<ul>
<li>Click on <code>Session</code> in the RStudio tool bar at the top of your screen</li>
<li>Click on <code>Set Working Directory</code> -&gt; <code>Choose Directory</code></li>
<li>Find your <code>phylogatr-results</code> folder and choose it
** for purposes of this exercise we renamed our folder <code>phylogatr-results-molluscs</code> since these are the data we downloaded and we didn’t want it to get confused with other <em>phylogatR</em> downloads</li>
<li>put your IUCN data in your <code>phylogatr-results</code> folder</li>
</ul>
<p><strong>The following examples will be from the Phylum Mollusca. Your plots should look similar but they won’t be exactly the same, if you downloaded data from another taxonomic group.</strong></p>
<p>Since all these packages will be used, load them all now. You might get some warning messages, but these are usually ok, so let’s ignore them. Make all the packages are installed first, and if R asks if you’d like to update other packages, go ahead and click yes.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="population-genetics-and-conservation.html#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(adegenet)</span>
<span id="cb1-2"><a href="population-genetics-and-conservation.html#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggplot2)</span>
<span id="cb1-3"><a href="population-genetics-and-conservation.html#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyr)</span>
<span id="cb1-4"><a href="population-genetics-and-conservation.html#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tools)</span>
<span id="cb1-5"><a href="population-genetics-and-conservation.html#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(pegas)</span>
<span id="cb1-6"><a href="population-genetics-and-conservation.html#cb1-6" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(stringr)</span></code></pre></div>
<div id="import-iucn-data" class="section level4 hasAnchor" number="5.3.2.1">
<h4><span class="header-section-number">5.3.2.1</span> Import IUCN data<a href="population-genetics-and-conservation.html#import-iucn-data" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>This imports the csv file and creates an R object called assessments. Remember we moved all the IUCN data into our phylogatr-results folder.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="population-genetics-and-conservation.html#cb2-1" aria-hidden="true" tabindex="-1"></a>assessments <span class="ot">&lt;-</span> <span class="fu">read.csv</span>(<span class="st">&quot;phylogatr-results-molluscs/assessments.csv&quot;</span>)</span></code></pre></div>
<p>In the upper right block of RStudio you will see an <code>Environment</code> tab. Click on it if it is not already chosen. Under <code>Global Environment</code> you will see the objects from your R session. You can click on the <code>assessments</code> object to view it.</p>
<p>You can use R to list the column headers.</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="population-genetics-and-conservation.html#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(assessments)</span></code></pre></div>
<pre><code>##  [1] &quot;assessmentId&quot;             &quot;internalTaxonId&quot;         
##  [3] &quot;scientificName&quot;           &quot;redlistCategory&quot;         
##  [5] &quot;redlistCriteria&quot;          &quot;yearPublished&quot;           
##  [7] &quot;assessmentDate&quot;           &quot;criteriaVersion&quot;         
##  [9] &quot;language&quot;                 &quot;rationale&quot;               
## [11] &quot;habitat&quot;                  &quot;threats&quot;                 
## [13] &quot;population&quot;               &quot;populationTrend&quot;         
## [15] &quot;range&quot;                    &quot;useTrade&quot;                
## [17] &quot;systems&quot;                  &quot;conservationActions&quot;     
## [19] &quot;realm&quot;                    &quot;yearLastSeen&quot;            
## [21] &quot;possiblyExtinct&quot;          &quot;possiblyExtinctInTheWild&quot;
## [23] &quot;scopes&quot;</code></pre>
<p>We are the most interested in columns 3 and 4, so lets clean it up a little so it is easier to use later.</p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="population-genetics-and-conservation.html#cb5-1" aria-hidden="true" tabindex="-1"></a>assessments <span class="ot">&lt;-</span> assessments[,<span class="dv">3</span><span class="sc">:</span><span class="dv">4</span>]</span></code></pre></div>
<p>Create a plot to look at the distribution of red listings in this group.</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="population-genetics-and-conservation.html#cb6-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(assessments, <span class="fu">aes</span>(<span class="at">x=</span>redlistCategory, <span class="at">fill=</span>redlistCategory)) <span class="sc">+</span> <span class="fu">geom_bar</span>() <span class="sc">+</span> <span class="fu">scale_fill_brewer</span>(<span class="at">palette =</span> <span class="st">&quot;Paired&quot;</span>) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">axis.title.x=</span><span class="fu">element_blank</span>(), <span class="at">legend.position =</span> <span class="st">&quot;none&quot;</span>) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">25</span>))</span></code></pre></div>
<p><img src="images/title-iucn-1.png" width="672" /></p>
<p>We can also view these in a table.</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="population-genetics-and-conservation.html#cb7-1" aria-hidden="true" tabindex="-1"></a><span class="fu">table</span>(assessments<span class="sc">$</span>redlistCategory)</span></code></pre></div>
<pre><code>## 
##             Critically Endangered                    Data Deficient 
##                               682                              2229 
##                        Endangered                           Extinct 
##                               586                               299 
##               Extinct in the Wild                     Least Concern 
##                                14                              3291 
## Lower Risk/conservation dependent          Lower Risk/least concern 
##                                 4                                17 
##        Lower Risk/near threatened                   Near Threatened 
##                               118                               575 
##                        Vulnerable 
##                              1032</code></pre>
<p><strong>What do all the categories mean?</strong> Hint: Your dataset might not contain all these categories. Some of them are older categories that need to be updated in the database.</p>
</div>
<div id="import-and-summarize-sequence-data" class="section level4 hasAnchor" number="5.3.2.2">
<h4><span class="header-section-number">5.3.2.2</span> Import and summarize sequence data<a href="population-genetics-and-conservation.html#import-and-summarize-sequence-data" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>To get the information we need from all the sequence alignments from our <em>phylogatR</em> download, we’ll first create a list of all the sequence alignments and put them in an object called <code>myfiles</code>.</p>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="population-genetics-and-conservation.html#cb9-1" aria-hidden="true" tabindex="-1"></a>myfiles <span class="ot">&lt;-</span> <span class="fu">list.files</span>(<span class="at">pattern =</span> <span class="st">&quot;.afa&quot;</span>, <span class="at">full.names=</span><span class="cn">TRUE</span>, <span class="at">recursive =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
<p>Here we loop through the list stored in <code>myfiles</code> to calculate genetic diversity for each sequence alignment. All data will be written to a csv file called <code>phylogatr-iucn.csv</code> and stored in the current working directory. This step will take a few minutes to run, and will take longer if you have a lot of species.</p>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="population-genetics-and-conservation.html#cb10-1" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (f <span class="cf">in</span> myfiles) {</span>
<span id="cb10-2"><a href="population-genetics-and-conservation.html#cb10-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-3"><a href="population-genetics-and-conservation.html#cb10-3" aria-hidden="true" tabindex="-1"></a>  <span class="co">#get species and gene name</span></span>
<span id="cb10-4"><a href="population-genetics-and-conservation.html#cb10-4" aria-hidden="true" tabindex="-1"></a>  sp <span class="ot">&lt;-</span> <span class="fu">basename</span>(f)</span>
<span id="cb10-5"><a href="population-genetics-and-conservation.html#cb10-5" aria-hidden="true" tabindex="-1"></a>  sp <span class="ot">&lt;-</span> <span class="fu">file_path_sans_ext</span>(sp)</span>
<span id="cb10-6"><a href="population-genetics-and-conservation.html#cb10-6" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb10-7"><a href="population-genetics-and-conservation.html#cb10-7" aria-hidden="true" tabindex="-1"></a>  <span class="co">#read in fasta file</span></span>
<span id="cb10-8"><a href="population-genetics-and-conservation.html#cb10-8" aria-hidden="true" tabindex="-1"></a>  seq <span class="ot">&lt;-</span> <span class="fu">fasta2DNAbin</span>(f, <span class="at">quiet=</span><span class="cn">TRUE</span>)</span>
<span id="cb10-9"><a href="population-genetics-and-conservation.html#cb10-9" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb10-10"><a href="population-genetics-and-conservation.html#cb10-10" aria-hidden="true" tabindex="-1"></a>  <span class="co">#get number of sequences</span></span>
<span id="cb10-11"><a href="population-genetics-and-conservation.html#cb10-11" aria-hidden="true" tabindex="-1"></a>  n <span class="ot">&lt;-</span> <span class="fu">nrow</span>(seq)</span>
<span id="cb10-12"><a href="population-genetics-and-conservation.html#cb10-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-13"><a href="population-genetics-and-conservation.html#cb10-13" aria-hidden="true" tabindex="-1"></a>  <span class="co">#calculate nucleotide diversity</span></span>
<span id="cb10-14"><a href="population-genetics-and-conservation.html#cb10-14" aria-hidden="true" tabindex="-1"></a>  pi <span class="ot">&lt;-</span> <span class="fu">nuc.div</span>(seq)</span>
<span id="cb10-15"><a href="population-genetics-and-conservation.html#cb10-15" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb10-16"><a href="population-genetics-and-conservation.html#cb10-16" aria-hidden="true" tabindex="-1"></a>  <span class="co">#write data to file</span></span>
<span id="cb10-17"><a href="population-genetics-and-conservation.html#cb10-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">write.table</span>(<span class="fu">data.frame</span>(sp, pi, n), <span class="at">file=</span><span class="st">&quot;phylogatr-iucn.csv&quot;</span>, <span class="at">quote=</span><span class="cn">FALSE</span>, <span class="at">row.names=</span><span class="cn">FALSE</span>, <span class="at">col.names=</span><span class="sc">!</span><span class="fu">file.exists</span>(<span class="st">&quot;phylogatr-data-iucn.csv&quot;</span>), <span class="at">append=</span><span class="cn">TRUE</span>, <span class="at">sep=</span><span class="st">&quot;,&quot;</span>)</span>
<span id="cb10-18"><a href="population-genetics-and-conservation.html#cb10-18" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
</div>
<div id="merge-iucn-and-genetic-data-results" class="section level4 hasAnchor" number="5.3.2.3">
<h4><span class="header-section-number">5.3.2.3</span> Merge IUCN and genetic data results<a href="population-genetics-and-conservation.html#merge-iucn-and-genetic-data-results" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>First, import the output from the previous step, then look at the column headers.</p>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb11-1"><a href="population-genetics-and-conservation.html#cb11-1" aria-hidden="true" tabindex="-1"></a>genetics <span class="ot">&lt;-</span> <span class="fu">read.csv</span>(<span class="st">&quot;phylogatr-results-molluscs/phylogatr-iucn.csv&quot;</span>)</span>
<span id="cb11-2"><a href="population-genetics-and-conservation.html#cb11-2" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(genetics)</span></code></pre></div>
<pre><code>## [1] &quot;sp&quot; &quot;pi&quot; &quot;n&quot;</code></pre>
<p>sp = species</p>
<p>pi = nuceotide diversity</p>
<p>n = the number of sequences</p>
<p>Now, we can merge the data from both databases. We have two objects in our R environment. 1) <code>assessments</code> has the IUCN data and 2) <code>genetics</code> has the calculated genetic diversity.</p>
<p>We’ll need to reformat the species names from <em>phylogatR</em> so they can match those found on the IUCN database. The species names will be merged and if there is a species that has data from both databases, it will be saved in a new R object.</p>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="population-genetics-and-conservation.html#cb13-1" aria-hidden="true" tabindex="-1"></a><span class="co">#reformat phylogatR species names</span></span>
<span id="cb13-2"><a href="population-genetics-and-conservation.html#cb13-2" aria-hidden="true" tabindex="-1"></a>genetics<span class="sc">$</span>sp <span class="ot">&lt;-</span> <span class="fu">sub</span>(<span class="st">&quot;-&quot;</span>, <span class="st">&quot; &quot;</span> ,genetics<span class="sc">$</span>sp)</span>
<span id="cb13-3"><a href="population-genetics-and-conservation.html#cb13-3" aria-hidden="true" tabindex="-1"></a>genetics <span class="ot">&lt;-</span> <span class="fu">separate</span>(genetics, sp, <span class="at">into=</span><span class="fu">c</span>(<span class="st">&quot;scientificName&quot;</span>, <span class="st">&quot;gene&quot;</span>), <span class="at">sep=</span><span class="st">&quot;-&quot;</span>)</span>
<span id="cb13-4"><a href="population-genetics-and-conservation.html#cb13-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-5"><a href="population-genetics-and-conservation.html#cb13-5" aria-hidden="true" tabindex="-1"></a><span class="co">#merge</span></span>
<span id="cb13-6"><a href="population-genetics-and-conservation.html#cb13-6" aria-hidden="true" tabindex="-1"></a>dataset <span class="ot">&lt;-</span> <span class="fu">merge</span>(assessments, genetics, <span class="at">by =</span> <span class="st">&quot;scientificName&quot;</span>)</span>
<span id="cb13-7"><a href="population-genetics-and-conservation.html#cb13-7" aria-hidden="true" tabindex="-1"></a><span class="fu">names</span>(dataset)</span></code></pre></div>
<pre><code>## [1] &quot;scientificName&quot;  &quot;redlistCategory&quot; &quot;gene&quot;            &quot;pi&quot;             
## [5] &quot;n&quot;</code></pre>
<p>Remember, you can click on objects in your R Environment to view them.</p>
<p>This is what the first six lines of the mollusc data look like.</p>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb15-1"><a href="population-genetics-and-conservation.html#cb15-1" aria-hidden="true" tabindex="-1"></a><span class="fu">head</span>(dataset)</span></code></pre></div>
<pre><code>##         scientificName redlistCategory gene          pi  n
## 1 Alasmidonta varicosa      Vulnerable  COI 0.000000000  3
## 2    Alloteuthis media  Data Deficient  COI 0.004959925 21
## 3 Alloteuthis subulata  Data Deficient  COI 0.003960396  3
## 4        Alopia livida   Least Concern  COI 0.014712006 11
## 5       Alopia monacha   Least Concern  COI 0.020174482  8
## 6      Amblema plicata   Least Concern  COI 0.006259781  3</code></pre>
</div>
<div id="data-analysis" class="section level4 hasAnchor" number="5.3.2.4">
<h4><span class="header-section-number">5.3.2.4</span> Data analysis<a href="population-genetics-and-conservation.html#data-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>First, we’ll make a plot to visualize our data. This is a plot of mean nucleotide diversity found within each red list category.</p>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb17-1"><a href="population-genetics-and-conservation.html#cb17-1" aria-hidden="true" tabindex="-1"></a><span class="co">#calculate mean nucleotide diversity for each group</span></span>
<span id="cb17-2"><a href="population-genetics-and-conservation.html#cb17-2" aria-hidden="true" tabindex="-1"></a>mean_pi <span class="ot">&lt;-</span> <span class="fu">aggregate</span>(dataset<span class="sc">$</span>pi, <span class="fu">list</span>(dataset<span class="sc">$</span>redlistCategory), mean, <span class="at">na.rm=</span>T)</span>
<span id="cb17-3"><a href="population-genetics-and-conservation.html#cb17-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-4"><a href="population-genetics-and-conservation.html#cb17-4" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(mean_pi, <span class="fu">aes</span>(<span class="at">x =</span> Group<span class="fl">.1</span>, <span class="at">y =</span> x, <span class="at">fill =</span> Group<span class="fl">.1</span>)) <span class="sc">+</span> <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">&quot;identity&quot;</span>) <span class="sc">+</span> <span class="fu">scale_fill_brewer</span>(<span class="at">palette =</span> <span class="st">&quot;Paired&quot;</span>) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">axis.title.x=</span><span class="fu">element_blank</span>(), <span class="at">legend.position =</span> <span class="st">&quot;none&quot;</span>) <span class="sc">+</span> <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">25</span>)) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">&quot;nucleotide diversity (pi)&quot;</span>)</span></code></pre></div>
<p><img src="images/title-pi-1.png" width="672" /></p>
<p>Now we can test whether there is a significant difference in nucleotide diversity between those species that are threatened and those that are not. A t-test compares two groups so first we’ll group categories into threatened and not-threatened and call them as such. We’ll remove the data deficient species (notice we are overwriting the <code>dataset</code> object and it will no longer contain the data deficient assessments).</p>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="population-genetics-and-conservation.html#cb18-1" aria-hidden="true" tabindex="-1"></a>dataset<span class="ot">&lt;-</span>dataset[<span class="sc">!</span>(dataset<span class="sc">$</span>redlistCategory<span class="sc">==</span><span class="st">&quot;Data Deficient&quot;</span>),]</span>
<span id="cb18-2"><a href="population-genetics-and-conservation.html#cb18-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb18-3"><a href="population-genetics-and-conservation.html#cb18-3" aria-hidden="true" tabindex="-1"></a>notthreatened<span class="ot">&lt;-</span><span class="fu">c</span>(<span class="st">&#39;Near Threatened&#39;</span>, <span class="st">&#39;Least Concern&#39;</span>, <span class="st">&#39;Lower Risk/least concern&#39;</span>, <span class="st">&#39;Lower Risk/near threatened&#39;</span>)</span>
<span id="cb18-4"><a href="population-genetics-and-conservation.html#cb18-4" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (i <span class="cf">in</span> <span class="dv">1</span><span class="sc">:</span><span class="fu">nrow</span>(dataset)) {</span>
<span id="cb18-5"><a href="population-genetics-and-conservation.html#cb18-5" aria-hidden="true" tabindex="-1"></a>  <span class="cf">if</span> (dataset<span class="sc">$</span>redlistCategory[i] <span class="sc">%in%</span> notthreatened) {</span>
<span id="cb18-6"><a href="population-genetics-and-conservation.html#cb18-6" aria-hidden="true" tabindex="-1"></a>    dataset<span class="sc">$</span>category[i] <span class="ot">&lt;-</span><span class="st">&#39;notthreatened&#39;</span> </span>
<span id="cb18-7"><a href="population-genetics-and-conservation.html#cb18-7" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb18-8"><a href="population-genetics-and-conservation.html#cb18-8" aria-hidden="true" tabindex="-1"></a>  <span class="cf">else</span> {</span>
<span id="cb18-9"><a href="population-genetics-and-conservation.html#cb18-9" aria-hidden="true" tabindex="-1"></a>    dataset<span class="sc">$</span>category[i] <span class="ot">&lt;-</span><span class="st">&#39;threatened&#39;</span></span>
<span id="cb18-10"><a href="population-genetics-and-conservation.html#cb18-10" aria-hidden="true" tabindex="-1"></a>   }</span>
<span id="cb18-11"><a href="population-genetics-and-conservation.html#cb18-11" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
<p>Compare the two groups.</p>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb19-1"><a href="population-genetics-and-conservation.html#cb19-1" aria-hidden="true" tabindex="-1"></a>threatened <span class="ot">&lt;-</span> dataset<span class="sc">$</span>pi[dataset<span class="sc">$</span>category<span class="sc">==</span><span class="st">&quot;threatened&quot;</span>]</span>
<span id="cb19-2"><a href="population-genetics-and-conservation.html#cb19-2" aria-hidden="true" tabindex="-1"></a>notthreatened <span class="ot">&lt;-</span> dataset<span class="sc">$</span>pi[dataset<span class="sc">$</span>category<span class="sc">==</span><span class="st">&quot;notthreatened&quot;</span>]</span>
<span id="cb19-3"><a href="population-genetics-and-conservation.html#cb19-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-4"><a href="population-genetics-and-conservation.html#cb19-4" aria-hidden="true" tabindex="-1"></a><span class="fu">t.test</span>(threatened, notthreatened)</span></code></pre></div>
<pre><code>## 
##  Welch Two Sample t-test
## 
## data:  threatened and notthreatened
## t = -2.2634, df = 50.678, p-value = 0.02793
## alternative hypothesis: true difference in means is not equal to 0
## 95 percent confidence interval:
##  -0.026176358 -0.001565606
## sample estimates:
##  mean of x  mean of y 
## 0.01434120 0.02821218</code></pre>
<p>In this case there is a significant difference in genetic diversity among those species that are listed as threatened vs. not threatened.</p>
</div>
</div>
</div>
<div id="click-and-play" class="section level2 hasAnchor" number="5.4">
<h2><span class="header-section-number">5.4</span> Click and play!<a href="population-genetics-and-conservation.html#click-and-play" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<iframe src="https://djparsons.shinyapps.io/App-1-conservation-DEMO/" width="750" height="700"></iframe>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="biodiversity-databases.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="species-limits.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
