<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>7 Predictive Phylogeography | PhylogatR</title>
  <meta name="description" content="" />
  <meta name="generator" content="bookdown 0.26.3 and GitBook 2.6.7" />

  <meta property="og:title" content="7 Predictive Phylogeography | PhylogatR" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="7 Predictive Phylogeography | PhylogatR" />
  
  
  

<meta name="author" content="Bryan Carstens, Sydney Decker, Anahi Espindola, Sarah Foltz, Abbi Green, Danielle Parsons, Tara Pelletier, Jordan Satler" />


<meta name="date" content="2022-08-03" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="species-limits.html"/>
<link rel="next" href="references.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>



  
  
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Background</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#how-to-use-this-book"><i class="fa fa-check"></i><b>1.1</b> How to use this book</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#prerequisites-if-using-r"><i class="fa fa-check"></i><b>1.2</b> Prerequisites if using R</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="index.html"><a href="index.html#r-and-rstudio"><i class="fa fa-check"></i><b>1.2.1</b> R and RStudio</a></li>
<li class="chapter" data-level="1.2.2" data-path="index.html"><a href="index.html#r-packages"><i class="fa fa-check"></i><b>1.2.2</b> R packages</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>2</b> Introduction</a>
<ul>
<li class="chapter" data-level="2.1" data-path="introduction.html"><a href="introduction.html#data-in-the-biological-sciences"><i class="fa fa-check"></i><b>2.1</b> Data in the biological sciences</a></li>
<li class="chapter" data-level="2.2" data-path="introduction.html"><a href="introduction.html#why-species"><i class="fa fa-check"></i><b>2.2</b> Why species?</a></li>
<li class="chapter" data-level="2.3" data-path="introduction.html"><a href="introduction.html#the-scientific-method"><i class="fa fa-check"></i><b>2.3</b> The Scientific Method</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="dna.html"><a href="dna.html"><i class="fa fa-check"></i><b>3</b> DNA</a>
<ul>
<li class="chapter" data-level="3.1" data-path="dna.html"><a href="dna.html#the-genetic-code"><i class="fa fa-check"></i><b>3.1</b> The genetic code</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="dna.html"><a href="dna.html#practice-translate-the-following-dna-sequence-into-amino-acids"><i class="fa fa-check"></i><b>3.1.1</b> Practice: Translate the following DNA sequence into amino acids</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="dna.html"><a href="dna.html#mutation"><i class="fa fa-check"></i><b>3.2</b> Mutation</a></li>
<li class="chapter" data-level="3.3" data-path="dna.html"><a href="dna.html#synonymous-versus-non-synonymous-genetic-variation"><i class="fa fa-check"></i><b>3.3</b> Synonymous versus non-synonymous genetic variation</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="dna.html"><a href="dna.html#practice-introduce-a-mutation"><i class="fa fa-check"></i><b>3.3.1</b> Practice: Introduce a mutation</a></li>
<li class="chapter" data-level="3.3.2" data-path="dna.html"><a href="dna.html#practice-introduce-an-insertion"><i class="fa fa-check"></i><b>3.3.2</b> Practice: Introduce an insertion</a></li>
<li class="chapter" data-level="3.3.3" data-path="dna.html"><a href="dna.html#practice-calculate-the-proportion-of-single-base-pair-mutations-that-are-synonymous"><i class="fa fa-check"></i><b>3.3.3</b> Practice: Calculate the proportion of single base pair mutations that are synonymous</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="dna.html"><a href="dna.html#probabilities-snps-and-genetic-variation"><i class="fa fa-check"></i><b>3.4</b> Probabilities, SNPs, and genetic variation</a></li>
<li class="chapter" data-level="3.5" data-path="dna.html"><a href="dna.html#modeling-sequence-variation"><i class="fa fa-check"></i><b>3.5</b> Modeling sequence variation</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="dna.html"><a href="dna.html#practice-estimate-base-pair-composition"><i class="fa fa-check"></i><b>3.5.1</b> Practice: Estimate base pair composition</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="dna.html"><a href="dna.html#organellar-genomes-and-barcoding-genes"><i class="fa fa-check"></i><b>3.6</b> Organellar genomes and barcoding genes</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html"><i class="fa fa-check"></i><b>4</b> Biodiversity Databases</a>
<ul>
<li class="chapter" data-level="4.1" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#global-biodiversity-information-facility-gbif"><i class="fa fa-check"></i><b>4.1</b> Global Biodiversity Information Facility (GBIF)</a></li>
<li class="chapter" data-level="4.2" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#genbank"><i class="fa fa-check"></i><b>4.2</b> GenBank</a></li>
<li class="chapter" data-level="4.3" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#bold"><i class="fa fa-check"></i><b>4.3</b> BOLD</a></li>
<li class="chapter" data-level="4.4" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#phylogatr"><i class="fa fa-check"></i><b>4.4</b> <em>PhylogatR</em></a></li>
<li class="chapter" data-level="4.5" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#using-phylogatr"><i class="fa fa-check"></i><b>4.5</b> Using <em>phylogatR</em></a></li>
<li class="chapter" data-level="4.6" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#practice"><i class="fa fa-check"></i><b>4.6</b> Practice</a>
<ul>
<li class="chapter" data-level="4.6.1" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#downloading-data-from-phylogatr"><i class="fa fa-check"></i><b>4.6.1</b> Downloading data from <em>phylogatR</em></a></li>
<li class="chapter" data-level="4.6.2" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#check-for-outliers"><i class="fa fa-check"></i><b>4.6.2</b> Check for outliers</a></li>
<li class="chapter" data-level="4.6.3" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#check-sequence-alignments"><i class="fa fa-check"></i><b>4.6.3</b> Check sequence alignments</a></li>
<li class="chapter" data-level="4.6.4" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#visualizing-geographic-coordinates"><i class="fa fa-check"></i><b>4.6.4</b> Visualizing geographic coordinates</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html"><i class="fa fa-check"></i><b>5</b> Population genetics and conservation</a>
<ul>
<li class="chapter" data-level="5.1" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#loss-of-genetic-diversity"><i class="fa fa-check"></i><b>5.1</b> Loss of genetic diversity</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#small-population-size"><i class="fa fa-check"></i><b>5.1.1</b> Small population size</a></li>
<li class="chapter" data-level="5.1.2" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#genetic-drift"><i class="fa fa-check"></i><b>5.1.2</b> Genetic drift</a></li>
<li class="chapter" data-level="5.1.3" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#gene-flow"><i class="fa fa-check"></i><b>5.1.3</b> Gene flow</a></li>
<li class="chapter" data-level="5.1.4" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#inbreeding"><i class="fa fa-check"></i><b>5.1.4</b> Inbreeding</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#units-of-conservation"><i class="fa fa-check"></i><b>5.2</b> Units of conservation</a></li>
<li class="chapter" data-level="5.3" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#practice-1"><i class="fa fa-check"></i><b>5.3</b> Practice</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#learning-objectives-1"><i class="fa fa-check"></i><b>5.3.1</b> Learning objectives:</a></li>
<li class="chapter" data-level="5.3.2" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#data"><i class="fa fa-check"></i><b>5.3.2</b> Data</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#click-and-play"><i class="fa fa-check"></i><b>5.4</b> Click and play!</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="species-limits.html"><a href="species-limits.html"><i class="fa fa-check"></i><b>6</b> Species limits</a>
<ul>
<li class="chapter" data-level="6.1" data-path="species-limits.html"><a href="species-limits.html#cryptic-genetic-diversity"><i class="fa fa-check"></i><b>6.1</b> Cryptic genetic diversity</a></li>
<li class="chapter" data-level="6.2" data-path="species-limits.html"><a href="species-limits.html#genetic-clustering"><i class="fa fa-check"></i><b>6.2</b> Genetic clustering</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="species-limits.html"><a href="species-limits.html#structure"><i class="fa fa-check"></i><b>6.2.1</b> Structure</a></li>
<li class="chapter" data-level="6.2.2" data-path="species-limits.html"><a href="species-limits.html#structurama"><i class="fa fa-check"></i><b>6.2.2</b> Structurama</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="species-limits.html"><a href="species-limits.html#tree-based-methods"><i class="fa fa-check"></i><b>6.3</b> Tree-based methods</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="species-limits.html"><a href="species-limits.html#general-mixed-yule-coalescent-gmyc"><i class="fa fa-check"></i><b>6.3.1</b> General mixed Yule Coalescent (GMYC)</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="species-limits.html"><a href="species-limits.html#practice-2"><i class="fa fa-check"></i><b>6.4</b> Practice</a>
<ul>
<li class="chapter" data-level="6.4.1" data-path="species-limits.html"><a href="species-limits.html#data-preparation"><i class="fa fa-check"></i><b>6.4.1</b> Data Preparation</a></li>
<li class="chapter" data-level="6.4.2" data-path="species-limits.html"><a href="species-limits.html#generate-a-phylogenetic-tree"><i class="fa fa-check"></i><b>6.4.2</b> Generate a Phylogenetic Tree</a></li>
<li class="chapter" data-level="6.4.3" data-path="species-limits.html"><a href="species-limits.html#create-a-summary-tree"><i class="fa fa-check"></i><b>6.4.3</b> Create a summary tree</a></li>
<li class="chapter" data-level="6.4.4" data-path="species-limits.html"><a href="species-limits.html#species-delimitation"><i class="fa fa-check"></i><b>6.4.4</b> Species Delimitation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html"><i class="fa fa-check"></i><b>7</b> Predictive Phylogeography</a>
<ul>
<li class="chapter" data-level="7.0.1" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#trait-data-and-response-variables"><i class="fa fa-check"></i><b>7.0.1</b> Trait data and response variables</a></li>
<li class="chapter" data-level="7.1" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#machine-learning-and-random-forest"><i class="fa fa-check"></i><b>7.1</b> Machine learning and random forest</a></li>
<li class="chapter" data-level="7.2" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#real-life-example"><i class="fa fa-check"></i><b>7.2</b> Real life example</a></li>
<li class="chapter" data-level="7.3" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#practice-3"><i class="fa fa-check"></i><b>7.3</b> Practice</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>8</b> References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"><img src="Segoe1.png" title="fig:" style="width:75.0%" alt="PhylogatR" /></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="predictive-phylogeography" class="section level1 hasAnchor" number="7">
<h1><span class="header-section-number">7</span> Predictive Phylogeography<a href="predictive-phylogeography.html#predictive-phylogeography" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>THIS PAGE IS UNDER CONSTRUCTION</p>
<p>Generally, and historically, phylogeography has been defined as the study of the spatial distribution of genetic lineages of species. This field in evolutionary biology has more recently evolved to identify specific events that have affected the spatial history of species, and has expanded from species-based studies to community and comparative approaches.</p>
<p><img src="phylogeography2.jpg" style="width:100.0%" /></p>
  <p><font size="1"><em> Phylogeography is the study of the spatial distribution of genetic lineages of species. Figure made by Danielle J. Parsons. </em></font></p>
<p>Community and comparative phylogeography seeks to understand the extent to which environmental changes have affected whole communities of organisms. For instance, this field seeks to ask whether species that share ecological characteristics are likely to respond similarly to specific changes in the environment in which they co-occur (e.g., temperature shifts, the arrival of new species to the community, geological events). Indeed, this is something that has been identified in several systems, in particular those involving strongly spatially restricted species such as those occurring in isolated ecosystems, or species that are involved in very specialized interactions such as one-to-one pollination systems. If it is possible to find specific shared spatial histories for species that co-occur or that are tightly associated, it follows that there are some rules “guiding” whether species share spatial histories or not. If we can identify the nature of those rules, then we can use them to predict what species will share histories usign data related to the rules, and even in the absence of information about their histories. This is exactly what the field of predictive phylogeography seeks to do: identify general ecological and/or environmental traits shared by species that display similar spatial histories, and use those traits to predict the spatial genetic characteristics of species that are yet unstudied.</p>
<div id="trait-data-and-response-variables" class="section level3 hasAnchor" number="7.0.1">
<h3><span class="header-section-number">7.0.1</span> Trait data and response variables<a href="predictive-phylogeography.html#trait-data-and-response-variables" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>In order to do this, predictive phylogeography uses, on the one hand, traits associated with the ecology or biology of the species in question. This data will be used to seek to identify those “rules” that drive the spatial genetic characteristics of the studied taxa. On the other hand, once the rules have been identified, they are applied to new unstudied taxa, for which the ecological and biological data is present, but the genetic data is absent. The application of these rules on new unstudied species allow to make predictions about their phylogeographic traits. If we think about this from a model perspective, we see that we have trait (explanatory) and response variables that are used to create a model, measure its accuracy, and then use it to make predictions. What are then these variables?</p>
<p><strong>Trait (explanatory) data:</strong> the trait data in these approaches is often environmental, biological, and/or ecological. Environmental data can be represented by climatic, soil, or geologic data. Biological data can refer to characteristics intrinsic to the organisms of study. For example, dispersal ability, mating type, taxonomic group, brood size, habit, etc. Finally, ecological data may be associated to information related to interactions the organisms are involved in, their trophic level, etc.</p>
<p><strong>Response variables:</strong> the variables used as response ones have been different in different studies, but they often involved some measure of genetic characteristics of the taxa in question. For instance, presence of specific levels of genetic divergence within the taxon, measures of genetic isolation across the landscape, amounts of genetic diversity, number of genetic clusters.</p>
</div>
<div id="machine-learning-and-random-forest" class="section level2 hasAnchor" number="7.1">
<h2><span class="header-section-number">7.1</span> Machine learning and random forest<a href="predictive-phylogeography.html#machine-learning-and-random-forest" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Machine learning methods are a family of methods in which an algorithm (a sort of mathematical model) is created to explore characteristics of a dataset. How does it do this? Let’s use a simple example. Imagine that we have a mixed bag of apples and oranges, and we would like to create a method that would tell us every time that we take a fruit out of that bag, what that fruit is. Machine learning methods can help us in this task. Apples are indeed different from oranges; they have different textures, shapes, flavors… A machine learning approach of doing this is to go to the store and get a lot of apples and oranges. For each apple and each orange, we can then collect data on all those characteristics that are intrinsic to apples and oranges, and we would end up with a table that would look like this:</p>
<p>
  <table style="width:100%">
  <tr>
  	<th>Fruit</th>
    <th>Texture</th>
    <th>Shape</th>
    <th>Flavor</th>
    <th>Color</th>
  </tr>
  <tr>
  	<td>Orange</td>
    <td>Rugose</td>
    <td>Spherical</td>
    <td>Citrus</td>
    <td>Orange</td>
  </tr>
  <tr>
  	<td>Apple</td>
    <td>Slick</td>
    <td>Conical</td>
    <td>Sweet</td>
    <td>Red</td>
  </tr>
  <tr>
  	<td>Apple</td>
    <td>Slick</td>
    <td>Conical</td>
    <td>Sweet</td>
    <td>Green</td>
  </tr>
  <tr>
  	<td>Orange</td>
    <td>Rugose</td>
    <td>Spherical</td>
    <td>Citrus</td>
    <td>Orange</td>
  </tr>
  <tr>
  	<td>Apple</td>
    <td>Slick</td>
    <td>Spherical</td>
    <td>Sweet</td>
    <td>Red</td>
  </tr>
  <tr>
  	<td>Orange</td>
    <td>Rugose</td>
    <td>Spherical</td>
    <td>Citrus</td>
    <td>Orange</td>
  </tr>
  <tr>
  	<td>Orange</td>
    <td>Rugose</td>
    <td>Spherical</td>
    <td>Citrus</td>
    <td>Orange</td>
  </tr>
  <tr>
  	<td>Apple</td>
    <td>Slick</td>
    <td>Conical</td>
    <td>Sweet</td>
    <td>Yellow</td>
  </tr>
  <tr>
  	<td>Orange</td>
    <td>Rugose</td>
    <td>Spherical</td>
    <td>Citrus</td>
    <td>Orange</td>
  </tr>
  <tr>
  	<td>Orange</td>
    <td>Rugose</td>
    <td>Spherical</td>
    <td>Citrus</td>
    <td>Yellow</td>
  </tr>
  <tr>
  	<td>Apple</td>
    <td>Slick</td>
    <td>Spherical</td>
    <td>Sweet</td>
    <td>Red</td>
  </tr>
  <tr>
  	<td>Apple</td>
    <td>Slick</td>
    <td>Conical</td>
    <td>Sweet</td>
    <td>Green</td>
  </tr>
</table>
</p>
<p>After creating this table, we can give this to the algorithm, which will try to look at the dataset and analyze its characteristics, so as to identify some of the general features that differentiate apples from oranges. After testing different configurations, the algorithm will (hopefully) identify a combination of traits that differentiate the two fruits. The identification of these features by the algorithm is what is referred to as “learning”, since the method allows the algorithm to “learn” something about the structure of the dataset. Now that the algorithm has “learned” how to differentiate apples from oranges, we can expose it to a fruit that comes out of our mixed bag of apples and oranges, measure the traits of that fruit, run them through the algorithm and the algorithm will tell us what that fruit may be. Although this is a very silly example, this is the general way in which these methods work. Today, these types of methods are widely used in many aspects of our everyday lives: face recognition (e.g., biometric data associated with your IDs), prediction of interests based on past behavior on social media (e.g., the more you use your phone to navigate on the internet, the more ads appear to be targeting you individually), etc.</p>
<p>Among the many types of machine learning methods, we will talk about one called “random forest”. Random forest is a method that uses descriptor data of a dataset to create a classifier that can be then used to predict the identity of new objects. Basically, the method uses a dataset where all characteristics of a set of identified objects are known (this is called the “training data”) to “learn” about the traits and patterns of the data. The specific way that random forest uses to do this is creating a set of classification trees. These are bifurcating trees, where a condition is applied to each node (e.g., in the case of our apples and oranges, a node could ask “is the texture Rugose or Slick?”), which, when all conditions are used leads to a final classification of all objects. We say that this method is a random forest because i) it creates many trees (= a forest 😊), and ii) each tree is create with a subset of our training data that has been selected at random and they are added to tree in a random order. After the method creates a random forest, it can be exposed to new unknown objects. The traits associated to these objects will be run through all the trees that form the forest, and the prediction of each tree will be counted. The final prediction will be done based on the majority vote of all these trees. For example, and going back to our apples and oranges case, if we had constructed 100 trees, and a new unknown fruit was predicted by 80 trees to be an apple, and by 20 to be an orange, we would say that our random forest predicts the fruit as an apple with an 80% (80/100) chance.</p>
<p><img src="random_fruit2.jpg" style="width:100.0%" /></p>
  <p><font size="1"><em> A random forest predictive model, as applied to the apples and oranges example described above. Figure made by Danielle J. Parsons. </em></font></p>
</div>
<div id="real-life-example" class="section level2 hasAnchor" number="7.2">
<h2><span class="header-section-number">7.2</span> Real life example<a href="predictive-phylogeography.html#real-life-example" class="anchor-section" aria-label="Anchor link to header"></a></h2>
</div>
<div id="practice-3" class="section level2 hasAnchor" number="7.3">
<h2><span class="header-section-number">7.3</span> Practice<a href="predictive-phylogeography.html#practice-3" class="anchor-section" aria-label="Anchor link to header"></a></h2>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="species-limits.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="references.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
