<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>8 References | PhylogatR</title>
  <meta name="description" content="" />
  <meta name="generator" content="bookdown 0.26.3 and GitBook 2.6.7" />

  <meta property="og:title" content="8 References | PhylogatR" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="8 References | PhylogatR" />
  
  
  

<meta name="author" content="Bryan Carstens, Sydney Decker, Anahi Espindola, Sarah Foltz, Abbi Green, Danielle Parsons, Tara Pelletier, Jordan Satler" />


<meta name="date" content="2022-08-01" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="predictive-phylogeography.html"/>

<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>




</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Background</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#how-to-use-this-book"><i class="fa fa-check"></i><b>1.1</b> How to use this book</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#prerequisites-if-using-r"><i class="fa fa-check"></i><b>1.2</b> Prerequisites if using R</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="index.html"><a href="index.html#r-and-rstudio"><i class="fa fa-check"></i><b>1.2.1</b> R and RStudio</a></li>
<li class="chapter" data-level="1.2.2" data-path="index.html"><a href="index.html#r-packages"><i class="fa fa-check"></i><b>1.2.2</b> R packages</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>2</b> Introduction</a>
<ul>
<li class="chapter" data-level="2.1" data-path="introduction.html"><a href="introduction.html#data-in-the-biological-sciences"><i class="fa fa-check"></i><b>2.1</b> Data in the biological sciences</a></li>
<li class="chapter" data-level="2.2" data-path="introduction.html"><a href="introduction.html#why-species"><i class="fa fa-check"></i><b>2.2</b> Why species?</a></li>
<li class="chapter" data-level="2.3" data-path="introduction.html"><a href="introduction.html#the-scientific-method"><i class="fa fa-check"></i><b>2.3</b> The Scientific Method</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="dna.html"><a href="dna.html"><i class="fa fa-check"></i><b>3</b> DNA</a>
<ul>
<li class="chapter" data-level="3.1" data-path="dna.html"><a href="dna.html#the-genetic-code"><i class="fa fa-check"></i><b>3.1</b> The genetic code</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="dna.html"><a href="dna.html#practice-translate-the-following-dna-sequence-into-amino-acids"><i class="fa fa-check"></i><b>3.1.1</b> Practice: Translate the following DNA sequence into amino acids</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="dna.html"><a href="dna.html#mutation"><i class="fa fa-check"></i><b>3.2</b> Mutation</a></li>
<li class="chapter" data-level="3.3" data-path="dna.html"><a href="dna.html#synonymous-versus-non-synonymous-genetic-variation"><i class="fa fa-check"></i><b>3.3</b> Synonymous versus non-synonymous genetic variation</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="dna.html"><a href="dna.html#practice-introduce-a-mutation"><i class="fa fa-check"></i><b>3.3.1</b> Practice: Introduce a mutation</a></li>
<li class="chapter" data-level="3.3.2" data-path="dna.html"><a href="dna.html#practice-introduce-an-insertion"><i class="fa fa-check"></i><b>3.3.2</b> Practice: Introduce an insertion</a></li>
<li class="chapter" data-level="3.3.3" data-path="dna.html"><a href="dna.html#practice-calculate-the-proportion-of-single-base-pair-mutations-that-are-synonymous"><i class="fa fa-check"></i><b>3.3.3</b> Practice: Calculate the proportion of single base pair mutations that are synonymous</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="dna.html"><a href="dna.html#probabilities-snps-and-genetic-variation"><i class="fa fa-check"></i><b>3.4</b> Probabilities, SNPs, and genetic variation</a></li>
<li class="chapter" data-level="3.5" data-path="dna.html"><a href="dna.html#modeling-sequence-variation"><i class="fa fa-check"></i><b>3.5</b> Modeling sequence variation</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="dna.html"><a href="dna.html#practice-estimate-base-pair-composition"><i class="fa fa-check"></i><b>3.5.1</b> Practice: Estimate base pair composition</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="dna.html"><a href="dna.html#organellar-genomes-and-barcoding-genes"><i class="fa fa-check"></i><b>3.6</b> Organellar genomes and barcoding genes</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html"><i class="fa fa-check"></i><b>4</b> Biodiversity Databases</a>
<ul>
<li class="chapter" data-level="4.1" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#global-biodiversity-information-facility-gbif"><i class="fa fa-check"></i><b>4.1</b> Global Biodiversity Information Facility (GBIF)</a></li>
<li class="chapter" data-level="4.2" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#genbank"><i class="fa fa-check"></i><b>4.2</b> GenBank</a></li>
<li class="chapter" data-level="4.3" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#bold"><i class="fa fa-check"></i><b>4.3</b> BOLD</a></li>
<li class="chapter" data-level="4.4" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#phylogatr"><i class="fa fa-check"></i><b>4.4</b> <em>PhylogatR</em></a></li>
<li class="chapter" data-level="4.5" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#using-phylogatr"><i class="fa fa-check"></i><b>4.5</b> Using <em>phylogatR</em></a></li>
<li class="chapter" data-level="4.6" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#practice"><i class="fa fa-check"></i><b>4.6</b> Practice</a>
<ul>
<li class="chapter" data-level="4.6.1" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#downloading-data-from-phylogatr"><i class="fa fa-check"></i><b>4.6.1</b> Downloading data from <em>phylogatR</em></a></li>
<li class="chapter" data-level="4.6.2" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#check-for-outliers"><i class="fa fa-check"></i><b>4.6.2</b> Check for outliers</a></li>
<li class="chapter" data-level="4.6.3" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#check-sequence-alignments"><i class="fa fa-check"></i><b>4.6.3</b> Check sequence alignments</a></li>
<li class="chapter" data-level="4.6.4" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#visualizing-geographic-coordinates"><i class="fa fa-check"></i><b>4.6.4</b> Visualizing geographic coordinates</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html"><i class="fa fa-check"></i><b>5</b> Population genetics and conservation</a>
<ul>
<li class="chapter" data-level="5.1" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#loss-of-genetic-diversity"><i class="fa fa-check"></i><b>5.1</b> Loss of genetic diversity</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#small-population-size"><i class="fa fa-check"></i><b>5.1.1</b> Small population size</a></li>
<li class="chapter" data-level="5.1.2" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#genetic-drift"><i class="fa fa-check"></i><b>5.1.2</b> Genetic drift</a></li>
<li class="chapter" data-level="5.1.3" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#gene-flow"><i class="fa fa-check"></i><b>5.1.3</b> Gene flow</a></li>
<li class="chapter" data-level="5.1.4" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#inbreeding"><i class="fa fa-check"></i><b>5.1.4</b> Inbreeding</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#units-of-conservation"><i class="fa fa-check"></i><b>5.2</b> Units of conservation</a></li>
<li class="chapter" data-level="5.3" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#practice-1"><i class="fa fa-check"></i><b>5.3</b> Practice</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#learning-objectives-1"><i class="fa fa-check"></i><b>5.3.1</b> Learning objectives:</a></li>
<li class="chapter" data-level="5.3.2" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#data"><i class="fa fa-check"></i><b>5.3.2</b> Data</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#click-and-play"><i class="fa fa-check"></i><b>5.4</b> Click and play!</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="species-limits.html"><a href="species-limits.html"><i class="fa fa-check"></i><b>6</b> Species limits</a>
<ul>
<li class="chapter" data-level="6.1" data-path="species-limits.html"><a href="species-limits.html#cryptic-genetic-diversity"><i class="fa fa-check"></i><b>6.1</b> Cryptic genetic diversity</a></li>
<li class="chapter" data-level="6.2" data-path="species-limits.html"><a href="species-limits.html#genetic-clustering"><i class="fa fa-check"></i><b>6.2</b> Genetic clustering</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="species-limits.html"><a href="species-limits.html#structure"><i class="fa fa-check"></i><b>6.2.1</b> Structure</a></li>
<li class="chapter" data-level="6.2.2" data-path="species-limits.html"><a href="species-limits.html#structurama"><i class="fa fa-check"></i><b>6.2.2</b> Structurama</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="species-limits.html"><a href="species-limits.html#tree-based-methods"><i class="fa fa-check"></i><b>6.3</b> Tree-based methods</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="species-limits.html"><a href="species-limits.html#general-mixed-yule-coalescent-gmyc"><i class="fa fa-check"></i><b>6.3.1</b> General mixed Yule Coalescent (GMYC)</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="species-limits.html"><a href="species-limits.html#practice-2"><i class="fa fa-check"></i><b>6.4</b> Practice</a>
<ul>
<li class="chapter" data-level="6.4.1" data-path="species-limits.html"><a href="species-limits.html#data-preparation"><i class="fa fa-check"></i><b>6.4.1</b> Data Preparation</a></li>
<li class="chapter" data-level="6.4.2" data-path="species-limits.html"><a href="species-limits.html#generate-a-phylogenetic-tree"><i class="fa fa-check"></i><b>6.4.2</b> Generate a Phylogenetic Tree</a></li>
<li class="chapter" data-level="6.4.3" data-path="species-limits.html"><a href="species-limits.html#create-a-summary-tree"><i class="fa fa-check"></i><b>6.4.3</b> Create a summary tree</a></li>
<li class="chapter" data-level="6.4.4" data-path="species-limits.html"><a href="species-limits.html#species-delimitation"><i class="fa fa-check"></i><b>6.4.4</b> Species Delimitation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html"><i class="fa fa-check"></i><b>7</b> Predictive Phylogeography</a>
<ul>
<li class="chapter" data-level="7.0.1" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#trait-data-and-response-variables"><i class="fa fa-check"></i><b>7.0.1</b> Trait data and response variables</a></li>
<li class="chapter" data-level="7.1" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#machine-learning-and-random-forest"><i class="fa fa-check"></i><b>7.1</b> Machine learning and random forest</a></li>
<li class="chapter" data-level="7.2" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#real-life-example"><i class="fa fa-check"></i><b>7.2</b> Real life example</a></li>
<li class="chapter" data-level="7.3" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#practice-3"><i class="fa fa-check"></i><b>7.3</b> Practice</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>8</b> References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"><img src="Segoe1.png" title="fig:" style="width:75.0%" alt="PhylogatR" /></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1 hasAnchor" number="8">
<h1><span class="header-section-number">8</span> References<a href="references.html#references" class="anchor-section" aria-label="Anchor link to header"></a></h1>

<p> Benson, D. A. <em>et al.</em> GenBank. <em>Nucleic Acids Research</em> <strong>46</strong>, D41–D47 (2018). </p>
  
<p> Causey, D. <em>et al.</em> Museum Collections and Taxonomy. <em>Science</em> <strong>305</strong>, 1106–1107 (2004). </p>
  
<p> Coimbra, R. T. F. <em>et al.</em> Whole-genome analysis of giraffe supports four distinct species. <em>Current Biology</em> <strong>31</strong>, 2929-2938.e5 (2021). </p>
  
<p> Dellicour, S. & Flot, J.-F. Delimiting Species-Poor Data Sets using Single Molecular Markers: A Study of Barcode Gaps, Haplowebs and GMYC. <em>Syst Biol</em> <strong>64</strong>, 900–908 (2015). </p>

<p> Dixon, J. D. <em>et al.</em> Genetic consequences of habitat fragmentation and loss: the case of the Florida black bear (Ursus americanus floridanus). <em>Conserv Genet</em> <strong>8</strong>, 455–464 (2007). </p>

<p> Dunbar, M. R., Cunningham, M. W., Wooding, J. B. & Roth, R. P. Cryptorchidism and Delayed Testicular Descent in Florida Black Bears. <em>Journal of Wildlife Diseases</em> <strong>32</strong>, 661–664 (1996). </p>

<p> Fennessy, J. <em>et al.</em> Multi-locus Analyses Reveal Four Giraffe Species Instead of One. <em>Current Biology</em> <strong>26</strong>, 2543–2549 (2016). </p>

<p> Guralnick, R. & Hill, A. Biodiversity informatics: automated approaches for documenting global biodiversity patterns and processes. <em>Bioinformatics</em> <strong>25</strong>, 421–428 (2009). </p>

<p> Huelsenbeck, J. P., Andolfatto, P. & Huelsenbeck, E. T. Structurama: Bayesian Inference of Population Structure. <em>Evol Bioinform Online</em> <strong>7</strong>strong> EBO.S6761 (2011). </p>

<p> Monaghan, M. T. <em>et al.</em> Accelerated Species Inventory on Madagascar Using Coalescent-Based Models of Species Delineation. <em>Systematic Biology</em> <strong>58</strong>, 298–311 (2009). </p>

<p> Nachman, M. W. & Crowell, S. L. Estimate of the Mutation Rate per Nucleotide in Humans. <em>Genetics</em> <strong>156</strong>, 297–304 (2000). </p>

<p> Pons, J. <em>et al.</em> Sequence-Based Species Delimitation for the DNA Taxonomy of Undescribed Insects. <em>Systematic Biology</em> <strong>55</strong>, 595–609 (2006). </p>
  
<p> Pritchard, J. K., Stephens, M. & Donnelly, P. Inference of Population Structure Using Multilocus Genotype Data. <em>Genetics</em> <strong>155</strong>, 945–959 (2000). </p>
  
<p> Reid, N. M. & Carstens, B. C. Phylogenetic estimation error can decrease the accuracy of species delimitation: a Bayesian implementation of the general mixed Yule-coalescent model. <em>BMC Evol Biol</em> <strong>12</strong>, 196 (2012). </p>
  
<p> Wheeler, Q. D. Taxonomic triage and the poverty of phylogeny. <em>Phil. Trans. R. Soc. Lond. B</em> <strong>359</strong>, 571–583 (2004). </p>
  
<p> Wieczorek, J. <em>et al.</em> Darwin Core: An Evolving Community-Developed Biodiversity Data Standard. <em>PLoS ONE</em> <strong>7</strong>, e29715 (2012). </p>
  
<p> Yesson, C. <em>et al.</em> How Global Is the Global Biodiversity Information Facility? <em>PLoS ONE</em> <strong>2</strong>, e1124 (2007). </p>
  
</div>
            </section>

          </div>
        </div>
      </div>
<a href="predictive-phylogeography.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
