<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>6 Species limits | PhylogatR</title>
  <meta name="description" content="" />
  <meta name="generator" content="bookdown 0.26.3 and GitBook 2.6.7" />

  <meta property="og:title" content="6 Species limits | PhylogatR" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="6 Species limits | PhylogatR" />
  
  
  

<meta name="author" content="Bryan Carstens (Ohio State University), Sydney Decker (Ohio State University), Anahi Espindola (University of Maryland), Sarah Foltz (Radford Universsity), Abbi Green (Radford University), Danielle Parsons (Ohio State University), Tara Pelletier (Radford University), Jordan Satler (Ohio State University). This work was funded by the National Science Foundation (DBI-1910623 and DBI-1911293)" />


<meta name="date" content="2022-08-03" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="population-genetics-and-conservation.html"/>
<link rel="next" href="predictive-phylogeography.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<link href="libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="libs/pagedtable-1.1/js/pagedtable.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Background</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#how-to-use-this-book"><i class="fa fa-check"></i><b>1.1</b> How to use this book</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#prerequisites-if-using-r"><i class="fa fa-check"></i><b>1.2</b> Prerequisites if using R</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="index.html"><a href="index.html#r-and-rstudio"><i class="fa fa-check"></i><b>1.2.1</b> R and RStudio</a></li>
<li class="chapter" data-level="1.2.2" data-path="index.html"><a href="index.html#r-packages"><i class="fa fa-check"></i><b>1.2.2</b> R packages</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>2</b> Introduction</a>
<ul>
<li class="chapter" data-level="2.1" data-path="introduction.html"><a href="introduction.html#data-in-the-biological-sciences"><i class="fa fa-check"></i><b>2.1</b> Data in the biological sciences</a></li>
<li class="chapter" data-level="2.2" data-path="introduction.html"><a href="introduction.html#why-species"><i class="fa fa-check"></i><b>2.2</b> Why species?</a></li>
<li class="chapter" data-level="2.3" data-path="introduction.html"><a href="introduction.html#the-scientific-method"><i class="fa fa-check"></i><b>2.3</b> The Scientific Method</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="dna.html"><a href="dna.html"><i class="fa fa-check"></i><b>3</b> DNA</a>
<ul>
<li class="chapter" data-level="3.1" data-path="dna.html"><a href="dna.html#the-genetic-code"><i class="fa fa-check"></i><b>3.1</b> The genetic code</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="dna.html"><a href="dna.html#practice-translate-the-following-dna-sequence-into-amino-acids"><i class="fa fa-check"></i><b>3.1.1</b> Practice: Translate the following DNA sequence into amino acids</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="dna.html"><a href="dna.html#mutation"><i class="fa fa-check"></i><b>3.2</b> Mutation</a></li>
<li class="chapter" data-level="3.3" data-path="dna.html"><a href="dna.html#synonymous-versus-non-synonymous-genetic-variation"><i class="fa fa-check"></i><b>3.3</b> Synonymous versus non-synonymous genetic variation</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="dna.html"><a href="dna.html#practice-introduce-a-mutation"><i class="fa fa-check"></i><b>3.3.1</b> Practice: Introduce a mutation</a></li>
<li class="chapter" data-level="3.3.2" data-path="dna.html"><a href="dna.html#practice-introduce-an-insertion"><i class="fa fa-check"></i><b>3.3.2</b> Practice: Introduce an insertion</a></li>
<li class="chapter" data-level="3.3.3" data-path="dna.html"><a href="dna.html#practice-calculate-the-proportion-of-single-base-pair-mutations-that-are-synonymous"><i class="fa fa-check"></i><b>3.3.3</b> Practice: Calculate the proportion of single base pair mutations that are synonymous</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="dna.html"><a href="dna.html#probabilities-snps-and-genetic-variation"><i class="fa fa-check"></i><b>3.4</b> Probabilities, SNPs, and genetic variation</a></li>
<li class="chapter" data-level="3.5" data-path="dna.html"><a href="dna.html#modeling-sequence-variation"><i class="fa fa-check"></i><b>3.5</b> Modeling sequence variation</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="dna.html"><a href="dna.html#practice-estimate-base-pair-composition"><i class="fa fa-check"></i><b>3.5.1</b> Practice: Estimate base pair composition</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="dna.html"><a href="dna.html#organellar-genomes-and-barcoding-genes"><i class="fa fa-check"></i><b>3.6</b> Organellar genomes and barcoding genes</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html"><i class="fa fa-check"></i><b>4</b> Biodiversity Databases</a>
<ul>
<li class="chapter" data-level="4.1" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#global-biodiversity-information-facility-gbif"><i class="fa fa-check"></i><b>4.1</b> Global Biodiversity Information Facility (GBIF)</a></li>
<li class="chapter" data-level="4.2" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#genbank"><i class="fa fa-check"></i><b>4.2</b> GenBank</a></li>
<li class="chapter" data-level="4.3" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#bold"><i class="fa fa-check"></i><b>4.3</b> BOLD</a></li>
<li class="chapter" data-level="4.4" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#phylogatr"><i class="fa fa-check"></i><b>4.4</b> <em>PhylogatR</em></a></li>
<li class="chapter" data-level="4.5" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#using-phylogatr"><i class="fa fa-check"></i><b>4.5</b> Using <em>phylogatR</em></a></li>
<li class="chapter" data-level="4.6" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#practice"><i class="fa fa-check"></i><b>4.6</b> Practice</a>
<ul>
<li class="chapter" data-level="4.6.1" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#downloading-data-from-phylogatr"><i class="fa fa-check"></i><b>4.6.1</b> Downloading data from <em>phylogatR</em></a></li>
<li class="chapter" data-level="4.6.2" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#check-for-outliers"><i class="fa fa-check"></i><b>4.6.2</b> Check for outliers</a></li>
<li class="chapter" data-level="4.6.3" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#check-sequence-alignments"><i class="fa fa-check"></i><b>4.6.3</b> Check sequence alignments</a></li>
<li class="chapter" data-level="4.6.4" data-path="biodiversity-databases.html"><a href="biodiversity-databases.html#visualizing-geographic-coordinates"><i class="fa fa-check"></i><b>4.6.4</b> Visualizing geographic coordinates</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html"><i class="fa fa-check"></i><b>5</b> Population genetics and conservation</a>
<ul>
<li class="chapter" data-level="5.1" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#loss-of-genetic-diversity"><i class="fa fa-check"></i><b>5.1</b> Loss of genetic diversity</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#small-population-size"><i class="fa fa-check"></i><b>5.1.1</b> Small population size</a></li>
<li class="chapter" data-level="5.1.2" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#genetic-drift"><i class="fa fa-check"></i><b>5.1.2</b> Genetic drift</a></li>
<li class="chapter" data-level="5.1.3" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#gene-flow"><i class="fa fa-check"></i><b>5.1.3</b> Gene flow</a></li>
<li class="chapter" data-level="5.1.4" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#inbreeding"><i class="fa fa-check"></i><b>5.1.4</b> Inbreeding</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#units-of-conservation"><i class="fa fa-check"></i><b>5.2</b> Units of conservation</a></li>
<li class="chapter" data-level="5.3" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#practice-1"><i class="fa fa-check"></i><b>5.3</b> Practice</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#learning-objectives-1"><i class="fa fa-check"></i><b>5.3.1</b> Learning objectives:</a></li>
<li class="chapter" data-level="5.3.2" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#data"><i class="fa fa-check"></i><b>5.3.2</b> Data</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="population-genetics-and-conservation.html"><a href="population-genetics-and-conservation.html#click-and-play"><i class="fa fa-check"></i><b>5.4</b> Click and play!</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="species-limits.html"><a href="species-limits.html"><i class="fa fa-check"></i><b>6</b> Species limits</a>
<ul>
<li class="chapter" data-level="6.1" data-path="species-limits.html"><a href="species-limits.html#cryptic-genetic-diversity"><i class="fa fa-check"></i><b>6.1</b> Cryptic genetic diversity</a></li>
<li class="chapter" data-level="6.2" data-path="species-limits.html"><a href="species-limits.html#genetic-clustering"><i class="fa fa-check"></i><b>6.2</b> Genetic clustering</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="species-limits.html"><a href="species-limits.html#structure"><i class="fa fa-check"></i><b>6.2.1</b> Structure</a></li>
<li class="chapter" data-level="6.2.2" data-path="species-limits.html"><a href="species-limits.html#structurama"><i class="fa fa-check"></i><b>6.2.2</b> Structurama</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="species-limits.html"><a href="species-limits.html#tree-based-methods"><i class="fa fa-check"></i><b>6.3</b> Tree-based methods</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="species-limits.html"><a href="species-limits.html#general-mixed-yule-coalescent-gmyc"><i class="fa fa-check"></i><b>6.3.1</b> General mixed Yule Coalescent (GMYC)</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="species-limits.html"><a href="species-limits.html#practice-2"><i class="fa fa-check"></i><b>6.4</b> Practice</a>
<ul>
<li class="chapter" data-level="6.4.1" data-path="species-limits.html"><a href="species-limits.html#data-preparation"><i class="fa fa-check"></i><b>6.4.1</b> Data Preparation</a></li>
<li class="chapter" data-level="6.4.2" data-path="species-limits.html"><a href="species-limits.html#generate-a-phylogenetic-tree"><i class="fa fa-check"></i><b>6.4.2</b> Generate a Phylogenetic Tree</a></li>
<li class="chapter" data-level="6.4.3" data-path="species-limits.html"><a href="species-limits.html#create-a-summary-tree"><i class="fa fa-check"></i><b>6.4.3</b> Create a summary tree</a></li>
<li class="chapter" data-level="6.4.4" data-path="species-limits.html"><a href="species-limits.html#species-delimitation"><i class="fa fa-check"></i><b>6.4.4</b> Species Delimitation</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html"><i class="fa fa-check"></i><b>7</b> Predictive Phylogeography</a>
<ul>
<li class="chapter" data-level="7.0.1" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#trait-data-and-response-variables"><i class="fa fa-check"></i><b>7.0.1</b> Trait data and response variables</a></li>
<li class="chapter" data-level="7.1" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#machine-learning-and-random-forest"><i class="fa fa-check"></i><b>7.1</b> Machine learning and random forest</a></li>
<li class="chapter" data-level="7.2" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#real-life-example"><i class="fa fa-check"></i><b>7.2</b> Real life example</a></li>
<li class="chapter" data-level="7.3" data-path="predictive-phylogeography.html"><a href="predictive-phylogeography.html#practice-3"><i class="fa fa-check"></i><b>7.3</b> Practice</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>8</b> References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./"><img src="Segoe1.png" title="fig:" style="width:75.0%" alt="PhylogatR" /></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="species-limits" class="section level1 hasAnchor" number="6">
<h1><span class="header-section-number">6</span> Species limits<a href="species-limits.html#species-limits" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>After <strong>speciation</strong>, we would expect species to become increasingly different from one another through the passage of time. We might expect these differences to accumulate at an equal rate among several axes of divergence: in their genome (molecular), in their physical appearance (morphology), and in their behavior. Essentially, we would predict a positive correlation between the time to a <strong>most recent common ancestor</strong> and the degree of molecular, morphological, and behavioral divergence between different species. But as typically happens in biology, this is not always the case.</p>
<p><img src="mrca.png" /></p>
<p><font size="1">Most recent common ancestor</font></p>
<div id="cryptic-genetic-diversity" class="section level2 hasAnchor" number="6.1">
<h2><span class="header-section-number">6.1</span> Cryptic genetic diversity<a href="species-limits.html#cryptic-genetic-diversity" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>As biologists started to sequence DNA and estimate <strong>phylogenetic relationships</strong> among individuals identified as belonging to the same species, patterns started to emerge suggesting the presence of deeper genetic divergence than one would expect within a species. This pattern has been observed across many groups from the tree of life, and is often the case for groups that have limited dispersal ability and are found in regions with complex geological histories and geographic landscapes. In particular, groups like mygalomorph spiders (<em>e.g.</em>, tarantulas, trapdoor spiders, purseweb spiders) and salamanders typically harbor extensive population structure, reflecting their limited dispersal abilities and often specific habitat requirements. Interpreting these structured genealogical patterns becomes even more challenging when the group of interest looks and behaves in a similar way. In particular, when a species is described based on morphology (<em>i.e.</em>, they all share a physical appearance suggesting a single species), yet harbors extensive population genetic structure (<em>i.e.</em>, they are genetically distinct from one another), we are left with this question–how many species do these individuals represent? This question has no simple answer.</p>
<p><img src="spiders-01.png" /></p>
<p><font size="1"><em>Hadronyche venenata</em> (Mygalomorphae, Atracidae) from Tasmania. Mygalomorph spiders (e.g., tarantulas, trapdoor spiders, and kin) are often morphologically conserved at shallow, phylogenetic levels leading to considerable cryptic species diversity. Photos by Jordan Satler</font></p>
<p>With increased sequencing effort across the <a href="http://tolweb.org/tree/" target="_blank">tree of life</a>, the finding of cryptic genetic diversity within what is often described as a single species (usually based on morphology) is becoming increasingly common. This result raises multiple questions. In particular, is this cryptic genetic diversity indicative of the presence of multiple <strong>cryptic species</strong> (species that look similar but are genetically distinct), and if so, what are the processes that have produced species that look and behave the same, yet are genetically distinct?</p>
<p>When we uncover cryptic genetic diversity, we want to know what that means for species-level biodiversity. Because species are the fundamental unit of analysis for studies spanning evolutionary biology and ecology, it is critical we first know what the species are. And although we often think of species as being defined units where the axes of divergence have been growing linearly through time, processes generating divergence and speciation vary across the tree of life. As such, and especially at the earliest stages of speciation, we are often working in these <em>grey areas</em> where it is unclear if genetic divergence reflects <em>within population-level</em> patterns or <em>between species-level</em> patterns. Methods and approaches are available for investigating what cryptic genetic diversity means for the speciation process and for the question of how many species are present – we outline several of these approaches below.</p>
</div>
<div id="genetic-clustering" class="section level2 hasAnchor" number="6.2">
<h2><span class="header-section-number">6.2</span> Genetic clustering<a href="species-limits.html#genetic-clustering" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Genetic clustering methods provide useful approaches for estimating the number of species in a set of sampled individuals. Because we consider these methods to be discovery approaches, we can test, without any prior knowledge, how many species our samples represent, and how the individuals are partitioned into species. Clustering methods are non-tree-based approaches that typically use a model to estimate which genetic cluster each individual most likely belongs. We can think of these approaches as starting with some number of genetic clusters, and asking, for each sampled individual, which genetic cluster is most likely to produce an individual with that genetic fingerprint. Thus, these methods group individuals into genetic clusters based on similarity in their genetic composition.</p>
<p>Two of the most popular genetic clustering methods are Structure (Pritchard et al. 2000) and Structurama (Huelsenbeck et al. 2011). Both are model-based approaches based on <strong>Hardy–Weinberg equilibrium</strong>, with the goal being to group individuals into genetic clusters by maximizing Hardy–Weinberg equilibrium within each cluster. When specifying a set number of clusters (represented as K), we can ask how our sampled individuals are drawn genetically from these different clusters. We can restrict our model to only allow individuals to be drawn from a single cluster, or we can allow individuals to be representative of multiple clusters. In the latter case, we account for processes like <strong>hybridization</strong> and <strong>admixture</strong>, where species are exchanging genetic material on some level, which may be a better reflection of the biological processes generating our current species diversity.</p>
<div id="structure" class="section level3 hasAnchor" number="6.2.1">
<h3><span class="header-section-number">6.2.1</span> Structure<a href="species-limits.html#structure" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Structure is the most popular genetic clustering method used by the scientific community (the original paper has 31,142 citations! Google Scholar, 12 May 2021). Implemented in a Bayesian framework, the user first sets the number of genetic clusters (K) and then estimates which cluster(s) could generate each individual’s genetic fingerprint. A popular approach for species delimitation is to analyze a data set at multiple K values, for example, in a set of 20 individuals, we could run the analysis estimating between two and ten genetic clusters. Then we can use different statistics to estimate the optimal number of genetic clusters and for example, if we determine that the optimal number for K is equal to four, we would say that, based on Structure, we estimate there are four species in our data set, We can then determine how the individuals are partitioned among those species. Although determining the number of K clusters with Structure is challenging and the robustness of the statistical basis is debated, this type of approach provides an estimate as to how many species represent our sampled data set.</p>
</div>
<div id="structurama" class="section level3 hasAnchor" number="6.2.2">
<h3><span class="header-section-number">6.2.2</span> Structurama<a href="species-limits.html#structurama" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Another clustering approach is implemented in Structurama. This method is appealing because instead of telling the program how many clusters we want to estimate, we can treat K as a random variable, and have the program estimate the optimal number of clusters for our data set. This is statistically sound and a preferred approach if we are unsure as to how many species our data set represents.</p>
<p>Although originally developed to identify population structure within species, clustering approaches have been useful for identifying potential species-level biodiversity within a group of interest. In particular, we can use these methods to try and identify all possible species present within a data set. We can then directly test these potential species with additional methods, and see if our initial results are validated, or we can use other methods to see if they generate similar results on what we think the species are in our group. If we observe <em>congruence</em> across methods, this would suggest what we consider to be our species are good estimates; if methods provide <em>incongruent</em> results, we should explore why results might differ across methods and which methods might be inappropriate (maybe there are model violations?) for our data set.</p>
<p><img src="structure.png" /></p>
<p><font size="1">Clustering</font></p>
</div>
</div>
<div id="tree-based-methods" class="section level2 hasAnchor" number="6.3">
<h2><span class="header-section-number">6.3</span> Tree-based methods<a href="species-limits.html#tree-based-methods" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>As opposed to genetic clustering, we can use <strong>phylogenetic trees</strong> to estimate the number of species present in a genetic dataset. These methods take into account the evolutionary history of a group, and not just simply sequence divergence, as in the clustering methods.</p>
<div id="general-mixed-yule-coalescent-gmyc" class="section level3 hasAnchor" number="6.3.1">
<h3><span class="header-section-number">6.3.1</span> General mixed Yule Coalescent (GMYC)<a href="species-limits.html#general-mixed-yule-coalescent-gmyc" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>We can use the information present in a phylogeny to address the question of how many species. In particular, we can use our expectations of the different processes that generate branching patterns in a gene tree to estimate how the tips of the tree—our sampled individuals—should be clustered into species. This is because macroevolutionary processes like speciation and extinction have different expectations for the timing of branching events as opposed to microevolutionary processes operating within a species. Developed by Pons et al. (2006), this approach is called the general mixed Yule coalescent (GMYC) model.</p>
<p>The GMYC model estimates when between species (<strong>interspecific</strong>) branching events transition to within species (<strong>intraspecific</strong>) branching events. Given a set of gene sequences from a group of individuals, we can use this model to generate expectations of how many species are in our data set and how those individuals are clustered into species. For groups of organisms that are poorly known (<em>i.e.</em>, understudied groups) or lack morphological or behavioral differences (<em>i.e.</em>, cryptic), this can be a useful approach for estimating species diversity. If we want to know what processes are important for contributing to speciation and extinction, for understanding how reproductive isolation evolves, and for understanding what shapes species’ distributions, we need to know what the species are. The GMYC can be a useful model for addressing this question.</p>
<p>Originally implemented in a Maximum Likelihood framework, the GMYC model has since been extended to a Bayesian framework to account for uncertainty in our gene tree estimates (Reid &amp; Carstens 2012). In addition, while the GMYC model tests for the single transition point between interspecific and intraspecific branching patterns, the model has also been extended to allow for multiple transitions in the branching patterns within a gene tree (Monaghan et al. 2009). Thus, if the speciation process has occurred at different times among the sampled species, allowing multiple transitions might better capture the processes generating species-level biodiversity.</p>
<p>Although the GMYC model is straight-forward to implement and can be useful for estimating species diversity, there are several potential issues that might mislead us in our attempt to address the question of how many species. For example, if a species contains considerable population structure, likely in dispersal-limited organisms (think small things that have to stay wet, like salamanders), the model can have trouble determining when the transition in branching patterns takes place. This is because population genetic structure looks similar to phylogenetic structure, and the lack of an obvious transition in branching patterns cannot be accurately detected by the model. But the important thing when using this approach—and any model-based method in general, is to be aware of the model assumptions and understanding when you might be misled, and why.</p>
<p><img src="gmyc.png" style="width:75.0%" /></p>
<p><font size="1">GMYC. A threshold is determined on this phylogenetic tree and all splits in the tree before this line are considered speciation events.</font></p>
<p>In general, the GMYC model is useful for estimating species-level biodiversity and is often an important first step when working on poorly known groups (<em>e.g.</em>, insects, arachnids) and in regions (<em>e.g.</em>, tropics) where species diversity for the group of interest has received little or no taxonomic attention. And because we can generate mitochondrial DNA sequence data from any animal, and use those data to estimate a gene tree, we can use this model as a first step for addressing the question of how many species.</p>
</div>
</div>
<div id="practice-2" class="section level2 hasAnchor" number="6.4">
<h2><span class="header-section-number">6.4</span> Practice<a href="species-limits.html#practice-2" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>This module is designed to walk you through the steps of species delimitation using the <strong>G</strong>eneralized <strong>M</strong>ixed <strong>Y</strong>ule <strong>C</strong>oalescent, also known as <strong>GMYC</strong>.</p>
<p>This module will walk you through the process of using GMYC to perform species delimitation on a set of named species using genetic information from a single gene. For this exercise, we’ll focus on the following steps: 1) preparing your data, 2) generating a phylogenetic tree, and 3) using that tree to perform species delimitation under the GMYC.</p>
<p>In this example, we’ll be looking at several formally named shrew species of the genus <em>Sorex</em>.</p>
<p><img src="shrews.jpg" /></p>
<p>Before beginning, make sure you have all of the packages required for this module, and then load the libraries.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="species-limits.html#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="co">#load necessary packages</span></span>
<span id="cb1-2"><a href="species-limits.html#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(beautier)</span>
<span id="cb1-3"><a href="species-limits.html#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(beastier)</span>
<span id="cb1-4"><a href="species-limits.html#cb1-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ape)</span>
<span id="cb1-5"><a href="species-limits.html#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(phangorn)</span>
<span id="cb1-6"><a href="species-limits.html#cb1-6" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(splits)</span>
<span id="cb1-7"><a href="species-limits.html#cb1-7" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(Biostrings)</span>
<span id="cb1-8"><a href="species-limits.html#cb1-8" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(DECIPHER)</span></code></pre></div>
<blockquote>
<p><strong>IMPORTANT:</strong> Before running any of the code below, make sure to tell R which working directory (i.e., folder) you want to work from.</p>
</blockquote>
<div id="data-preparation" class="section level3 hasAnchor" number="6.4.1">
<h3><span class="header-section-number">6.4.1</span> Data Preparation<a href="species-limits.html#data-preparation" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><img src="step1.jpg" /></p>
<div id="create-a-multi-species-alignment" class="section level4 hasAnchor" number="6.4.1.1">
<h4><span class="header-section-number">6.4.1.1</span> Create a multi-species alignment<a href="species-limits.html#create-a-multi-species-alignment" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>In this module we’ll use data from multiple species. Automated species delimitation methods, such as GMYC, can be done using single-species datasets, but they tend to perform better on datasets containing more than one species (Dellicour et al. 2015). We are working from a folder called “phylogatr-results-species”, which contains the data downloaded from <a href="https://phylogatr.org" target="_blank">PhylogatR</a>. We are going to import genetic sequence data from the gene CYTB for several shrew species of the genus <em>Sorex</em>.</p>
<p>First, we need to store the path names of all of the fasta files for our chosen gene. Make sure <code>pattern</code> is set to the name of the gene you want to use. Here, we’ll generate a list of paths to all of the fasta files for the gene CYTB (i.e., all files within <code>phylogatr-results</code> that end in CYTB.fa) using the function <code>list.files</code> and store this list in R as the object <code>myfiles</code>.</p>
<p>We can then use the <code>basename()</code> function to determine which species have data for our selected gene, CYTB. Here, we can see that our new dataset will contain CYTB data from 5 shrew species.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="species-limits.html#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="co">#get paths to CYTB fasta files in species folders (all CYTB.fa files in &quot;phylogatr-results&quot; folder)</span></span>
<span id="cb2-2"><a href="species-limits.html#cb2-2" aria-hidden="true" tabindex="-1"></a>myfiles<span class="ot">&lt;-</span><span class="fu">list.files</span>(<span class="at">path =</span> <span class="st">&quot;phylogatr-results-species&quot;</span>, <span class="at">pattern=</span><span class="st">&quot;CYTB.fa&quot;</span>, <span class="at">recursive =</span> <span class="cn">TRUE</span>, <span class="at">full.names=</span><span class="cn">TRUE</span>)</span>
<span id="cb2-3"><a href="species-limits.html#cb2-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-4"><a href="species-limits.html#cb2-4" aria-hidden="true" tabindex="-1"></a><span class="co">#list species in dataset</span></span>
<span id="cb2-5"><a href="species-limits.html#cb2-5" aria-hidden="true" tabindex="-1"></a><span class="fu">basename</span>(myfiles)</span></code></pre></div>
<pre><code>## [1] &quot;Sorex-bendirii-CYTB.fa&quot;   &quot;Sorex-cinereus-CYTB.fa&quot;  
## [3] &quot;Sorex-monticolus-CYTB.fa&quot; &quot;Sorex-palustris-CYTB.fa&quot; 
## [5] &quot;Sorex-vagrans-CYTB.fa&quot;</code></pre>
<p>Next, we will use the <code>lapply()</code> function to read the DNA sequences from each of our selected fasta files into R and then use the <code>do.call</code> function to combine all of the sequences into a single object, which we’ll store in R as <code>fa_seq</code>.</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="species-limits.html#cb4-1" aria-hidden="true" tabindex="-1"></a><span class="co">#import each fasta file and put them into a list  </span></span>
<span id="cb4-2"><a href="species-limits.html#cb4-2" aria-hidden="true" tabindex="-1"></a>fa_seq <span class="ot">=</span> <span class="fu">lapply</span>(myfiles, readDNAStringSet)</span>
<span id="cb4-3"><a href="species-limits.html#cb4-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-4"><a href="species-limits.html#cb4-4" aria-hidden="true" tabindex="-1"></a><span class="co">#combine all sequences in the list into a single object</span></span>
<span id="cb4-5"><a href="species-limits.html#cb4-5" aria-hidden="true" tabindex="-1"></a>fa_seq <span class="ot">=</span> <span class="fu">do.call</span>(c,fa_seq)</span></code></pre></div>
<p>Now that we have an object containing all of the CYTB sequences for each of our selected species, our next step is to align the sequences (i.e., arrange them to identify regions of similarity).</p>
<p>First, we’ll prepare our data for alignment by removing any gaps in the DNA sequences using the <code>RemoveGaps</code> function. Then, we can align the sequences using the <code>AlignSeqs()</code> function and store this alignment in R as the object <code>align_seq</code>. We’ll also save this alignment in our working directory as the file <code>align_seq.afa</code>.</p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="species-limits.html#cb5-1" aria-hidden="true" tabindex="-1"></a><span class="co">#remove any sequence gaps </span></span>
<span id="cb5-2"><a href="species-limits.html#cb5-2" aria-hidden="true" tabindex="-1"></a>fa_seq <span class="ot">&lt;-</span> <span class="fu">RemoveGaps</span>(fa_seq,</span>
<span id="cb5-3"><a href="species-limits.html#cb5-3" aria-hidden="true" tabindex="-1"></a>           <span class="at">removeGaps =</span> <span class="st">&quot;all&quot;</span>)</span>
<span id="cb5-4"><a href="species-limits.html#cb5-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-5"><a href="species-limits.html#cb5-5" aria-hidden="true" tabindex="-1"></a><span class="co">#align the sequences </span></span>
<span id="cb5-6"><a href="species-limits.html#cb5-6" aria-hidden="true" tabindex="-1"></a>align_seq <span class="ot">&lt;-</span> <span class="fu">AlignSeqs</span>(fa_seq)</span>
<span id="cb5-7"><a href="species-limits.html#cb5-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-8"><a href="species-limits.html#cb5-8" aria-hidden="true" tabindex="-1"></a><span class="co">#save the new alignment as an aligned fasta file (al_seq.afa) in the &quot;phylogatr-results&quot; folder</span></span>
<span id="cb5-9"><a href="species-limits.html#cb5-9" aria-hidden="true" tabindex="-1"></a><span class="fu">writeXStringSet</span>(align_seq,</span>
<span id="cb5-10"><a href="species-limits.html#cb5-10" aria-hidden="true" tabindex="-1"></a>   <span class="at">file =</span> <span class="st">&quot;align_seq.afa&quot;</span>, <span class="at">append =</span> <span class="cn">FALSE</span>)</span></code></pre></div>
</div>
<div id="create-beast-input-file-xml-file" class="section level4 hasAnchor" number="6.4.1.2">
<h4><span class="header-section-number">6.4.1.2</span> Create BEAST input file (xml file)<a href="species-limits.html#create-beast-input-file-xml-file" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Next, we need to convert our newly created sequence alignment into a format able to be analyzed by BEAST (i.e., an xml file). A BEAST xml file contains everything needed for a BEAST analysis, including the genetic data we want to analyze as well as instructions on how we want the analysis to be run. When generating a BEAST xml file, we have the option to edit these instructions by changing certain settings in the file.</p>
<p>To create a BEAST xml file, we’ll use the <code>create_beast2_input_file</code> function from the R package <code>beautier</code>. The argument <code>input_filename</code> directs R to your sequence alignment file, while <code>output_filename</code> tells R what to name the output file and where to put it. We’ll set <code>input_filename</code> to the name of our alignment file, “align_seq.afa”, and we’ll set <code>output_filename</code> as “beast_input.xml” to have R save the new xml file in our “phylogatr-results’ folder as”beast_input.xml". Run the following code to generate an xml file from your sequence alignment, using BEAST’s <a href="https://github.com/ropensci/beautier" target="_blank">default parameters</a>.</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="species-limits.html#cb6-1" aria-hidden="true" tabindex="-1"></a><span class="co">#generate an xml file (default parameter settings)</span></span>
<span id="cb6-2"><a href="species-limits.html#cb6-2" aria-hidden="true" tabindex="-1"></a><span class="fu">create_beast2_input_file</span>(</span>
<span id="cb6-3"><a href="species-limits.html#cb6-3" aria-hidden="true" tabindex="-1"></a>          <span class="at">input_filename =</span> <span class="st">&quot;align_seq.afa&quot;</span>,</span>
<span id="cb6-4"><a href="species-limits.html#cb6-4" aria-hidden="true" tabindex="-1"></a>          <span class="at">output_filename =</span> <span class="st">&quot;beast_input.xml&quot;</span>)</span></code></pre></div>
<div id="so-many-settings" class="section level5 hasAnchor" number="6.4.1.2.1">
<h5><span class="header-section-number">6.4.1.2.1</span> <img src="settings.svg" style="width:6.0%" />So many settings!<a href="species-limits.html#so-many-settings" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<blockquote>
<p><strong>The following examples demonstrate how parameter values can be adjusted to set properties such as model of nucleotide substitution and proportion of invariant sites in your xml file. If you want to stick with the default parameter settings</strong><a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a><strong>, proceed to the next section.</strong></p>
</blockquote>
<p>As you can see in the code above, the only parameters we defined when creating our xml file were <code>input_filename</code>, which is the location of our sequence alignment file, and <code>output_filename</code>, which is what we want to name our output xml file. These are just two of <strong>many</strong> parameters that we can adjust when creating xml files through the <code>create_beast2_input_file</code> function. Additional parameters include <code>site_model</code>, <code>clock_model</code>, <code>tree_priors</code>, and <code>mrca_prior</code>, among others. More information on setting xml file parameters can be found <a href="https://github.com/ropensci/beautier">here</a>.</p>
<p><strong>Example 1:</strong> By adding the argument <code>site_model = create_jc69_site_model()</code> to the previous function, we can use our sequence alignment to generate an xml file using the JC69 substitution model. Note that while <code>input_filename</code> stays the same, we’ll change <code>output_filename</code> to “beast_input_jc69.xml”, so that our previous file doesn’t get overwritten.</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="species-limits.html#cb7-1" aria-hidden="true" tabindex="-1"></a><span class="co">#create xml file (JC69 substitution model)</span></span>
<span id="cb7-2"><a href="species-limits.html#cb7-2" aria-hidden="true" tabindex="-1"></a><span class="fu">create_beast2_input_file</span>(</span>
<span id="cb7-3"><a href="species-limits.html#cb7-3" aria-hidden="true" tabindex="-1"></a>           <span class="at">input_filename =</span> <span class="st">&quot;align_seq.afa&quot;</span>,</span>
<span id="cb7-4"><a href="species-limits.html#cb7-4" aria-hidden="true" tabindex="-1"></a>           <span class="at">output_filename =</span> <span class="st">&quot;beast_input_jc69.xml&quot;</span>,</span>
<span id="cb7-5"><a href="species-limits.html#cb7-5" aria-hidden="true" tabindex="-1"></a>           <span class="at">site_model =</span> <span class="fu">create_jc69_site_model</span>())</span></code></pre></div>
<p><strong>Example 2:</strong> This time we’ll generate an xml file using the HKY85 model and save it in our “phylogatr-results” folder as “beast_input_hky.xml”. We’ll set our substitution model to HKY by adding the argument <code>site_model = create_hky_site_model</code>. We can also change the proportion of invariant sites by specifying the value of <code>prop_invariant</code> under the <code>gamma_site_model</code> argument.</p>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="species-limits.html#cb8-1" aria-hidden="true" tabindex="-1"></a><span class="co">#create xml file (HKY substitution model and prop invariant = 0.3)</span></span>
<span id="cb8-2"><a href="species-limits.html#cb8-2" aria-hidden="true" tabindex="-1"></a><span class="fu">create_beast2_input_file</span>(</span>
<span id="cb8-3"><a href="species-limits.html#cb8-3" aria-hidden="true" tabindex="-1"></a>           <span class="at">input_filename =</span> <span class="st">&quot;align_seq.afa&quot;</span>,</span>
<span id="cb8-4"><a href="species-limits.html#cb8-4" aria-hidden="true" tabindex="-1"></a>           <span class="at">output_filename =</span> <span class="st">&quot;beast_input_hky.xml&quot;</span>,</span>
<span id="cb8-5"><a href="species-limits.html#cb8-5" aria-hidden="true" tabindex="-1"></a>           <span class="at">site_model =</span> <span class="fu">create_hky_site_model</span>(</span>
<span id="cb8-6"><a href="species-limits.html#cb8-6" aria-hidden="true" tabindex="-1"></a>             <span class="at">gamma_site_model =</span> <span class="fu">create_gamma_site_model</span>(<span class="at">prop_invariant =</span> <span class="fl">0.3</span>)))</span></code></pre></div>
</div>
</div>
</div>
<div id="generate-a-phylogenetic-tree" class="section level3 hasAnchor" number="6.4.2">
<h3><span class="header-section-number">6.4.2</span> Generate a Phylogenetic Tree<a href="species-limits.html#generate-a-phylogenetic-tree" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><img src="step2.jpg" /></p>
<div id="bayesian-evolutionary-analysis-beast" class="section level4 hasAnchor" number="6.4.2.1">
<h4><span class="header-section-number">6.4.2.1</span> Bayesian evolutionary analysis (BEAST)<a href="species-limits.html#bayesian-evolutionary-analysis-beast" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Now that we have our genetic data stored in an xml file, we can use the program BEAST to infer a set of evolutionary trees, based on our data. To do this, we’ll use the function <code>run_beast2()</code>. Set the argument <code>input_filename</code> to the name of the xml file you want to use and run the code below.</p>
<p><em>Note that, depending on the number of species included, this analysis may take longer to complete.</em> You can track the progress of your analysis using the output written to the screen.</p>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="species-limits.html#cb9-1" aria-hidden="true" tabindex="-1"></a><span class="co">#run BEAST using your previously generated xml file</span></span>
<span id="cb9-2"><a href="species-limits.html#cb9-2" aria-hidden="true" tabindex="-1"></a><span class="fu">run_beast2</span>(</span>
<span id="cb9-3"><a href="species-limits.html#cb9-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">input_filename =</span> <span class="st">&quot;beast_input.xml&quot;</span>)</span></code></pre></div>
<p>When the analysis is complete, BEAST will create two new files in your working directory. We will use the <code>align_seq.trees</code> file in the next step of this module.</p>
</div>
</div>
<div id="create-a-summary-tree" class="section level3 hasAnchor" number="6.4.3">
<h3><span class="header-section-number">6.4.3</span> Create a summary tree<a href="species-limits.html#create-a-summary-tree" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>BEAST gives us a set of evolutionary trees inferred from our genetic data. However, we can only use a single tree for species delimitation (under GMYC), so we need to generate a summary tree. There are several different approaches to summarizing trees, which you can learn more about <a href="https://beast.community/summarizing_trees" target="_blank">here</a>. We’ll use a <em>M</em>aximum <em>C</em>lade <em>C</em>redibility (<em>MCC</em>) tree to summarize our BEAST results.</p>
<p>First, load your <code>align_seq.trees</code> file into R using the <code>read.nexus</code> function, and save it as the object <code>trees</code>. Then use the function <code>maxCladeCred</code> to generate an MCC tree from <code>trees</code>, and the resulting summary tree in R as <code>mcc_tree</code>.</p>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="species-limits.html#cb10-1" aria-hidden="true" tabindex="-1"></a><span class="co">#load newly generated .trees file into R</span></span>
<span id="cb10-2"><a href="species-limits.html#cb10-2" aria-hidden="true" tabindex="-1"></a>trees <span class="ot">&lt;-</span> <span class="fu">read.nexus</span>(<span class="at">file =</span> <span class="st">&quot;align_seq.trees&quot;</span>)</span>
<span id="cb10-3"><a href="species-limits.html#cb10-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-4"><a href="species-limits.html#cb10-4" aria-hidden="true" tabindex="-1"></a><span class="co">#generate an MCC summary tree </span></span>
<span id="cb10-5"><a href="species-limits.html#cb10-5" aria-hidden="true" tabindex="-1"></a>mcc_tree <span class="ot">&lt;-</span> <span class="fu">maxCladeCred</span>(trees, <span class="at">tree =</span> <span class="cn">TRUE</span>, <span class="at">part =</span> <span class="cn">NULL</span>, <span class="at">rooted =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</div>
<div id="species-delimitation" class="section level3 hasAnchor" number="6.4.4">
<h3><span class="header-section-number">6.4.4</span> Species Delimitation<a href="species-limits.html#species-delimitation" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><img src="step3.jpg" /></p>
<div id="generalized-mixed-yule-coalescent-gmyc" class="section level4 hasAnchor" number="6.4.4.1">
<h4><span class="header-section-number">6.4.4.1</span> Generalized Mixed Yule Coalescent (GMYC)<a href="species-limits.html#generalized-mixed-yule-coalescent-gmyc" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Now that we have a summary tree, we’re finally ready for species delimitation! To do this, we’ll use the function <code>gmyc</code> to perform GMYC species delimitation on our recently created summary tree (<code>mcc_tree</code>). Our results will be stored in R as the object <code>my_gmyc</code>.</p>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb11-1"><a href="species-limits.html#cb11-1" aria-hidden="true" tabindex="-1"></a><span class="co">#run GMYC species delimitation</span></span>
<span id="cb11-2"><a href="species-limits.html#cb11-2" aria-hidden="true" tabindex="-1"></a>my_gmyc <span class="ot">&lt;-</span> <span class="fu">gmyc</span>(mcc_tree, <span class="at">method =</span> <span class="st">&quot;single&quot;</span>, <span class="at">quiet =</span> <span class="cn">TRUE</span>)</span></code></pre></div>
</div>
<div id="evaluate-your-results" class="section level4 hasAnchor" number="6.4.4.2">
<h4><span class="header-section-number">6.4.4.2</span> Evaluate your results<a href="species-limits.html#evaluate-your-results" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Let’s take a look at our results! We can access our GMYC results by using the <code>summary()</code> function, as shown below. The resulting output gives us the number of species delimited from our dataset and the significance of our GMYC model.</p>
<p><strong>So, how many species did our analysis identify?</strong> To answer this question, we’ll turn to the <strong><em>number of ML entities</em></strong> and the <strong><em>number of ML clusters</em></strong> listed above.</p>
<ul>
<li><strong>number of ML entities:</strong> the total number of species identified</li>
<li><strong>number of ML clusters:</strong> the number of species identified that contain two or more samples (sequences)</li>
<li><strong>result of LR test:</strong> can be used to estimate model significance</li>
</ul>
<p>In this example, we can see that GMYC identified 4 distinct clusters and 7 distinct entities from our dataset containing 18 named species of shrew.</p>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="species-limits.html#cb12-1" aria-hidden="true" tabindex="-1"></a><span class="co">#get a summary of GMYC results</span></span>
<span id="cb12-2"><a href="species-limits.html#cb12-2" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(my_gmyc)</span></code></pre></div>
<pre><code>## Result of GMYC species delimitation
## 
##  method: single
##  likelihood of null model:   2042.43
##  maximum likelihood of GMYC model:   2054.566
##  likelihood ratio:   24.27266
##  result of LR test:  5.36115e-06***
## 
##  number of ML clusters:  15
##  confidence interval:    10-29
## 
##  number of ML entities:  22
##  confidence interval:    11-37
## 
##  threshold time: -0.004094423</code></pre>
<p>NOTE: Relevant output values are highlighted above, but see <a href="https://tmfujis.wordpress.com/2013/04/23/how-to-run-gmyc/" target="_blank">here</a> and <a href="https://francoismichonneau.net/gmyc-tutorial/" target="_blank">here</a> for a more detailed explanation of how to interpret GMYC results.</p>
<p><strong>How do we know which sequences have been assigned to which species?</strong> To create a table listing sequence-species associations, we can use the <code>spec.list</code> function. Each entry in the resulting table represents an individual sequence in our dataset.</p>
<ul>
<li><strong>sample_name:</strong> the sequence’s PhylogatR ID</li>
<li><strong>GMYC_spec:</strong> species the sequence has been assigned to</li>
</ul>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb14-1"><a href="species-limits.html#cb14-1" aria-hidden="true" tabindex="-1"></a><span class="co">#get species assignments for each individual </span></span>
<span id="cb14-2"><a href="species-limits.html#cb14-2" aria-hidden="true" tabindex="-1"></a><span class="fu">spec.list</span>(my_gmyc)</span></code></pre></div>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["GMYC_spec"],"name":[1],"type":["dbl"],"align":["right"]},{"label":["sample_name"],"name":[2],"type":["chr"],"align":["left"]}],"data":[{"1":"1","2":"AF154539_897029958"},{"1":"1","2":"KF950119_1145223738"},{"1":"1","2":"KF950120_1145223752"},{"1":"1","2":"JN889608_1145172001"},{"1":"1","2":"KF950114_1145169589"},{"1":"1","2":"KF823920_1145257526"},{"1":"1","2":"KF950098_897061853"},{"1":"1","2":"KF950117_1145223659"},{"1":"1","2":"KF950122_1145225516"},{"1":"1","2":"KF950123_1145225543"},{"1":"1","2":"FJ667517_897067864"},{"1":"1","2":"KF823916_1145256925"},{"1":"1","2":"AF154540_2234228170"},{"1":"1","2":"KF823918_1145256962"},{"1":"1","2":"KF950103_897087916"},{"1":"1","2":"AF238007_897061137"},{"1":"1","2":"AF238042_897040902"},{"1":"1","2":"KF823917_1145256952"},{"1":"1","2":"KF823915_1145256120"},{"1":"1","2":"KF823921_1145273853"},{"1":"1","2":"KF950099_897074562"},{"1":"1","2":"KF950115_1145173509"},{"1":"1","2":"FJ667514_1145170948"},{"1":"1","2":"FJ667519_897067712"},{"1":"1","2":"KF950094_897040046"},{"1":"1","2":"AF238008_897036932"},{"1":"1","2":"KF950108_897036932"},{"1":"1","2":"AF238009_897029418"},{"1":"1","2":"KF950105_897029418"},{"1":"1","2":"AF238013_897037936"},{"1":"1","2":"KF950109_897037936"},{"1":"1","2":"KF950111_897076557"},{"1":"1","2":"FJ667515_1145174119"},{"1":"1","2":"FJ667516_1145174383"},{"1":"1","2":"FJ667518_897067501"},{"1":"1","2":"KF950118_1145223647"},{"1":"1","2":"KF823919_1145257436"},{"1":"2","2":"GU731516_1656052134"},{"1":"2","2":"GU731519_1656052038"},{"1":"2","2":"GU731518_1656052061"},{"1":"2","2":"GU731520_1656052079"},{"1":"2","2":"GU731521_1656052067"},{"1":"2","2":"GU731517_665917592"},{"1":"2","2":"GU731522_1656051968"},{"1":"3","2":"AF154546_1145368346"},{"1":"3","2":"JN889534_1145095306"},{"1":"3","2":"FJ667522_1145381959"},{"1":"3","2":"AF238019_1145367997"},{"1":"3","2":"FJ667523_1145367997"},{"1":"3","2":"KF950126_1145367997"},{"1":"3","2":"AF154547_1145328815"},{"1":"3","2":"AF154549_1145343754"},{"1":"3","2":"AF238018_1145343752"},{"1":"4","2":"AF154548_1145335622"},{"1":"4","2":"AF238016_1145335620"},{"1":"4","2":"AF238017_1145335605"},{"1":"5","2":"AF154541_897028875"},{"1":"5","2":"KF950102_897084809"},{"1":"5","2":"KF950147_897026141"},{"1":"5","2":"KF950156_897029300"},{"1":"5","2":"AF237994_897061189"},{"1":"5","2":"KF950110_897061189"},{"1":"5","2":"KF950095_897049828"},{"1":"5","2":"KF950144_897026037"},{"1":"5","2":"KF950124_1145254106"},{"1":"5","2":"AF154545_897077425"},{"1":"5","2":"AF237999_897077619"},{"1":"5","2":"KF950113_897077619"},{"1":"5","2":"KF950101_897084587"},{"1":"5","2":"KF950121_1145223816"},{"1":"5","2":"KF950157_897029308"},{"1":"5","2":"KF950142_897025845"},{"1":"5","2":"KF950143_897025847"},{"1":"5","2":"KF950137_897080646"},{"1":"5","2":"KF950152_897029140"},{"1":"5","2":"KF950150_897026419"},{"1":"5","2":"KF950158_897029372"},{"1":"5","2":"KF950104_897028875"},{"1":"5","2":"KF950155_897029244"},{"1":"5","2":"KF950159_897029373"},{"1":"5","2":"KF950146_897026140"},{"1":"5","2":"KF950151_897029139"},{"1":"5","2":"KF950164_897029479"},{"1":"5","2":"AF154542_897061621"},{"1":"5","2":"KF950138_1145343712"},{"1":"5","2":"AF238041_1145343712"},{"1":"5","2":"AF237995_897026138"},{"1":"5","2":"KF950148_897026396"},{"1":"5","2":"KF950149_897026398"},{"1":"5","2":"AF237996_897029756"},{"1":"5","2":"KF950106_897029756"},{"1":"5","2":"AF238006_897061744"},{"1":"5","2":"AF237997_897036931"},{"1":"5","2":"AF237998_897029148"},{"1":"5","2":"KF950107_897036931"},{"1":"5","2":"KF950145_897026061"},{"1":"5","2":"AF154543_897071037"},{"1":"5","2":"AF238002_897071065"},{"1":"5","2":"KF950093_897037840"},{"1":"5","2":"AF238005_897061098"},{"1":"5","2":"KF950096_897057145"},{"1":"5","2":"KF950153_897029206"},{"1":"5","2":"KF950154_897029207"},{"1":"6","2":"AF238000_897049795"},{"1":"6","2":"KF950083_897049795"},{"1":"6","2":"AF238001_897049859"},{"1":"6","2":"KF950080_897049815"},{"1":"6","2":"KF950081_897050580"},{"1":"6","2":"KF950082_897050551"},{"1":"6","2":"KF950084_897029408"},{"1":"6","2":"KF950161_897029410"},{"1":"6","2":"KF950160_897029406"},{"1":"6","2":"KF950162_897029430"},{"1":"6","2":"KF950163_897029433"},{"1":"7","2":"AF238031_897060967"},{"1":"7","2":"AF238032_897060912"},{"1":"7","2":"KF302762_1145331513"},{"1":"8","2":"AF238033_897026225"},{"1":"8","2":"KF302770_1145170570"},{"1":"8","2":"KF302776_1145254027"},{"1":"8","2":"KF302780_1145288397"},{"1":"8","2":"KF302801_897041524"},{"1":"8","2":"KF302802_897041526"},{"1":"8","2":"KF302800_897041522"},{"1":"8","2":"KF302805_897058140"},{"1":"8","2":"KF302804_897042350"},{"1":"8","2":"KF302771_1145172188"},{"1":"9","2":"JN889611_1145095410"},{"1":"9","2":"KF302783_1145346003"},{"1":"9","2":"KF302775_1145225869"},{"1":"9","2":"KF302779_1145261371"},{"1":"9","2":"KF302799_897041521"},{"1":"9","2":"KF302803_897041530"},{"1":"9","2":"JN889612_1145182523"},{"1":"9","2":"KF302767_1145151628"},{"1":"9","2":"KF302764_1145151894"},{"1":"9","2":"KF302772_1145183700"},{"1":"9","2":"KF302777_1145258920"},{"1":"9","2":"KF302778_1145258938"},{"1":"9","2":"KF302782_1145345993"},{"1":"9","2":"KF302781_1145288400"},{"1":"10","2":"KF302766_1145150686"},{"1":"10","2":"KF302773_1145184588"},{"1":"10","2":"KF302784_1145380228"},{"1":"11","2":"KF302768_1145152442"},{"1":"11","2":"KF302769_1145152470"},{"1":"12","2":"AF154551_897076490"},{"1":"12","2":"KF302838_1145183878"},{"1":"12","2":"KF950116_1145183862"},{"1":"12","2":"GU731515_1656052231"},{"1":"12","2":"GU731523_1656051961"},{"1":"12","2":"GU731524_1656051915"},{"1":"13","2":"JN889696_897041429"},{"1":"13","2":"JQ778864_1145172307"},{"1":"13","2":"JQ778857_1145170472"},{"1":"13","2":"JQ778863_1145171995"},{"1":"13","2":"JQ778884_1145278948"},{"1":"13","2":"JQ778871_1145254880"},{"1":"13","2":"JQ778888_1145278941"},{"1":"13","2":"JQ778896_1145277945"},{"1":"13","2":"JQ778865_1145176261"},{"1":"13","2":"JQ778868_1145256838"},{"1":"13","2":"JQ778858_1145170423"},{"1":"13","2":"JQ778859_1145170869"},{"1":"13","2":"JQ778860_1145170902"},{"1":"13","2":"JQ778873_1145278975"},{"1":"13","2":"JQ778875_1145278999"},{"1":"13","2":"JQ778883_1145278964"},{"1":"13","2":"JN889697_897084942"},{"1":"13","2":"JQ778882_1145278970"},{"1":"13","2":"JN889699_897091190"},{"1":"13","2":"JQ778866_1145176241"},{"1":"13","2":"JQ778872_1145278965"},{"1":"13","2":"JQ778892_1145278984"},{"1":"13","2":"JQ778885_1145278959"},{"1":"13","2":"JQ778891_1145278996"},{"1":"13","2":"JQ778855_1145170576"},{"1":"13","2":"JQ778862_1145172024"},{"1":"13","2":"JQ778869_1145254875"},{"1":"13","2":"JQ778876_1145278969"},{"1":"13","2":"JQ778879_1145278949"},{"1":"13","2":"JQ778881_1145278986"},{"1":"13","2":"JQ778877_1145278985"},{"1":"13","2":"JQ778880_1145278927"},{"1":"13","2":"JQ778890_1145279007"},{"1":"13","2":"JQ743213_1145256179"},{"1":"13","2":"JQ743215_1145256148"},{"1":"13","2":"JQ778853_897106625"},{"1":"13","2":"JQ778854_1145169418"},{"1":"13","2":"JQ778874_1145279006"},{"1":"13","2":"JQ778887_1145278953"},{"1":"13","2":"JQ778886_1145278956"},{"1":"13","2":"JQ778856_1145170610"},{"1":"13","2":"JQ778878_1145278952"},{"1":"13","2":"JQ778889_1145278922"},{"1":"13","2":"JQ778861_1145170933"},{"1":"13","2":"JQ778867_1145256968"},{"1":"13","2":"JQ778870_1145254867"},{"1":"14","2":"JN889689_1145328758"},{"1":"14","2":"JN889693_1145328906"},{"1":"14","2":"JN889694_1145331443"},{"1":"14","2":"JN889690_1145328819"},{"1":"14","2":"JN889691_1145328825"},{"1":"14","2":"JN889692_1145328828"},{"1":"15","2":"JN889685_1145262976"},{"1":"15","2":"JN889686_1145262974"},{"1":"15","2":"JN889687_1145263702"},{"1":"15","2":"JN889695_1145365034"},{"1":"15","2":"JQ085943_1145262864"},{"1":"16","2":"AF238034_897087993"},{"1":"17","2":"FJ667521_1145342210"},{"1":"18","2":"JN889535_1145183719"},{"1":"19","2":"JN889684_1145262959"},{"1":"20","2":"JN889688_2427818875"},{"1":"21","2":"KF302765_1145262889"},{"1":"22","2":"KF302832_1145262884"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p><img src="shrew2.jpg" /></p>
</div>
<div id="bonus-are-my-results-supported" class="section level4 hasAnchor" number="6.4.4.3">
<h4><span class="header-section-number">6.4.4.3</span> BONUS: Are my results supported?<a href="species-limits.html#bonus-are-my-results-supported" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Congrats - you’ve made it through the process of GMYC species delimitation!</p>
<p>So, what’s next? Once you have your species delimitation results, it’s important to evaluate whether or not your results are supported. While the likelihood values included in our GMYC results can be used to evaluate overall support for our species delimitation model, we can also look at how that support is spread across our evolutionary tree.</p>
<p>The following code can be used to estimate support values for the branches of our summary MCC tree. The higher the support value, the more confidence we can have in the particular relationship it represents. It’s wise to be wary of areas in your tree with lower support values, as they may indicate a lack of signal or conflicting signal in the data that prevents us from making meaningful inferences about those particular evolutionary relationships.</p>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb15-1"><a href="species-limits.html#cb15-1" aria-hidden="true" tabindex="-1"></a><span class="co">#estimate support values</span></span>
<span id="cb15-2"><a href="species-limits.html#cb15-2" aria-hidden="true" tabindex="-1"></a>yule_support <span class="ot">&lt;-</span> <span class="fu">gmyc.support</span>(my_gmyc)  </span>
<span id="cb15-3"><a href="species-limits.html#cb15-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb15-4"><a href="species-limits.html#cb15-4" aria-hidden="true" tabindex="-1"></a><span class="co">#only plot relevant support values</span></span>
<span id="cb15-5"><a href="species-limits.html#cb15-5" aria-hidden="true" tabindex="-1"></a><span class="fu">is.na</span>(yule_support[yule_support <span class="sc">==</span> <span class="dv">0</span>]) <span class="ot">&lt;-</span> <span class="cn">TRUE</span> </span>
<span id="cb15-6"><a href="species-limits.html#cb15-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb15-7"><a href="species-limits.html#cb15-7" aria-hidden="true" tabindex="-1"></a><span class="co">#plot tree and support values (</span><span class="al">NOTE</span><span class="co">: run both these lines of code at the same time)</span></span>
<span id="cb15-8"><a href="species-limits.html#cb15-8" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(mcc_tree, <span class="at">cex=</span>.<span class="dv">6</span>, <span class="at">no.margin=</span><span class="cn">TRUE</span>)         </span>
<span id="cb15-9"><a href="species-limits.html#cb15-9" aria-hidden="true" tabindex="-1"></a><span class="fu">nodelabels</span>(<span class="fu">round</span>(yule_support, <span class="dv">2</span>), <span class="at">cex=</span>.<span class="dv">7</span>)     </span></code></pre></div>
<p><img src="images/unnamed-chunk-14-1.png" width="672" /></p>
<p>We can also look at how our delimited species are spread geographically. Run the code below to combine your GMYC delimitation results with their associated geographic occurrence data to visualize how your species are spread across the globe. In the resulting figure, each delimited species is assigned a to a unique color and individuals are plotted according to their collection locality.</p>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb16-1"><a href="species-limits.html#cb16-1" aria-hidden="true" tabindex="-1"></a><span class="co">#load required packages</span></span>
<span id="cb16-2"><a href="species-limits.html#cb16-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tools)</span>
<span id="cb16-3"><a href="species-limits.html#cb16-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggplot2)</span>
<span id="cb16-4"><a href="species-limits.html#cb16-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(maps)</span></code></pre></div>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb17-1"><a href="species-limits.html#cb17-1" aria-hidden="true" tabindex="-1"></a><span class="co">#save GMYC species delimitation results and occurrences</span></span>
<span id="cb17-2"><a href="species-limits.html#cb17-2" aria-hidden="true" tabindex="-1"></a>spec_delim<span class="ot">&lt;-</span><span class="fu">spec.list</span>(my_gmyc)</span></code></pre></div>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="species-limits.html#cb18-1" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span> (f <span class="cf">in</span> myfiles) {</span>
<span id="cb18-2"><a href="species-limits.html#cb18-2" aria-hidden="true" tabindex="-1"></a>  sp <span class="ot">&lt;-</span> <span class="fu">basename</span>(f)</span>
<span id="cb18-3"><a href="species-limits.html#cb18-3" aria-hidden="true" tabindex="-1"></a>  sp <span class="ot">&lt;-</span> <span class="fu">file_path_sans_ext</span>(sp)</span>
<span id="cb18-4"><a href="species-limits.html#cb18-4" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb18-5"><a href="species-limits.html#cb18-5" aria-hidden="true" tabindex="-1"></a>  s <span class="ot">&lt;-</span> <span class="fu">sub</span>(<span class="st">&quot;/[^/]+$&quot;</span>, <span class="st">&quot;&quot;</span>, f)</span>
<span id="cb18-6"><a href="species-limits.html#cb18-6" aria-hidden="true" tabindex="-1"></a>  c <span class="ot">&lt;-</span> <span class="fu">paste</span>(s, <span class="st">&quot;/occurrences.txt&quot;</span>, <span class="at">sep=</span><span class="st">&quot;&quot;</span>)</span>
<span id="cb18-7"><a href="species-limits.html#cb18-7" aria-hidden="true" tabindex="-1"></a>  occurrences <span class="ot">&lt;-</span> <span class="fu">read.delim</span>(c)</span>
<span id="cb18-8"><a href="species-limits.html#cb18-8" aria-hidden="true" tabindex="-1"></a>  xy_id <span class="ot">&lt;-</span> occurrences[,<span class="fu">c</span>(<span class="dv">1</span>,<span class="dv">5</span><span class="sc">:</span><span class="dv">4</span>)]</span>
<span id="cb18-9"><a href="species-limits.html#cb18-9" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb18-10"><a href="species-limits.html#cb18-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">write.table</span>(<span class="fu">data.frame</span>(sp, xy_id), <span class="at">file=</span><span class="st">&quot;d_occur.csv&quot;</span>, <span class="at">quote=</span><span class="cn">FALSE</span>, <span class="at">row.names=</span><span class="cn">FALSE</span>, <span class="at">col.names=</span><span class="sc">!</span><span class="fu">file.exists</span>(<span class="st">&quot;d_occur.csv&quot;</span>), <span class="at">append=</span><span class="cn">TRUE</span>, <span class="at">sep=</span><span class="st">&quot;,&quot;</span>)</span>
<span id="cb18-11"><a href="species-limits.html#cb18-11" aria-hidden="true" tabindex="-1"></a>}</span></code></pre></div>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb19-1"><a href="species-limits.html#cb19-1" aria-hidden="true" tabindex="-1"></a><span class="co">#read in occurrence data from PhylogatR</span></span>
<span id="cb19-2"><a href="species-limits.html#cb19-2" aria-hidden="true" tabindex="-1"></a>occurrences<span class="ot">&lt;-</span><span class="fu">as.data.frame</span>(<span class="fu">read.table</span>(<span class="st">&quot;d_occur.csv&quot;</span>, <span class="at">header=</span><span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">&quot;,&quot;</span>))</span>
<span id="cb19-3"><a href="species-limits.html#cb19-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-4"><a href="species-limits.html#cb19-4" aria-hidden="true" tabindex="-1"></a><span class="co">#merge occurrences and species delimitation results</span></span>
<span id="cb19-5"><a href="species-limits.html#cb19-5" aria-hidden="true" tabindex="-1"></a>merge<span class="ot">&lt;-</span><span class="fu">merge</span>(spec_delim, occurrences, <span class="at">by.x =</span> <span class="st">&quot;sample_name&quot;</span>, <span class="at">by.y =</span> <span class="st">&quot;phylogatr_id&quot;</span>)</span>
<span id="cb19-6"><a href="species-limits.html#cb19-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-7"><a href="species-limits.html#cb19-7" aria-hidden="true" tabindex="-1"></a><span class="co">#plot species groups on map</span></span>
<span id="cb19-8"><a href="species-limits.html#cb19-8" aria-hidden="true" tabindex="-1"></a>world<span class="ot">&lt;-</span><span class="fu">map_data</span>(<span class="st">&quot;world&quot;</span>)</span>
<span id="cb19-9"><a href="species-limits.html#cb19-9" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>()<span class="sc">+</span> </span>
<span id="cb19-10"><a href="species-limits.html#cb19-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_polygon</span>(<span class="at">data =</span> world, <span class="fu">aes</span>(<span class="at">x=</span>long, <span class="at">y =</span> lat, <span class="at">group =</span> group), <span class="at">fill =</span> <span class="st">&quot;lightgrey&quot;</span>, <span class="at">color =</span> <span class="st">&quot;gray30&quot;</span>, <span class="at">size =</span> .<span class="dv">25</span>)<span class="sc">+</span></span>
<span id="cb19-11"><a href="species-limits.html#cb19-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">coord_fixed</span>(<span class="fl">1.3</span>)<span class="sc">+</span></span>
<span id="cb19-12"><a href="species-limits.html#cb19-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">&quot;Longitude&quot;</span>)<span class="sc">+</span></span>
<span id="cb19-13"><a href="species-limits.html#cb19-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">&quot;Latitude&quot;</span>)<span class="sc">+</span></span>
<span id="cb19-14"><a href="species-limits.html#cb19-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggtitle</span>(<span class="st">&quot;GMYC Species Delimitation Results&quot;</span>)<span class="sc">+</span></span>
<span id="cb19-15"><a href="species-limits.html#cb19-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="at">data =</span> merge, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">as.numeric</span>(longitude), <span class="at">y =</span> <span class="fu">as.numeric</span>(latitude), <span class="at">color =</span> <span class="fu">factor</span>(GMYC_spec)), <span class="at">size =</span> <span class="fl">0.3</span>)<span class="sc">+</span></span>
<span id="cb19-16"><a href="species-limits.html#cb19-16" aria-hidden="true" tabindex="-1"></a> <span class="fu">labs</span>(<span class="at">color =</span> <span class="st">&quot;GMYC species&quot;</span>)</span></code></pre></div>
<p><img src="images/unnamed-chunk-18-1.png" width="672" /></p>

</div>
</div>
</div>
</div>
<div class="footnotes">
<hr />
<ol start="1">
<li id="fn1"><p>See <a href="https://github.com/ropensci/beautier" target="_blank">here</a> for <code>beautier</code> parameter settings<a href="species-limits.html#fnref1" class="footnote-back">↩︎</a></p></li>
</ol>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="population-genetics-and-conservation.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="predictive-phylogeography.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
